<!DOCTYPE html>
<html>
 <head>
  <meta charset="utf-8"/>
 </head>
 <body>
  <div class="page" data-page-id="0">
   <h1>
    The genetic prehistory of southern Africa
   </h1>
   <p>
    Joseph K. Pickrell
    <sup>
     1*
    </sup>
    , Nick Patterson
    <sup>
     2
    </sup>
    , Chiara Barbieri
    <sup>
     3,14
    </sup>
    , Falko Berthold
    <sup>
     3,15
    </sup>
    , Linda Gerlach
    <sup>
     3,15
    </sup>
    , Tom Güldemann
    <sup>
     4
    </sup>
    , Blesswell Kure
    <sup>
     5
    </sup>
    , Sununguko Wata Mpoloka
    <sup>
     6
    </sup>
    , Hiroshi Nakagawa
    <sup>
     7
    </sup>
    , Christfried Naumann
    <sup>
     4
    </sup>
    , Mark Lipson
    <sup>
     8
    </sup>
    , Po-Ru Loh
    <sup>
     8
    </sup>
    , Joseph Lachance
    <sup>
     9
    </sup>
    , Joanna Mountain
    <sup>
     10
    </sup>
    , Carlos Bustamante
    <sup>
     11
    </sup>
    , Bonnie Berger
    <sup>
     8
    </sup>
    , Sarah Tishkoff
    <sup>
     9
    </sup>
    , Brenna Henn
    <sup>
     11
    </sup>
    , Mark Stoneking
    <sup>
     12
    </sup>
    , David Reich
    <sup>
     1,2*
    </sup>
    , Brigitte Pakendorf
    <sup>
     3,13*
    </sup>
   </p>
   <p>
    <sup>
     1
    </sup>
    Department of Genetics, Harvard Medical School, Boston
   </p>
   <p>
    <sup>
     2
    </sup>
    Broad Institute of MIT and Harvard, Cambridge
   </p>
   <p>
    <sup>
     3
    </sup>
    Max Planck Research Group on Comparative Population Linguistics, MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     4
    </sup>
    Seminar für Afrikawissenschaften, Humboldt University, Berlin and Department of Linguistics, MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     5
    </sup>
    Department of Aesthetics and Communication, Aarhus University, Aarhus
   </p>
   <p>
    <sup>
     6
    </sup>
    Department of Biological Sciences, University of Botswana
   </p>
   <p>
    <sup>
     7
    </sup>
    Institute of Global Studies, Tokyo University of Foreign Studies, Tokyo
   </p>
   <p>
    <sup>
     8
    </sup>
    Department of Mathematics and Computer Science and Artificial Intelligence Laboratory, MIT, Cambridge
   </p>
   <p>
    <sup>
     9
    </sup>
    Departments of Biology and Genetics, University of Pennsylvania, Philadelphia
   </p>
   <p>
    <sup>
     10
    </sup>
    23andMe, Mountain View
   </p>
   <p>
    <sup>
     11
    </sup>
    Department of Genetics, Stanford University
   </p>
   <p>
    <sup>
     12
    </sup>
    Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     13
    </sup>
    Current affiliation: Laboratoire Dynamique du Langage, UMR5596, CNRS and Université Lyon Lumière 2
   </p>
   <p>
    <sup>
     14
    </sup>
    Current affiliation: Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     15
    </sup>
    Current affiliation: Department of Linguistics, MPI for Evolutionary Anthropology, Leipzig and Seminar für Afrikawissenschaften, Humboldt University, Berlin
   </p>
   <p>
    *To whom correspondence should be addressed:
    <a href="mailto:joseph_pickrell@hms.harvard.edu">joseph_pickrell@hms.harvard.edu</a>
    (JKP)
    <a href="mailto:reich@genetics.med.harvard.edu">reich@genetics.med.harvard.edu</a>
    (DR),
    <a href="mailto:Brigitte.Pakendorf@ish-lyon.cnrs.fr">Brigitte.Pakendorf@ish-lyon.cnrs.fr</a>
    (BP),
   </p>
   <p>
    17 September, 2012
   </p>
   <p>
    To appear in
    <i>
     Nature Communications
    </i>
   </p>
  </div>
  <div class="page" data-page-id="1">
   <h2>
    <b>
     Abstract
    </b>
   </h2>
   <p>
    Southern and eastern African populations that speak non-Bantu languages with click consonants are known to harbour some of the most ancient genetic lineages in humans, but their relationships are poorly understood. Here, we report data from 23 populations analyzed at over half a million single nucleotide polymorphisms, using a genome-wide array designed for studying human history. The southern African Khoisan fall into two genetic groups, loosely corresponding to the northwestern and southeastern Kalahari, which we show separated within the last 30,000 years. We find that all individuals derive at least a few percent of their genomes from admixture with non-Khoisan populations that began approximately 1,200 years ago. In addition, the east African Hadza and Sandawe derive a fraction of their ancestry from admixture with a population related to the Khoisan, supporting the hypothesis of an ancient link between southern and eastern Africa.
   </p>
  </div>
  <div class="page" data-page-id="2">
   <h2>
    Introduction
   </h2>
   <p>
    The prehistory of the populations of southern Africa that speak non-Bantu languages with click consonants (here called Khoisan without implying linguistic unity) is poorly understood. A major open question concerns the relationships among these populations, who harbour extensive linguistic diversity (there are three Khoisan language families
    <sup>
     1-4
    </sup>
    ) as well as variable modes of subsistence (while most Khoisan groups are hunter-gatherers, some are pastoralists). A second major question is the historical relationship of the southern African populations to two populations in eastern Africa that are or previously were hunter-gatherers and that also speak languages with click consonants (the Hadza and Sandawe). It has been hypothesized that the eastern Africans descend in part from a Khoisan-related hunter-gatherer population that once occupied a region ranging over much of southern and eastern Africa
    <sup>
     5
    </sup>
    . However, the anthropological and archaeological evidence for this hypothesis is contested
    <sup>
     6,7
    </sup>
    . Apart from shared use of click consonants, there is no linguistic evidence that the non-Bantu languages in southern Africa and Hadza stem from a common ancestor
    <sup>
     8-10
    </sup>
    , although a potential ancestral link between Sandawe and the Khoe-Kwadi family has been suggested
    <sup>
     4,11
    </sup>
    .
   </p>
   <p>
    Genomic studies have the potential to shed new light on the history of the Khoisan. Previous genetic studies based largely on single loci (mitochondrial DNA and the Y chromosome) have documented that the Khoisan carry some of the most ancient lineages in humans
    <sup>
     12,13
    </sup>
    and have suggested deep genetic links between the Khoisan and the Sandawe and Hadza
    <sup>
     13
    </sup>
    . However, single locus studies have limited resolution. While some genome-wide studies have included southern Africans, they have largely focused on a single Khoisan group, making it impossible to elucidate relationships among these populations
    <sup>
     14-16
    </sup>
    . The few studies of more than one Khoisan group have not included enough populations to form a clear picture of the pattern of sub-structure and population relationships within southern Africa
    <sup>
     17,18
    </sup>
    .
   </p>
   <p>
    Here we present a high-resolution study of the genomic relationships of southern and eastern African populations that speak languages characterized by heavy use of click consonants. Our study capitalizes on three novel resources: (1) a unique collection of southern African DNA samples encompassing most of the linguistic and cultural diversity of Khoisan groups; (2) a single nucleotide polymorphism (SNP) array that is the first to include polymorphisms discovered in Khoisan; and (3) new methods of statistical analysis, some of which we introduce here for the first time, that allow us to make inferences about historical relationships even in the presence of admixture.
   </p>
   <h2>
    Results
   </h2>
   <h3>
    Data set
   </h3>
   <p>
    We genotyped 565,259 SNPs in 187 individuals from 22 African populations (16 Khoisan populations and 5 neighboring populations speaking Bantu languages shown in Figure 1A, plus
   </p>
  </div>
  <div class="page" data-page-id="3">
   <p>
    the Hadza) using the Affymetrix Human Origins array
    <sup>
     19
    </sup>
    . This array is specifically designed for studies of population history: it contains panels of SNPs discovered by sequencing a single individual of known ancestry (including a Khoisan individual), providing precise control of the SNP ascertainment scheme and making it possible to answer questions that are more difficult to address using data from SNP arrays designed for medical genetics. We genotyped populations speaking languages from all three Khoisan language families (Tuu, Kx'a, and Khoe-Kwadi
    <sup>
     1-4,8
    </sup>
    ) (Supplementary Table S1, Supplementary Figure S1). We then merged the data with whole-genome sequencing data from five Sandawe individuals and five Hadza individuals
    <sup>
     20
    </sup>
    . Finally, we supplemented this with previously collected Affymetrix Human Origins array data on Dinka, Mbuti, Biaka, Yoruba, and other African and non-African populations
    <sup>
     19,21
    </sup>
    .
   </p>
   <h3>
    <b>
     The Khoisan genetically cluster into two major groups
    </b>
   </h3>
   <p>
    We performed a qualitative exploration of southern African population relationships using principal component analysis (PCA)
    <sup>
     22
    </sup>
    (Supplementary Figures S2-S5). We capitalized on the design of the Human Origins array by performing the analysis using three different panels of SNPs, each of which reveals different aspects of population structure (Supplementary Figure S4). The Yoruba SNPs highlight structure within non-Khoisan Africans, while the French SNPs highlight European ancestry in the Nama (consistent with historic documentation
    <sup>
     23
    </sup>
    ) and hint at European or east African ancestry in some Khoe groups (Supplementary Figures S4, S6). The SNPs ascertained in a Jul'hoan individual (HGDP "San") reveal structure invisible to the other panels (Figure 1B). The PCA based on these SNPs divides Africans into three broad clusters: a predominantly non-Khoisan cluster and two Khoisan clusters. The Khoisan clusters do not correspond to linguistic affiliation; while one is comprised of Jul'hoan_North and Jul'hoan_South, who speak closely related languages/dialects, the other includes populations speaking languages belonging to all three language families (Supplementary Figure S1). The Khoisan clusters instead reflect geography, corresponding roughly to the northwestern and southeastern Kalahari (Figure 1A). On a fine scale, the PCA plot also identifies substructure within individual populations (Supplementary Methods), as well as cases of discordance between linguistic and genetic affiliation that suggest language shift with little accompanying gene flow. A particularly striking example is the Damara, who cluster with non-Khoisan populations despite speaking a Khoe language. This suggests that the Damara were a non-Khoisan population who acquired their language from their Khoisan neighbours (the Nama
    <sup>
     24
    </sup>
    ) with little Khoisan gene flow
    <sup>
     25
    </sup>
    .
   </p>
   <h3>
    <b>
     Admixture in southern Africa is primarily related to the Bantu Expansion
    </b>
   </h3>
   <p>
    A number of populations occupy intermediate positions between the three major clusters (non-Khoisan, northwestern Kalahari, and southeastern Kalahari) in Figure 1B. This suggests historical gene flow; however, PCA does not constitute a formal test of admixture. We next created a filtered dataset, excluding individuals who were outliers with respect to others from the same self-identified ethno-linguistic group (Supplementary Table S2, Supplementary Figure S7).
   </p>
  </div>
  <div class="page" data-page-id="4">
   <p>
    Formal tests for a history of mixture (“three population tests”
    <sup>
     26
    </sup>
    ) confirmed many examples of population mixture (Supplementary Table S3). Most populations are admixed between a non-Khoisan population and a population from either the northwestern Kalahari or the southeastern Kalahari cluster (Supplementary Table S3). The one exception is the Naro, who are genetically admixed between northwestern and southeastern Kalahari populations, just as they are intermediate geographically (Figure 1A).
   </p>
   <p>
    Several Khoisan populations that are at the extremes of Figure 1B—the Jul’hoan_North, Jul’hoan_South, ≠Hoan, Taa_North, and Taa_East—do not show evidence of admixture by formal three-population tests; some of these also show no evidence of admixture in STRUCTURE-like analyses (Supplementary Figure S8). This is intriguing because if these populations were indeed unadmixed, they could be used as representatives of the ancestral northwestern and southeastern Kalahari populations. However, the three-population tests have limited power, and STRUCTURE-like methods may not be able to detect admixture if there is no unadmixed relative in the dataset (Supplementary Figure S9). We therefore developed a novel test for admixture that takes advantage of the fact that if and only if population mixture occurred, we expect to detect linkage disequilibrium (LD)—non-random association of SNP genotypes—that is correlated to the allele frequency differences between the two ancestral populations
    <sup>
     27,28
    </sup>
    (Supplementary Figure S10). In all five populations, we observe LD that decays exponentially with genetic distance. This is evidence that all Khoisan populations in our study, even the most isolated, are admixed with non-Khoisan populations (Figure 2A; Supplementary Figures S11, S12).
   </p>
   <p>
    To estimate the proportion of admixture in the different Khoisan populations and to estimate when it occurred, we performed a quantitative analysis of the LD decay. Using population genetic theory presented in the Supplementary Methods, we show how the
    <i>
     proportion
    </i>
    of admixture can be derived from the
    <i>
     amplitude
    </i>
    of the exponential curve, that is, the point from which LD begins to decay. It has previously been shown that the
    <i>
     time
    </i>
    since admixture can be derived from the
    <i>
     rate
    </i>
    of LD decay
    <sup>
     27,29,30
    </sup>
    , and we also use this information below. The amplitude provides evidence of approximately 6% non-Khoisan ancestry in the Jul’hoan_North (Figure 2A, Supplementary Methods). We then inferred the admixture proportions in the other southern Africans using a modified
    <math>
     f_4
    </math>
    ratio estimate
    <sup>
     19
    </sup>
    that accounts for the admixture in the reference population (Figure 2B, Supplementary Table 4, Supplementary Methods). The estimated proportion of non-Khoisan ancestry in non-Bantu speakers ranges from 6% (Jul’hoan_North) to around 90% (Damara) (Figure 2B).
   </p>
   <p>
    We next estimated the time of admixture based on the extent of the LD. Ideally we would like to infer a distribution of times to learn when the gene flow began and when it reached its peak
    <sup>
     31
    </sup>
    , but with current methods it is not possible to make robust statements about mixture events that are older than a dozen generations (due to errors in inference of local ancestry
    <sup>
     30
    </sup>
    ). Instead, we estimate a single date for the gene flow, which can be thought of as the weighted average of the admixture times
    <sup>
     27
    </sup>
    . We estimated this separately in each southern African population (Figure 2C;
   </p>
  </div>
  <div class="page" data-page-id="5">
   <p>
    Supplementary Table S4, Supplementary Figure S13). The earliest dates are around 40 generations (approximately 1,200 years) in the past, and the most recent dates are within the past few hundred years (though many of the populations with recent dates show evidence of additional gene flow before this; Supplementary Figure S14). These dates are consistent with archeological evidence for the arrival of both east African pastoralists as well as agriculturalists (probably Bantu speakers) in southern Africa 2,000-1,200 years ago
    <sup>
     32-35
    </sup>
    . PCA shows that the majority of admixture in the Khoisan is more closely related to the Yoruba (from west Africa, linguistically related to Bantu speakers) than to the Dinka (from northeastern Africa) (Supplementary Figure S5), though our data are consistent with additional east African ancestry in some Khoe-speakers (Supplementary Methods).
   </p>
   <h3>
    <b>
     NW and SE Kalahari Khoisan split within the last 30,000 years
    </b>
   </h3>
   <p>
    To infer the date of population separation between the northwestern and southeastern Kalahari Khoisan, we developed a new methodology enabled by the design of the Human Origins array. The method is based on the rate at which Jul'hoan-ascertained SNPs are observed to be monomorphic in the other populations. The excess of monomorphic SNPs beyond that expected due to genetic drift alone reflects new mutations that have arisen in the Jul'hoan_North since the two populations split, and thus provides a measure of the time since the split (Supplementary Methods, Supplementary Figure S15). We verified that this approach can provide accurate estimates of population split dates by simulation (Supplementary Figures S16, S17), and estimated that the split of the northwestern and southeastern Kalahari Khoisan occurred in the last 30,000 years (Supplementary Figures S18, S19). However, this date is likely overestimated due to Bantu-related gene flow in these populations, and so should be treated as an upper bound (Supplementary Figure S17).
   </p>
   <h3>
    <b>
     A genetic link between southern and eastern African populations
    </b>
   </h3>
   <p>
    We examined two eastern African populations that speak languages with click consonants (Hadza and Sandawe) along with representative southern African populations using
    <i>
     TreeMix
    </i>
    <sup>
     36
    </sup>
    . This method fits a population graph—a generalization of a phylogenetic tree that incorporates the possibility of population mixture—to the allele frequency correlation patterns among a set of sampled populations.
    <i>
     TreeMix
    </i>
    infers that the Hadza are admixed between a Khoisan population (equally related to both the northwestern and southeastern Kalahari groups) and a population most closely related to the Dinka, with about
    <math>
     23 \pm 2\%
    </math>
    Khoisan-related ancestry (Supplementary Figure S20). The Sandawe show a similar signal, though weaker;
    <i>
     TreeMix
    </i>
    estimates that the Sandawe trace about
    <math>
     18 \pm 2\%
    </math>
    of their ancestry to admixture with a population related to the Khoisan (Supplementary Figure S21).
   </p>
   <p>
    <i>
     TreeMix
    </i>
    fits a single model to a large number of populations, and in principle the finding of deep connections between southern and eastern Africans could be an artifact of modeling a complex history with a single admixture event. To explore the robustness of this finding, we used a four-
   </p>
  </div>
  <div class="page" data-page-id="6">
   <p>
    population test
    <sup>
     26
    </sup>
    to determine whether the tree [Chimp, Jul'hoan_North,[Hadza,Dinka]] is a good fit to the genome-wide allele frequencies. This tree fails with a Z-score of -4.8 (
    <math>
     p = 8 \times 10^{-7}
    </math>
    ), indicating an excess of correlation in allele frequencies between the Jul'hoan_North and Hadza. A consistent signal is seen in the Sandawe, although it is weaker (Z-score of -2.1;
    <math>
     p = 0.018
    </math>
    ). Both the Hadza and the Sandawe show evidence of western Eurasian ancestry (perhaps reflecting gene flow from previously-admixed neighboring populations
    <sup>
     16
    </sup>
    ); the weaker signal of relatedness between the Khoisan and the Sandawe may be due to a higher proportion of west Eurasian ancestry in the Sandawe (Supplementary Figure S22). These findings are consistent with the hypothesis that the Hadza and Sandawe harbor a proportion of their ancestry from a population related to southern Africans. Alternatively, more gene flow from an (as yet undiscovered) archaic human population into the ancestors of the Dinka than the Hadza (or Sandawe) could produce this signal. It has been suggested that the Mbuti and Biaka populations in central Africa may also be related to the Khoisan
    <sup>
     16,17
    </sup>
    ; while our analyses show that these populations do carry deep human lineages, they do not share the signal of relatedness to the Khoisan that we are focusing on here (Supplementary Figure S23). In sum, these results strongly suggest a genetic link between populations in southern and eastern Africa that speak non-Bantu languages with heavy use of click consonants.
   </p>
   <h3>
    <b>
     A unified model for the relationship of southern and eastern Africans
    </b>
   </h3>
   <p>
    We used
    <i>
     TreeMix
    </i>
    to build a unified model for the ancestral relationships between the Khoisan and eastern African populations, taking into account the confounding factor that all the populations harbor recent admixture. To do this, we extended
    <i>
     TreeMix
    </i>
    to subtract out the effect of gene flow from non-Khoisan populations (Supplementary Methods). This analysis provides strong evidence for a shared origin for the Khoisan-related genetic material in the Hadza and Sandawe. The Khoisan-related ancestry in the Hadza and Sandawe forms one clade, while the southern African Khoisan form a second clade consisting of the northwestern and southeastern Kalahari groups (Figure 3).
   </p>
   <h2>
    <b>
     Discussion
    </b>
   </h2>
   <p>
    Our analysis of diverse southern and eastern African populations has documented deep structure in southern Africa that was previously unknown: a division between NW and SE Kalahari groups that arose within the past 30,000 years. We have also detected admixture in all Khoisan reflecting gene flow from Bantu-speaking agriculturalists and/or eastern African pastoralists within the past 1,200 years. Finally, we demonstrate an ancient link between the Khoisan and the Hadza and Sandawe in eastern Africa. This has implications for the geographic origin of modern humans, for which both eastern and southern Africa have been proposed
    <sup>
     17,37,38
    </sup>
    . Present-day populations in southern and eastern Africa are located on both sides of the deepest split of the tree (Figure 3), and thus from the perspective of phylogeography, our results are equally consistent with both of these locations as the origin of modern humans.
   </p>
  </div>
  <div class="page" data-page-id="7">
   <h2>
    Methods
   </h2>
   <h3>
    Data
   </h3>
   <p>
    The southern African samples included in this study were collected in various locations in Botswana and Namibia as part of a multidisciplinary project, after ethical clearance by the Review Board of the University of Leipzig and with prior permission of the Ministry of Youth, Sport and Culture of Botswana and the Ministry of Health and Social Services of Namibia. Approximately 2ml of saliva were collected in tubes containing 2ml of stabilizing buffer. Each sample was genotyped on the Affymetrix Human Origins array
    <sup>
     19
    </sup>
    and merged with additional samples
    <sup>
     19–21
    </sup>
    (Supplementary Methods).
   </p>
   <p>
    The SNPs on the Human Origins array are organized into panels of SNPs discovered in different individuals. Except where otherwise noted, we restrict ourselves to using the 150,425 autosomal SNPs discovered in a single Jul'hoan_North (HGDP "San") individual. The exceptions to this are all ROLLOFF analyses (e.g., Figures 2A and 2C), where we used all 565,259 autosomal SNPs on the array. For analyses including the Hadza and Sandawe, some SNPs were removed due to genotyping or sequencing errors (Supplementary Methods); the corresponding number of Jul'hoan-ascertained SNPs used when analyzing these populations was 146,843.
   </p>
   <h3>
    Analysis of population structure and mixture
   </h3>
   <p>
    PCA was performed using smartpca
    <sup>
     22
    </sup>
    v9003. We tested for admixture using three- and four-population tests
    <sup>
     19
    </sup>
    . To estimate admixture dates, we used ROLLOFF v625
    <sup>
     27
    </sup>
    .
   </p>
   <p>
    To estimate the admixture proportion of the Jul'hoan_North, we binned SNPs according to the genetic distance between them (with a bin size of 0.01 cM), and for each pair of SNPs we calculated the linkage disequilibrium between them as well as the product of the allele frequency differences between the Jul'hoan_North and the Yoruba. In each bin, we regressed the amount of LD against the product of allele frequency differences, and fit an exponential curve to the resulting regression coefficients. The intercept of the fitted exponential curve is expected to be
    <math>
     f/(1-f)
    </math>
    , where
    <math>
     f
    </math>
    is the mixture fraction (see Supplementary Methods for details).
   </p>
   <h3>
    Estimating population divergence times
   </h3>
   <p>
    To date the split between the NW and SE Kalahari groups, we developed a new method based on the fact that after the split of two populations, a given lineage from one of the populations accumulates mutations (that are not observed in the other population) at a clock-like rate that is proportional to years. Our method enables us to count these mutations, and convert this count to absolute time (see Supplementary Methods for details).
   </p>
  </div>
  <div class="page" data-page-id="8">
   <h3>
    <b>
     Building population trees
    </b>
   </h3>
   <p>
    To build population trees in the presence of admixture, we modified the
    <i>
     TreeMix
    </i>
    model
    <sup>
     36
    </sup>
    . We first constructed a tree using unadmixed populations (Chimpanzee, Yoruba, Dinka, Europeans, and East Asians) and then added admixed Khoisan populations to this tree using their estimated admixture proportions (see Supplementary Methods for details).
   </p>
  </div>
  <div class="page" data-page-id="9">
   <h2>
    Figure Legends
   </h2>
   <p>
    <b>
     Figure 1: Population structure in southern Africa. A. Approximate locations of sampled populations.
    </b>
    Populations are colored according to linguistic affiliation, as displayed in the legend and Figure S1. The speckled region is the Kalahari semi-desert.
    <b>
     B. PCA on SNPs ascertained in a Jul'hoan individual.
    </b>
    Shown are the positions of each individual along the first and second axes of genetic variation, with symbols denoting the individual's population and linguistic affiliation using the same color coding as in panel A.
   </p>
   <p>
    <b>
     Figure 2: All Khoisan populations are admixed. A. Admixture LD in the Jul'hoan_North.
    </b>
    For each pair of SNPs in the Jul'hoan_North (black) or the Yoruba (grey) we estimate the linkage disequilibrium as well as the product of the differences in allele frequency between the Jul'hoan_North and the Yoruba. (We use the Yoruba as a proxy for the non-Khoisan, presumably Bantu-speaking ancestral population because there has been very little change in allele frequencies between Niger-Congo-speaking groups). We then binned pairs of SNPs by the genetic distance between them. For each bin, we plot the regression coefficient (over SNP pairs in the bin) from regressing the level of LD on the product of the allele frequency differences. The rate at which this curve decays is informative about the date of admixture, while the amplitude of the curve is informative about the proportion of admixture (Supplementary Methods). In black is the curve if we assume the Jul'hoan_North are admixed; in grey if we assume the Yoruba are admixed (which serves as a negative control). The red line is the exponential curve fitted to the black points.
    <b>
     B. Estimates of mixture proportions.
    </b>
    We used the modified
    <math>
     f_4
    </math>
    ratio
    <sup>
     19
    </sup>
    (Supplementary Methods) to estimate the fraction of non-Khoisan ancestry in each southern African population.
    <b>
     C. Estimates of mixture dates.
    </b>
    We used the rate at which admixture LD decays to estimate dates of admixture for all southern African populations (Supplementary Methods). We plot the means, with ranges representing one standard error. Not shown are the Wambo, who have no detectable curve, and hence may be unadmixed. The estimates of the mixture proportions and dates are also presented in Supplementary Table S4.
   </p>
   <p>
    <b>
     Figure 3: Relationships among Khoisan and eastern Africans after removing non-Khoisan admixture.
    </b>
    We extended
    <i>
     TreeMix
    </i>
    to build trees after subtracting out the effect of known admixture (Supplementary Methods) and then applied it to the Khoisan (excluding the Damara, who are genetically close to non-Khoisan). Populations are coloured according to their linguistic affiliation (Khoisan) or geographic location (dark grey = non-Khoisan African, light grey = Eurasian), and the chimpanzee was used as an outgroup. The bar chart next to each population shows the estimated ancestry proportions for each population: blue is the proportion of Khoisan ancestry, and red is the proportion of non-Khoisan ancestry. Note that the actual source of these two ancestries may vary among populations. The proportions are not identical to those presented in Figure 2B because of small differences in how they are estimated. The black dots show splits supported by more than 95% of bootstrap replicates, and the grey dots those supported by more than 80% of bootstrap replicates.
   </p>
  </div>
  <div class="page" data-page-id="10">
   <h2>
    <b>
     Author Contributions
    </b>
   </h2>
   <p>
    The southern African samples were collected by CB, FB, LG, TG, BK, SM, HN, CN, MS, and BP. Hadza and Sandawe samples and genotypes were provided by ST, JL, BH, JM, and CB. Analysis was performed primarily by JP and NP, with regular input and guidance from DR, MS, and BP. BB, PR, and ML contributed to the method for estimating admixture levels from LD. The study was designed by TG, DR, NP, MS and BP. The manuscript was written by JP, NP, MS, DR, and BP, with input from all authors.
   </p>
   <h2>
    <b>
     Acknowledgements
    </b>
   </h2>
   <p>
    This study focuses on the prehistory of populations as reflected in their genetic variation. It does not intend to evaluate the self-identification or cultural identity of any group, which consist of much more than just genetic ancestry. We sincerely thank all the sample donors for their participation in this study, the governments of Botswana and Namibia for supporting our research, Berendt Nakwe and Justin Magabe for assistance with sample collection, Serena Tucci, Vera Lede, Roland Schröder and Anne Butthof for assistance with sample preparation, and Marike Schreiber for drawing Figure 1A and Supplementary Figure 1. We thank Graham Coop, Jonathan Pritchard, Alan Barnard, and Gertrud Boden for comments on an earlier version of this manuscript. S.W.M. thanks the University of Botswana for research leave. This work, as part of the European Science Foundation EUROCORES Programme EuroBABEL, was supported by grants from the Deutsche Forschungsgemeinschaft (to BP and TG), by a Grant-in-Aid for Scientific Research (B), Ref. 19401019, Japan Society for the Promotion of Science (to HN), as well as by funds from the Max Planck Society (to BP and MS). ST was funded by NSF grant BCS-0827436 and NIH grants GM076637 and ES022577. JL was funded by NIH NRSA postdoctoral fellowship HG006648. JP, NP and DR were funded by NIH grant GM100233 and NSF HOMINID grant #1032255.
   </p>
  </div>
  <div class="page" data-page-id="11">
   <h2>
    References
   </h2>
   <ol style="list-style-type: none">
    <li>
     1. Heine, B. &amp; Honken, H. The Kx'a Family: A New Khoisan Genealogy.
     <i>
      Journal of Asian and African Studies
     </i>
     <b>
      79
     </b>
     , 5–36 (2010).
    </li>
    <li>
     2. Güldemann, T.
     <i>
      Studies in Tuu (Southern Khoisan)
     </i>
     . (Institut für Afrikanistik, Universität Leipzig: Leipzig, 2005).
    </li>
    <li>
     3. Güldemann, T. Reconstruction through de-construction: The marking of person, gender, and number in the Khoe family and Kwadi.
     <i>
      Diachronica
     </i>
     <b>
      21
     </b>
     , 251–306 (2004).
    </li>
    <li>
     4. Güldemann, Tom &amp; Elderkin, Edward D. On external genealogical relationships of the Khoe family.
     <i>
      Khoisan languages and linguistics: proceedings of the 1st International Symposium January 4-8, 2003, Riezlern/Kleinwalsertal
     </i>
     (2010).
    </li>
    <li>
     5. Tobias, P. V. Bushman hunter-gatherers: a study in human ecology.
     <i>
      Ecological studies in southern Africa
     </i>
     69–86 (1964).
    </li>
    <li>
     6. Morris, A. G. The myth of the East African 'Bushmen'.
     <i>
      The South African Archaeological Bulletin
     </i>
     85–90 (2003).
    </li>
    <li>
     7. Schepartz, L. A. Who were the later Pleistocene eastern Africans?
     <i>
      African archaeological review
     </i>
     <b>
      6
     </b>
     , 57–72 (1988).
    </li>
    <li>
     8. Sands, B. E.
     <i>
      Eastern and southern African Khoisan: evaluating claims of distant linguistic relationships
     </i>
     . (R. Köppe: Cologne, 1998).
    </li>
    <li>
     9. Güldemann, T. &amp; Vossen, R. Khoisan.
     <i>
      African languages: an introduction
     </i>
     99–122 (2000).
    </li>
    <li>
     10. Güldemann, T. Greenberg's 'case' for Khoisan: the morphological evidence.
     <i>
      Problems of linguistic-historical reconstruction in Africa
     </i>
     .
     <b>
      19
     </b>
     , 123–153 (2008).
    </li>
    <li>
     11. Elderkin, E. D. Diachronic inferences from basic sentence and noun structure in Central Khoisan and Sandawe.
     <i>
      Tagungsberichte des Internationalen Symposions 'Afrikanische Wildbeuter', Sankt Augustin, Januar 3-5, 1985
     </i>
     <b>
      7
     </b>
     , 131–156 (1986).
    </li>
    <li>
     12. Knight, A.
     <i>
      et al.
     </i>
     African Y chromosome and mtDNA divergence provides insight into the history of click languages.
     <i>
      Curr Biol
     </i>
     <b>
      13
     </b>
     , 464–73 (2003).
    </li>
    <li>
     13. Tishkoff, S. A.
     <i>
      et al.
     </i>
     History of click-speaking populations of Africa inferred from mtDNA and Y chromosome genetic variation.
     <i>
      Mol. Biol. Evol.
     </i>
     <b>
      24
     </b>
     , 2180–2195 (2007).
    </li>
    <li>
     14. Li, J. Z.
     <i>
      et al.
     </i>
     Worldwide human relationships inferred from genome-wide patterns of variation.
     <i>
      Science
     </i>
     <b>
      319
     </b>
     , 1100–1104 (2008).
    </li>
    <li>
     15. Rosenberg, N. A.
     <i>
      et al.
     </i>
     Genetic structure of human populations.
     <i>
      Science
     </i>
     <b>
      298
     </b>
     , 2381–2385 (2002).
    </li>
    <li>
     16. Tishkoff, S. A.
     <i>
      et al.
     </i>
     The genetic structure and history of Africans and African Americans.
     <i>
      Science
     </i>
     <b>
      324
     </b>
     , 1035–1044 (2009).
    </li>
    <li>
     17. Henn, B. M.
     <i>
      et al.
     </i>
     Hunter-gatherer genomic diversity suggests a southern African origin for modern humans.
     <i>
      Proc. Natl. Acad. Sci. U.S.A.
     </i>
     <b>
      108
     </b>
     , 5154–5162 (2011).
    </li>
    <li>
     18. Schuster, S. C.
     <i>
      et al.
     </i>
     Complete Khoisan and Bantu genomes from southern Africa.
     <i>
      Nature
     </i>
     <b>
      463
     </b>
     , 943–947 (2010).
    </li>
    <li>
     19. Patterson, N. J.
     <i>
      et al.
     </i>
     Ancient Admixture in Human History.
     <i>
      Genetics
     </i>
     (2012).doi:10.1534/genetics.112.145037
    </li>
    <li>
     20. Lachance, J.
     <i>
      et al.
     </i>
     Evolutionary History and Adaptation from High-Coverage Whole-Genome Sequences of Diverse African Hunter-Gatherers.
     <i>
      Cell
     </i>
     (2012).
    </li>
    <li>
     21. Meyer, M.
     <i>
      et al.
     </i>
     A High-Coverage Genome Sequence from an Archaic Denisovan Individual.
     <i>
      Science
     </i>
     (2012).doi:10.1126/science.1224344
    </li>
   </ol>
  </div>
  <div class="page" data-page-id="12">
   <ol style="list-style-type: none">
    <li>
     22. Patterson, N., Price, A. L. &amp; Reich, D. Population structure and eigenanalysis.
     <i>
      PLoS Genet
     </i>
     <b>
      2
     </b>
     , e190 (2006).
    </li>
    <li>
     23. Wallace, M.
     <i>
      A History of Namibia: From the Beginning to 1990
     </i>
     . (Columbia University Press: 2011).
    </li>
    <li>
     24. Barnard, A.
     <i>
      Hunters and herders of southern Africa: a comparative ethnography of the Khoisan peoples
     </i>
     .
     <b>
      85
     </b>
     , (Cambridge University Press: Cambridge, 1992).
    </li>
    <li>
     25. Nurse, G. T., Lane, A. &amp; Jenkins, T. Sero-genetic studies on the Dama of South West Africa.
     <i>
      Annals of Human Biology
     </i>
     <b>
      3
     </b>
     , 33–50 (1976).
    </li>
    <li>
     26. Reich, D., Thangaraj, K., Patterson, N., Price, A. L. &amp; Singh, L. Reconstructing Indian population history.
     <i>
      Nature
     </i>
     <b>
      461
     </b>
     , 489–94 (2009).
    </li>
    <li>
     27. Moorjani, P.
     <i>
      et al.
     </i>
     The history of African gene flow into Southern Europeans, Levantines, and Jews.
     <i>
      PLoS Genet
     </i>
     <b>
      7
     </b>
     , e1001373 (2011).
    </li>
    <li>
     28. Machado, C. A., Kliman, R. M., Markert, J. A. &amp; Hey, J. Inferring the history of speciation from multilocus DNA sequence data: the case of
     <i>
      Drosophila pseudoobscura
     </i>
     and close relatives.
     <i>
      Mol. Biol. Evol.
     </i>
     <b>
      19
     </b>
     , 472–488 (2002).
    </li>
    <li>
     29. Gravel, S. Population genetics models of local ancestry.
     <i>
      Genetics
     </i>
     <b>
      191
     </b>
     , 607–619 (2012).
    </li>
    <li>
     30. Price, A. L.
     <i>
      et al.
     </i>
     Sensitive detection of chromosomal segments of distinct ancestry in admixed populations.
     <i>
      PLoS Genet
     </i>
     <b>
      5
     </b>
     , e1000519 (2009).
    </li>
    <li>
     31. Pool, J. E. &amp; Nielsen, R. Inference of historical changes in migration rate from the lengths of migrant tracts.
     <i>
      Genetics
     </i>
     <b>
      181
     </b>
     , 711–9 (2009).
    </li>
    <li>
     32. Phillipson, D. W.
     <i>
      African archaeology
     </i>
     . (Cambridge University Press: Cambridge, 2005).
    </li>
    <li>
     33. Kinahan, J. From the Beginning: The Archaeological Evidence.
     <i>
      A History of Namibia. From the Beginning to 1990
     </i>
     15–43 (2011).
    </li>
    <li>
     34. Segobye, A. Early Farming Communities.
     <i>
      Ditswa Mmung: The Archaeology of Botswana
     </i>
     101–114 (1998).
    </li>
    <li>
     35. Reid, A., Sadr, K. &amp; Hanson-James, N. Herding Traditions.
     <i>
      Ditswa Mmung: The Archaeology of Botswana
     </i>
     81–100 (1998).
    </li>
    <li>
     36. Pickrell, J. K. &amp; Pritchard, J. K. Inference of population splits and mixtures from genome-wide allele frequency data.
     <i>
      ArXiv e-prints
     </i>
     (2012).at &lt;
     <a href="http://arxiv.org/abs/1206.2332">http://arxiv.org/abs/1206.2332</a>
     &gt;
    </li>
    <li>
     37. Ramachandran, S.
     <i>
      et al.
     </i>
     Support from the relationship of genetic and geographic distance in human populations for a serial founder effect originating in Africa.
     <i>
      Proceedings of the National Academy of Sciences of the United States of America
     </i>
     <b>
      102
     </b>
     , 15942 (2005).
    </li>
    <li>
     38. Gonder, M. K., Mortensen, H. M., Reed, F. A., De Sousa, A. &amp; Tishkoff, S. A. Whole-mtDNA genome sequence analysis of ancient African lineages.
     <i>
      Molecular biology and evolution
     </i>
     <b>
      24
     </b>
     , 757–768 (2007).
    </li>
   </ol>
  </div>
  <div class="page" data-page-id="13">
   <img alt="Map of Botswana showing the distribution of various ethnic groups. Surrounding countries Angola, Namibia, and Zambia are also labeled. Groups include Himba, Wambo, Dinka, Yoruba, BantuSouthAfrica, Mbukushu, Mandenka, Tswana, and Kgalagadi in the Non-Khoisan category; Naro, Hailom, Khwe, Shua, Tshwa, Gilana, Glui, Nama, and Damara in the Khoe-Kwadi category; and Jul'hoan groups in the Kx'a category; and Taa groups in the Tuu category. PCA plot showing PC1 (x-axis, -0.05 to 0.10) and PC2 (y-axis, -0.15 to 0.15). The plot shows the distribution of various ethnic groups. The Jul'hoan groups (Kx'a) are clustered at the top right, while the Taa groups (Tuu) are clustered at the bottom right. The Khoe-Kwadi groups are clustered in the center, and the Non-Khoisan groups are clustered at the top left." src="7c1f9e78e0f033d391b687f1652f6e47_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     <b>
      A. Map
     </b>
    </p>
    <p>
     <b>
      Non-Khoisan
     </b>
    </p>
    <ul style="list-style-type: none">
     <li>
      □ Himba
     </li>
     <li>
      ○ Wambo
     </li>
     <li>
      △ Dinka
     </li>
     <li>
      + Yoruba
     </li>
     <li>
      × BantuSouthAfrica
     </li>
     <li>
      ◇ BantuKenya
     </li>
     <li>
      ▽ Mbukushu
     </li>
     <li>
      ■ Mandenka
     </li>
     <li>
      * Tswana
     </li>
     <li>
      ♦ Kgalagadi
     </li>
    </ul>
    <p>
     <b>
      Khoe-Kwadi
     </b>
    </p>
    <ul style="list-style-type: none">
     <li>
      □ Naro
     </li>
     <li>
      ○ Hailom
     </li>
     <li>
      △ Khwe
     </li>
     <li>
      + Shua
     </li>
     <li>
      × Tshwa
     </li>
     <li>
      ◇ Gilana
     </li>
     <li>
      ▽ Glui
     </li>
     <li>
      ■ Nama
     </li>
     <li>
      ♦ Damara
     </li>
    </ul>
    <p>
     <b>
      Kx'a
     </b>
    </p>
    <ul style="list-style-type: none">
     <li>
      □ Jul'hoan_North
     </li>
     <li>
      ○ Jul'hoan_South
     </li>
     <li>
      △ Ixuun
     </li>
     <li>
      + #Hoan
     </li>
    </ul>
    <p>
     <b>
      Tuu
     </b>
    </p>
    <ul style="list-style-type: none">
     <li>
      □ Taa_West
     </li>
     <li>
      ○ Taa_North
     </li>
     <li>
      △ Taa_East
     </li>
    </ul>
    <p>
     <b>
      B. PCA (Jul'hoan ascertainment)
     </b>
    </p>
    <div class="img-alt">
     Map of Botswana showing the distribution of various ethnic groups. Surrounding countries Angola, Namibia, and Zambia are also labeled. Groups include Himba, Wambo, Dinka, Yoruba, BantuSouthAfrica, Mbukushu, Mandenka, Tswana, and Kgalagadi in the Non-Khoisan category; Naro, Hailom, Khwe, Shua, Tshwa, Gilana, Glui, Nama, and Damara in the Khoe-Kwadi category; and Jul'hoan groups in the Kx'a category; and Taa groups in the Tuu category. PCA plot showing PC1 (x-axis, -0.05 to 0.10) and PC2 (y-axis, -0.15 to 0.15). The plot shows the distribution of various ethnic groups. The Jul'hoan groups (Kx'a) are clustered at the top right, while the Taa groups (Tuu) are clustered at the bottom right. The Khoe-Kwadi groups are clustered in the center, and the Non-Khoisan groups are clustered at the top left.
    </div>
   </div>
   <p>
    <b>
     Figure 1
    </b>
   </p>
  </div>
  <div class="page" data-page-id="14">
   <img alt="Scatter plot of Regression coefficient vs Distance (Morgans) for Jul'hoan. The y-axis ranges from 0.00 to 0.06, and the x-axis ranges from 0.00 to 0.20. Data points are marked with 'x'. Two red lines are fitted: one labeled 'Bantu -&gt; Jul'hoan_North' which starts at a high regression coefficient and decreases, and another labeled 'Jul'hoan_North -&gt; Bantu' which starts at a low regression coefficient and remains relatively flat. Scatter plot of Proportion non-Khoisan ancestry vs various populations. The x-axis ranges from 0.0 to 1.0. Populations are plotted along a diagonal line from (0,0) to (1,1). Populations include Jul'hoan_North (fixed at 6%), Jul'hoan_South, Naro, Taa_North, Taa_West, Taa_East, #Hoan, iXuun, Gilana, Hallom, Tshwa, Kgalagadi, Shua, Khwe, Tswana, Mbukushu, Damara, Himba, and Wambo. Scatter plot of Mixture time (generations in past) vs various populations. The x-axis ranges from 0 to 50. Populations are plotted along a diagonal line from (0,0) to (50,50). Populations include Taa_West, Taa_East, #Hoan, Gilana, Mbukushu, iXuun, Tshwa, Shua, Kgalagadi, Tswana, Damara, Khwe, Hallom, Taa_North, Jul'hoan_South, Jul'hoan_North, Naro, and Himba." src="71ab4df17511d75261da8d462d643b1a_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     <b>
      A. Admixture LD in Jul'hoan
     </b>
    </p>
    <p>
     Regression coefficient
    </p>
    <p>
     Distance (Morgans)
    </p>
    <p>
     <b>
      B. Admixture proportions
     </b>
    </p>
    <p>
     Proportion non-Khoisan ancestry
    </p>
    <p>
     <b>
      C. Mixture dating
     </b>
    </p>
    <p>
     Mixture time (generations in past)
    </p>
    <div class="img-alt">
     Scatter plot of Regression coefficient vs Distance (Morgans) for Jul'hoan. The y-axis ranges from 0.00 to 0.06, and the x-axis ranges from 0.00 to 0.20. Data points are marked with 'x'. Two red lines are fitted: one labeled 'Bantu -&gt; Jul'hoan_North' which starts at a high regression coefficient and decreases, and another labeled 'Jul'hoan_North -&gt; Bantu' which starts at a low regression coefficient and remains relatively flat. Scatter plot of Proportion non-Khoisan ancestry vs various populations. The x-axis ranges from 0.0 to 1.0. Populations are plotted along a diagonal line from (0,0) to (1,1). Populations include Jul'hoan_North (fixed at 6%), Jul'hoan_South, Naro, Taa_North, Taa_West, Taa_East, #Hoan, iXuun, Gilana, Hallom, Tshwa, Kgalagadi, Shua, Khwe, Tswana, Mbukushu, Damara, Himba, and Wambo. Scatter plot of Mixture time (generations in past) vs various populations. The x-axis ranges from 0 to 50. Populations are plotted along a diagonal line from (0,0) to (50,50). Populations include Taa_West, Taa_East, #Hoan, Gilana, Mbukushu, iXuun, Tshwa, Shua, Kgalagadi, Tswana, Damara, Khwe, Hallom, Taa_North, Jul'hoan_South, Jul'hoan_North, Naro, and Himba.
    </div>
   </div>
   <p>
    <b>
     Figure 2
    </b>
   </p>
  </div>
  <div class="page" data-page-id="15">
   <img alt="Phylogenetic tree showing genetic relationships between various human populations and chimpanzees, with ancestry proportions for non-Khoisan and Khoisan ancestry." src="ad29805cd4f64ad2828e14feb66de664_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     This phylogenetic tree illustrates the genetic drift and ancestry proportions of various human populations relative to a chimpanzee outgroup. The x-axis measures the drift parameter from 0.04 to 0.12. The tree branches from the Chimp outgroup towards West African (Yoruba), East African (Dinka, Hadza, Sandawe), and non-African (French, Basque, Han, Oroqen) populations, as well as several southern African Khoisan-speaking groups.
    </p>
    <p>
     Legend:
    </p>
    <ul style="list-style-type: none">
     <li>
      <span style="color: #d65f5f;">
       ■
      </span>
      non-Khoisan ancestry
     </li>
     <li>
      <span style="color: #5f9ea0;">
       ■
      </span>
      Khoisan ancestry
     </li>
    </ul>
    <table border="1">
     <caption>
      Approximate Ancestry Proportions and Drift Parameters
     </caption>
     <thead>
      <tr>
       <th>
        Population
       </th>
       <th>
        Non-Khoisan Ancestry (Red Bar)
       </th>
       <th>
        Khoisan Ancestry (Teal Bar)
       </th>
      </tr>
     </thead>
     <tbody>
      <tr>
       <td>
        Hadza
       </td>
       <td>
        0.092 - 0.102
       </td>
       <td>
        0.102 - 0.103
       </td>
      </tr>
      <tr>
       <td>
        Sandawe
       </td>
       <td>
        0.082 - 0.089
       </td>
       <td>
        0.089 - 0.090
       </td>
      </tr>
      <tr>
       <td>
        ≠Hoan
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.094
       </td>
      </tr>
      <tr>
       <td>
        Taa_East
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.094
       </td>
      </tr>
      <tr>
       <td>
        Shua
       </td>
       <td>
        0.082 - 0.090
       </td>
       <td>
        0.090 - 0.096
       </td>
      </tr>
      <tr>
       <td>
        G|ana
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.090
       </td>
      </tr>
      <tr>
       <td>
        Tshwa
       </td>
       <td>
        0.084 - 0.088
       </td>
       <td>
        0.088 - 0.092
       </td>
      </tr>
      <tr>
       <td>
        Taa_North
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.095
       </td>
      </tr>
      <tr>
       <td>
        Taa_West
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.093
       </td>
      </tr>
      <tr>
       <td>
        Naro
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.091
       </td>
      </tr>
      <tr>
       <td>
        Khwe
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.090
       </td>
      </tr>
      <tr>
       <td>
        Ju|'hoan_North
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.100
       </td>
      </tr>
      <tr>
       <td>
        Ju|'hoan_South
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.100
       </td>
      </tr>
      <tr>
       <td>
        !Xuun
       </td>
       <td>
        0.082 - 0.085
       </td>
       <td>
        0.085 - 0.090
       </td>
      </tr>
      <tr>
       <td>
        Hailom
       </td>
       <td>
        0.084 - 0.088
       </td>
       <td>
        0.088 - 0.092
       </td>
      </tr>
     </tbody>
    </table>
    <p>
     Drift parameter values for other nodes: Chimp (~0.04), Yoruba (~0.075), Dinka (~0.095), French/Basque/Han/Oroqen (~0.105).
    </p>
    <div class="img-alt">
     Phylogenetic tree showing genetic relationships between various human populations and chimpanzees, with ancestry proportions for non-Khoisan and Khoisan ancestry.
    </div>
   </div>
   <p>
    <b>
     Figure 3
    </b>
   </p>
  </div>
  <div class="page" data-page-id="16">
   <h2>
    Supplementary Information for: The genetic prehistory of southern Africa
   </h2>
   <p>
    Joseph K. Pickrell
    <sup>
     1,*
    </sup>
    , Nick Patterson
    <sup>
     2
    </sup>
    , Chiara Barbieri
    <sup>
     3,14
    </sup>
    , Falko Berthold
    <sup>
     3,15
    </sup>
    , Linda Gerlach
    <sup>
     3,15
    </sup>
    , Tom Güldemann
    <sup>
     4
    </sup>
    , Blesswell Kure
    <sup>
     5
    </sup>
    , Sununguko Wata Mpoloka
    <sup>
     6
    </sup>
    , Hiroshi Nakagawa
    <sup>
     7
    </sup>
    , Christfried Naumann
    <sup>
     4
    </sup>
    , Mark Lipson
    <sup>
     8
    </sup>
    , Po-Ru Loh
    <sup>
     8
    </sup>
    , Joseph Lachance
    <sup>
     9
    </sup>
    , Joanna L. Mountain
    <sup>
     10
    </sup>
    , Carlos D. Bustamante
    <sup>
     11
    </sup>
    , Bonnie Berger
    <sup>
     8
    </sup>
    , Sarah Tishkoff
    <sup>
     9
    </sup>
    , Brenna M. Henn
    <sup>
     11
    </sup>
    , Mark Stoneking
    <sup>
     12
    </sup>
    , David Reich
    <sup>
     1,2,*
    </sup>
    , Brigitte Pakendorf
    <sup>
     3,13,*
    </sup>
   </p>
   <p>
    <sup>
     1
    </sup>
    Department of Genetics, Harvard Medical School, Boston
   </p>
   <p>
    <sup>
     2
    </sup>
    Broad Institute of MIT and Harvard, Cambridge
   </p>
   <p>
    <sup>
     3
    </sup>
    Max Planck Research Group on Comparative Population Linguistics,
    <br/>
    MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     4
    </sup>
    Seminar für Afrikawissenschaften, Humboldt University, Berlin and
    <br/>
    Department of Linguistics, MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     5
    </sup>
    Department of Aesthetics and Communication, Aarhus University, Aarhus
   </p>
   <p>
    <sup>
     6
    </sup>
    Department of Biological Sciences, University of Botswana
   </p>
   <p>
    <sup>
     7
    </sup>
    Institute of Global Studies, Tokyo University of Foreign Studies, Tokyo
   </p>
   <p>
    <sup>
     8
    </sup>
    Department of Mathematics and Computer Science and Artificial Intelligence Laboratory, MIT, Cambridge
   </p>
   <p>
    <sup>
     9
    </sup>
    Departments of Biology and Genetics, University of Pennsylvania, Philadelphia
   </p>
   <p>
    <sup>
     10
    </sup>
    23andMe, Inc., Mountain View
   </p>
   <p>
    <sup>
     11
    </sup>
    Department of Genetics, Stanford University, Palo Alto
   </p>
   <p>
    <sup>
     12
    </sup>
    Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     13
    </sup>
    Current affiliation: Laboratoire Dynamique du Langage, UMR5596, CNRS and Université Lyon Lumière 2
   </p>
   <p>
    <sup>
     14
    </sup>
    Current affiliation: Department of Evolutionary Genetics, MPI for Evolutionary Anthropology, Leipzig
   </p>
   <p>
    <sup>
     15
    </sup>
    Current affiliation: Department of Linguistics, MPI for Evolutionary Anthropology, Leipzig
    <br/>
    and Seminar für Afrikawissenschaften, Humboldt University, Berlin
   </p>
   <p>
    * To whom correspondence should be addressed: Brigitte.Pakendorf@ish-lyon.cnrs.fr (BP)
    <br/>
    reich@genetics.med.harvard.edu (DR), joseph_pickrell@hms.harvard.edu (JP)
   </p>
   <p>
    November 27, 2024
   </p>
  </div>
  <div class="page" data-page-id="17">
   <h2>
    Contents
   </h2>
   <table>
    <tr>
     <td>
      1
     </td>
     <td>
      Supplementary Figures
     </td>
     <td>
      <a href="#block-18-0">2</a>
     </td>
    </tr>
    <tr>
     <td>
      2
     </td>
     <td>
      Supplementary Tables
     </td>
     <td>
      <a href="#block-41-0">25</a>
     </td>
    </tr>
    <tr>
     <td>
      3
     </td>
     <td>
      Supplementary Methods
     </td>
     <td>
      <a href="#block-45-0">29</a>
     </td>
    </tr>
    <tr>
     <td>
      3.1
     </td>
     <td>
      Data
     </td>
     <td>
      <a href="#block-45-0">29</a>
     </td>
    </tr>
    <tr>
     <td>
      3.1.1
     </td>
     <td>
      Sampling
     </td>
     <td>
      <a href="#block-45-0">29</a>
     </td>
    </tr>
    <tr>
     <td>
      3.1.2
     </td>
     <td>
      Genotyping
     </td>
     <td>
      <a href="#block-45-5">29</a>
     </td>
    </tr>
    <tr>
     <td>
      3.1.3
     </td>
     <td>
      Merging data from Lachance et al. [20]
     </td>
     <td>
      <a href="#block-46-0">30</a>
     </td>
    </tr>
    <tr>
     <td>
      3.1.4
     </td>
     <td>
      Filtering “outlier” individuals
     </td>
     <td>
      <a href="#block-46-3">30</a>
     </td>
    </tr>
    <tr>
     <td>
      3.2
     </td>
     <td>
      Clustering analyses
     </td>
     <td>
      <a href="#block-46-4">30</a>
     </td>
    </tr>
    <tr>
     <td>
      3.2.1
     </td>
     <td>
      PCA
     </td>
     <td>
      <a href="#block-47-0">31</a>
     </td>
    </tr>
    <tr>
     <td>
      3.2.2
     </td>
     <td>
      ADMIXTURE analyses
     </td>
     <td>
      <a href="#block-47-5">31</a>
     </td>
    </tr>
    <tr>
     <td>
      3.2.3
     </td>
     <td>
      European ancestry in the Nama
     </td>
     <td>
      <a href="#block-48-1">32</a>
     </td>
    </tr>
    <tr>
     <td>
      3.3
     </td>
     <td>
      Three- and four-population tests
     </td>
     <td>
      <a href="#block-48-3">32</a>
     </td>
    </tr>
    <tr>
     <td>
      3.4
     </td>
     <td>
      Using the decay of linkage disequilibrium to test for historical admixture
     </td>
     <td>
      <a href="#block-49-2">33</a>
     </td>
    </tr>
    <tr>
     <td>
      3.4.1
     </td>
     <td>
      Motivation
     </td>
     <td>
      <a href="#block-49-2">33</a>
     </td>
    </tr>
    <tr>
     <td>
      3.4.2
     </td>
     <td>
      Methods
     </td>
     <td>
      <a href="#block-49-6">33</a>
     </td>
    </tr>
    <tr>
     <td>
      3.4.3
     </td>
     <td>
      Simulations
     </td>
     <td>
      <a href="#block-50-5">34</a>
     </td>
    </tr>
    <tr>
     <td>
      3.4.4
     </td>
     <td>
      Application to the Khoisan
     </td>
     <td>
      <a href="#block-51-1">35</a>
     </td>
    </tr>
    <tr>
     <td>
      3.4.5
     </td>
     <td>
      The
      <math>
       f_4
      </math>
      ratio test in the presence of admixed ancestral populations
     </td>
     <td>
      <a href="#block-51-4">35</a>
     </td>
    </tr>
    <tr>
     <td>
      3.5
     </td>
     <td>
      Estimating mixture dates with ROLLOFF
     </td>
     <td>
      <a href="#block-52-2">36</a>
     </td>
    </tr>
    <tr>
     <td>
      3.6
     </td>
     <td>
      Estimating population split times
     </td>
     <td>
      <a href="#block-52-6">36</a>
     </td>
    </tr>
    <tr>
     <td>
      3.6.1
     </td>
     <td>
      Motivation
     </td>
     <td>
      <a href="#block-52-6">36</a>
     </td>
    </tr>
    <tr>
     <td>
      3.6.2
     </td>
     <td>
      Methods
     </td>
     <td>
      <a href="#block-53-2">37</a>
     </td>
    </tr>
    <tr>
     <td>
      3.6.3
     </td>
     <td>
      Estimation of
      <math>
       \tau
      </math>
     </td>
     <td>
      <a href="#block-54-4">38</a>
     </td>
    </tr>
    <tr>
     <td>
      3.6.4
     </td>
     <td>
      Calibration
     </td>
     <td>
      <a href="#block-55-6">39</a>
     </td>
    </tr>
    <tr>
     <td>
      3.6.5
     </td>
     <td>
      Simulations
     </td>
     <td>
      <a href="#block-56-0">40</a>
     </td>
    </tr>
    <tr>
     <td>
      3.6.6
     </td>
     <td>
      Application to the Khoisan
     </td>
     <td>
      <a href="#block-56-4">40</a>
     </td>
    </tr>
    <tr>
     <td>
      3.7
     </td>
     <td>
      <i>
       TreeMix
      </i>
      analyses
     </td>
     <td>
      <a href="#block-57-2">41</a>
     </td>
    </tr>
    <tr>
     <td>
      3.7.1
     </td>
     <td>
      Analysis of the Hadza
     </td>
     <td>
      <a href="#block-57-2">41</a>
     </td>
    </tr>
    <tr>
     <td>
      3.7.2
     </td>
     <td>
      Analysis of the Sandawe
     </td>
     <td>
      <a href="#block-58-2">42</a>
     </td>
    </tr>
    <tr>
     <td>
      3.7.3
     </td>
     <td>
      West Eurasian ancestry in the Sandawe and Hadza.
     </td>
     <td>
      <a href="#block-58-2">42</a>
     </td>
    </tr>
    <tr>
     <td>
      3.7.4
     </td>
     <td>
      Modification of
      <i>
       TreeMix
      </i>
      to include known admixture
     </td>
     <td>
      <a href="#block-59-0">43</a>
     </td>
    </tr>
    <tr>
     <td>
      3.7.5
     </td>
     <td>
      Analysis of the Mbuti and Biaka
     </td>
     <td>
      <a href="#block-59-4">43</a>
     </td>
    </tr>
   </table>
  </div>
  <div class="page" data-page-id="18">
   <h1>
    1 Supplementary Figures
   </h1>
   <img alt="Phylogenetic tree of African languages showing relationships between the KXa, Tuu, Khoe-Kwadi, and Niger-Congo families." src="7f25db95ce3916c0e09803b861a2f7bc_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     The diagram illustrates the relationships between various African language families and populations. The main branches are:
    </p>
    <ul style="list-style-type: none">
     <li>
      <b>
       KXa FAMILY
      </b>
      (formerly known as Ju-ǂHoan (Güldemann 2008))
      <ul style="list-style-type: none">
       <li>
        ǂHoan
       </li>
       <li>
        Ju (formerly known as Northern Khoisan (Greenberg 1963))
        <ul style="list-style-type: none">
         <li>
          Northwest
         </li>
         <li>
          South
         </li>
         <li>
          Southwest
         </li>
         <li>
          !Xuun
         </li>
         <li>
          Ju!ʼhoan North, Ju!ʼhoan South
         </li>
        </ul>
       </li>
      </ul>
     </li>
     <li>
      <b>
       TUU FAMILY
      </b>
      (formerly known as Southern Khoisan)
      <ul style="list-style-type: none">
       <li>
        !Ui
       </li>
       <li>
        Taa-Lower Nossob
       </li>
       <li>
        Lower Nossob
       </li>
       <li>
        Taa
       </li>
       <li>
        Taa, West, Taa, North, Taa, East
       </li>
      </ul>
     </li>
     <li>
      <b>
       KHOE-KWADI FAMILY
      </b>
      <ul style="list-style-type: none">
       <li>
        Kwadi
       </li>
       <li>
        Khoe (formerly known as Central Khoisan (Greenberg 1963))
        <ul style="list-style-type: none">
         <li>
          Khoekhoe
          <ul style="list-style-type: none">
           <li>
            North
           </li>
           <li>
            South
           </li>
          </ul>
         </li>
         <li>
          Kalahari
          <ul style="list-style-type: none">
           <li>
            East
            <ul style="list-style-type: none">
             <li>
              Shua
             </li>
             <li>
              Tshwa
             </li>
            </ul>
           </li>
           <li>
            West
            <ul style="list-style-type: none">
             <li>
              Khoe
             </li>
             <li>
              Gijana
             </li>
             <li>
              Naro
             </li>
            </ul>
           </li>
           <li>
            Khwe
           </li>
           <li>
            Gijana, Gijui
           </li>
           <li>
            Naro
           </li>
          </ul>
         </li>
        </ul>
       </li>
       <li>
        Name, Damara, Hai||om
       </li>
      </ul>
     </li>
     <li>
      <b>
       NIGER-CONGO PHYLUM
      </b>
      <ul style="list-style-type: none">
       <li>
        Mande
       </li>
       <li>
        Kordofanian
       </li>
       <li>
        Atlantic
       </li>
       <li>
        Volta-Congo
        <ul style="list-style-type: none">
         <li>
          Kru
          <ul style="list-style-type: none">
           <li>
            Kwa
           </li>
           <li>
            North Volta
           </li>
           <li>
            Benue-Congo
            <ul style="list-style-type: none">
             <li>
              Bantu
             </li>
             <li>
              Yoruboid
             </li>
            </ul>
           </li>
          </ul>
         </li>
         <li>
          Bantu
         </li>
         <li>
          Yoruba
         </li>
        </ul>
       </li>
      </ul>
     </li>
     <li>
      Himba, Wambo, Mbukushu, Tswana, Kgalagadi
     </li>
    </ul>
    <div class="img-alt">
     Phylogenetic tree of African languages showing relationships between the KXa, Tuu, Khoe-Kwadi, and Niger-Congo families.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S1: Relationships between African languages spoken by populations in this study.
    </b>
    In bold are populations included in this study; the Hadza and Sandawe are not shown because they are linguistic isolates.
   </p>
  </div>
  <div class="page" data-page-id="19">
   <img alt="PCA plot showing the positions of individuals from various populations along principal component axes PC1 and PC2. The plot includes African populations (Hadza, Biaka, Mbuti, Himba, Wambo, Dinka, Yoruba, BantuSouthAfrica, BantuKenya, Mbukushu, Mandenka, Tswana, Kgalagadi, Naro, Damara, Haillom, Shua, Tshwa) and non-African populations (Nama, Gillana, Glui, Jul'hoan_North, Jul'hoan_South, !Xuun, #Hoan, Taa_West, Taa_North, Taa_East, French, Han, Oroqen, Basque). The African populations are clustered at the top left, while non-African populations are clustered at the bottom right." src="3102c32204f998dba666e1e915d5babf_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     PC2
    </p>
    <p>
     PC1
    </p>
    <table border="1">
     <tr>
      <td>
       ● Hadza
      </td>
      <td>
       ■ Nama
      </td>
     </tr>
     <tr>
      <td>
       ■ Biaka
      </td>
      <td>
       ◇ Gillana
      </td>
     </tr>
     <tr>
      <td>
       ■ Mbuti
      </td>
      <td>
       ▼ Glui
      </td>
     </tr>
     <tr>
      <td>
       □ Himba
      </td>
      <td>
       ■ Jul'hoan_North
      </td>
     </tr>
     <tr>
      <td>
       ○ Wambo
      </td>
      <td>
       ○ Jul'hoan_South
      </td>
     </tr>
     <tr>
      <td>
       ▲ Dinka
      </td>
      <td>
       ▲ !Xuun
      </td>
     </tr>
     <tr>
      <td>
       ■ Yoruba
      </td>
      <td>
       ▲ #Hoan
      </td>
     </tr>
     <tr>
      <td>
       × BantuSouthAfrica
      </td>
      <td>
       ■ Taa_West
      </td>
     </tr>
     <tr>
      <td>
       ◇ BantuKenya
      </td>
      <td>
       ○ Taa_North
      </td>
     </tr>
     <tr>
      <td>
       ▼ Mbukushu
      </td>
      <td>
       ▲ Taa_East
      </td>
     </tr>
     <tr>
      <td>
       ■ Mandenka
      </td>
      <td>
       □ French
      </td>
     </tr>
     <tr>
      <td>
       ■ Tswana
      </td>
      <td>
       ○ Han
      </td>
     </tr>
     <tr>
      <td>
       ◆ Kgalagadi
      </td>
      <td>
       ▲ Oroqen
      </td>
     </tr>
     <tr>
      <td>
       □ Naro
      </td>
      <td>
       × Basque
      </td>
     </tr>
     <tr>
      <td>
       ■ Damara
      </td>
      <td>
      </td>
     </tr>
     <tr>
      <td>
       ○ Haillom
      </td>
      <td>
      </td>
     </tr>
     <tr>
      <td>
       × Shua
      </td>
      <td>
      </td>
     </tr>
     <tr>
      <td>
       × Tshwa
      </td>
      <td>
      </td>
     </tr>
    </table>
    <div class="img-alt">
     PCA plot showing the positions of individuals from various populations along principal component axes PC1 and PC2. The plot includes African populations (Hadza, Biaka, Mbuti, Himba, Wambo, Dinka, Yoruba, BantuSouthAfrica, BantuKenya, Mbukushu, Mandenka, Tswana, Kgalagadi, Naro, Damara, Haillom, Shua, Tshwa) and non-African populations (Nama, Gillana, Glui, Jul'hoan_North, Jul'hoan_South, !Xuun, #Hoan, Taa_West, Taa_North, Taa_East, French, Han, Oroqen, Basque). The African populations are clustered at the top left, while non-African populations are clustered at the bottom right.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S2: PCA including non-African populations.
    </b>
    We performed principal component analysis on the genotype matrix of individuals using smartpca [22] using the SNPs ascertained in a Jul'hoan_North individual. Plotted are the positions of each individual along principal component axes one and two. The colors and symbols for each population are depicted in the legend.
   </p>
  </div>
  <div class="page" data-page-id="20">
   <img alt="PCA plot of African populations showing PC1 vs PC2. The plot displays various populations clustered along the axes, with Himba and Wambo at the top left, and Taa-related populations at the top right. A legend on the left lists populations from Himba to Hallom, and a legend on the right lists populations from Khwe to Taa_East." src="5ed9189841659dfb01f809b8e3b21f74_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     PC2
    </p>
    <p>
     PC1
    </p>
    <table border="1">
     <tr>
      <td>
       ◻ Himba
      </td>
      <td>
       ▲ Khwe
      </td>
     </tr>
     <tr>
      <td>
       ○ Wambo
      </td>
      <td>
       + Shua
      </td>
     </tr>
     <tr>
      <td>
       ▲ Dinka
      </td>
      <td>
       × Tshwa
      </td>
     </tr>
     <tr>
      <td>
       + Yoruba
      </td>
      <td>
       ◼ Nama
      </td>
     </tr>
     <tr>
      <td>
       × BantuSouthAfrica
      </td>
      <td>
       ◇ Gliana
      </td>
     </tr>
     <tr>
      <td>
       ◇ BantuKenya
      </td>
      <td>
       ▽ Glui
      </td>
     </tr>
     <tr>
      <td>
       ▽ Mbukushu
      </td>
      <td>
       ◻ Jul'hoan_North
      </td>
     </tr>
     <tr>
      <td>
       ◼ Mandenka
      </td>
      <td>
       ○ Jul'hoan_South
      </td>
     </tr>
     <tr>
      <td>
       * Tswana
      </td>
      <td>
       ▲ IXuun
      </td>
     </tr>
     <tr>
      <td>
       ◆ Kgalagadi
      </td>
      <td>
       + #Hoan
      </td>
     </tr>
     <tr>
      <td>
       ◻ Naro
      </td>
      <td>
       ◻ Taa_West
      </td>
     </tr>
     <tr>
      <td>
       * Damara
      </td>
      <td>
       ◻ Taa_North
      </td>
     </tr>
     <tr>
      <td>
       ○ Hallom
      </td>
      <td>
       ▲ Taa_East
      </td>
     </tr>
    </table>
    <div class="img-alt">
     PCA plot of African populations showing PC1 vs PC2. The plot displays various populations clustered along the axes, with Himba and Wambo at the top left, and Taa-related populations at the top right. A legend on the left lists populations from Himba to Hallom, and a legend on the right lists populations from Khwe to Taa_East.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S3: PCA of African populations using all the SNPs on the chip.
    </b>
    Each individual is represented by a point, and the color and style of the point is displayed in the caption.
   </p>
  </div>
  <div class="page" data-page-id="21">
   <img alt="Supplementary Figure S4: PCA on SNPs from different ascertainment panels. The figure contains three panels (A, B, C) showing PCA plots of genetic data. Panel A is titled 'A. PCA (Jul'hoan ascertainment)' and shows PC1 vs PC2 with a wide range of PC2 values from -0.15 to 0.15. Panel B is titled 'B. PCA (Yoruba ascertainment)' and shows PC1 vs PC2 with PC2 values from -0.2 to 0.1. Panel C is titled 'C. PCA (French ascertainment)' and shows PC1 vs PC2 with PC2 values from 0.0 to 0.4. All panels share a common legend for population symbols and colors." src="60e9207be66a64332619bb4b667fe67b_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     <b>
      Legend:
     </b>
    </p>
    <ul style="list-style-type: none">
     <li>
      □ Himba
     </li>
     <li>
      ○ Wambo
     </li>
     <li>
      △ Dinka
     </li>
     <li>
      + Yoruba
     </li>
     <li>
      × BantuSouthAfrica
     </li>
     <li>
      ◇ BantuKenya
     </li>
     <li>
      ▽ Mbukushu
     </li>
     <li>
      ■ Mandenka
     </li>
     <li>
      * Tswana
     </li>
     <li>
      ◆ Kgalagadi
     </li>
     <li>
      □ Naro
     </li>
     <li>
      * Damara
     </li>
     <li>
      ○ Haillom
     </li>
     <li>
      △ Khwe
     </li>
     <li>
      + Shua
     </li>
     <li>
      × Tshwa
     </li>
     <li>
      ■ Nama
     </li>
     <li>
      ◇ Gilana
     </li>
     <li>
      ▽ Gli
     </li>
     <li>
      □ Jul'hoan_North
     </li>
     <li>
      ○ Jul'hoan_South
     </li>
     <li>
      △ !Xuun
     </li>
     <li>
      + #Hoan
     </li>
     <li>
      □ Taa_West
     </li>
     <li>
      ○ Taa_North
     </li>
     <li>
      △ Taa_East
     </li>
    </ul>
    <div class="img-alt">
     Supplementary Figure S4: PCA on SNPs from different ascertainment panels. The figure contains three panels (A, B, C) showing PCA plots of genetic data. Panel A is titled 'A. PCA (Jul'hoan ascertainment)' and shows PC1 vs PC2 with a wide range of PC2 values from -0.15 to 0.15. Panel B is titled 'B. PCA (Yoruba ascertainment)' and shows PC1 vs PC2 with PC2 values from -0.2 to 0.1. Panel C is titled 'C. PCA (French ascertainment)' and shows PC1 vs PC2 with PC2 values from 0.0 to 0.4. All panels share a common legend for population symbols and colors.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S4: PCA on SNPs from different ascertainment panels.
    </b>
    In each panel, each point represents an individual. The color and style of each point corresponds to the population of the individual as displayed in the legend.
    <b>
     A. Ju'hoan_North ascertainment.
    </b>
    This is same data presented in Figure 1B in the main text, but is included for comparison.
    <b>
     B. Yoruba ascertainment.
    </b>
    <b>
     C. French ascertainment
    </b>
   </p>
  </div>
  <div class="page" data-page-id="22">
   <img alt="PCA plot showing PC1 vs PC2. The x-axis (PC1) ranges from -0.1 to 0.2, and the y-axis (PC2) ranges from -0.3 to 0.1. Data points are clustered along a diagonal line from top-left to bottom-right, with outliers at the top-left and bottom-left." src="0f79a59f3766fc341ff688a23692c1d9_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     PC2
    </p>
    <p>
     PC1
    </p>
    <div class="img-alt">
     PCA plot showing PC1 vs PC2. The x-axis (PC1) ranges from -0.1 to 0.2, and the y-axis (PC2) ranges from -0.3 to 0.1. Data points are clustered along a diagonal line from top-left to bottom-right, with outliers at the top-left and bottom-left.
    </div>
   </div>
   <table border="1">
    <tbody>
     <tr>
      <td>
       □
      </td>
      <td>
       Himba
      </td>
     </tr>
     <tr>
      <td>
       ○
      </td>
      <td>
       Wambo
      </td>
     </tr>
     <tr>
      <td>
       ▲
      </td>
      <td>
       Dinka
      </td>
     </tr>
     <tr>
      <td>
       +
      </td>
      <td>
       Yoruba
      </td>
     </tr>
     <tr>
      <td>
       ×
      </td>
      <td>
       BantuSouthAfrica
      </td>
     </tr>
     <tr>
      <td>
       ◇
      </td>
      <td>
       BantuKenya
      </td>
     </tr>
     <tr>
      <td>
       ▽
      </td>
      <td>
       Mbukushu
      </td>
     </tr>
     <tr>
      <td>
       ■
      </td>
      <td>
       Mandenka
      </td>
     </tr>
     <tr>
      <td>
       *
      </td>
      <td>
       Tswana
      </td>
     </tr>
     <tr>
      <td>
       ◆
      </td>
      <td>
       Kgalagadi
      </td>
     </tr>
     <tr>
      <td>
       □
      </td>
      <td>
       Naro
      </td>
     </tr>
     <tr>
      <td>
       •
      </td>
      <td>
       Damara
      </td>
     </tr>
     <tr>
      <td>
       ○
      </td>
      <td>
       Hailom
      </td>
     </tr>
    </tbody>
   </table>
   <table border="1">
    <tbody>
     <tr>
      <td>
       ▲
      </td>
      <td>
       Khwe
      </td>
     </tr>
     <tr>
      <td>
       +
      </td>
      <td>
       Shua
      </td>
     </tr>
     <tr>
      <td>
       ×
      </td>
      <td>
       Tshwa
      </td>
     </tr>
     <tr>
      <td>
       ◇
      </td>
      <td>
       Gilana
      </td>
     </tr>
     <tr>
      <td>
       □
      </td>
      <td>
       Ju|'hoan_North
      </td>
     </tr>
     <tr>
      <td>
       ○
      </td>
      <td>
       Ju|'hoan_South
      </td>
     </tr>
     <tr>
      <td>
       ▲
      </td>
      <td>
       !Xuun
      </td>
     </tr>
     <tr>
      <td>
       +
      </td>
      <td>
       #Hoan
      </td>
     </tr>
     <tr>
      <td>
       □
      </td>
      <td>
       Taa_West
      </td>
     </tr>
     <tr>
      <td>
       ○
      </td>
      <td>
       Taa_North
      </td>
     </tr>
     <tr>
      <td>
       ▲
      </td>
      <td>
       Taa_East
      </td>
     </tr>
    </tbody>
   </table>
   <p>
    <b>
     Supplementary Figure S5: PCA projection using Ju|'hoan_North, Yoruba, and Dinka.
    </b>
    We identified principal components using only the Ju|'hoan_North, Yoruba, and Dinka, then projected the other samples (excluding outliers) onto these axes. All Khoisan populations fall on a line between the Ju|'hoan_North and the neighboring Bantu-speaking populations. This is consistent with the variation in non-Khoisan admixture in these populations being due to variation in admixture with neighboring agriculturist populations.
   </p>
  </div>
  <div class="page" data-page-id="23">
   <img alt="Supplementary Figure S6: Tests of treeness and Admixture LD for the Nama and Shua populations." src="9ce50bc10864dc86e1cdee4be08f1897_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     <b>
      A. Tests of treeness
     </b>
    </p>
    <p>
     This plot shows the
     <math>
      f_4
     </math>
     statistic for various southern African populations. The x-axis is
     <math>
      f_4(\text{Yoruba}, X; \text{Han}, \text{French})
     </math>
     , ranging from -0.0005 to 0.0015. The y-axis lists populations: Shua, Nama, Haillom, Khwe, Taa_West, Mbukushu, Gllana, Tswana, Damara, !Xuun, Himba, Tshwa, !Hoan, Taa_North, Kgalagadi, Naro, Wambo, Ju!hoan_North, Taa_East, and Ju!hoan_South. Points in red have a Z-score greater than 3.
    </p>
    <p>
     <b>
      B. Admixture LD
     </b>
    </p>
    <p>
     This plot shows the correlation of linkage disequilibrium (LD) versus distance in Morgans. The x-axis is Distance (Morgans) from 0.00 to 0.30. The y-axis is Correlation from 0.00 to 0.08. Two data series are shown: Nama (black 'x' markers) and Shua (grey 'x' markers). Red lines represent fitted exponential decay curves for both populations.
    </p>
    <div class="img-alt">
     Supplementary Figure S6: Tests of treeness and Admixture LD for the Nama and Shua populations.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S6: The Nama have recent non-African ancestry.
    </b>
    <b>
     A. Four-population tests.
    </b>
    We performed four-population tests on the tree topology
    <math>
     [[\text{Yoruba}, X], [\text{Han}, \text{French}]]
    </math>
    , where
    <math>
     X
    </math>
    represents any southern African population. Plotted is the value of the
    <math>
     f_4
    </math>
    statistic when each southern African population is used. Error bars show a single standard error, and points in red have a Z-score greater than 3.
    <b>
     B. Admixture LD.
    </b>
    We ran ROLLOFF on the Nama and the Shua using the Ju!hoan_North and the French as the mixing populations. There is a clear decay in the Nama (the shift away from the x-axis is indicative of variable ancestry across individuals, which is visually apparent in Supplementary Figure S
    <a href="#block-19-0">2)</a>
    and a less obvious decay in the Shua. The red lines show the fitted exponential curves.
   </p>
  </div>
  <div class="page" data-page-id="24">
   <img alt="Six PCA plots for different populations: Ju'hoan_South, Taa_North, Shua, !Xuun, Kgalagadi, and Naro. Each plot shows PC1 (x-axis, -0.15 to 0.10) and PC2 (y-axis, -0.15 to 0.10). Black circles represent the main population. Red circles represent excluded individuals, and green circles represent kept individuals. The populations are distributed across the plots, with some showing distinct clusters of excluded or kept individuals." src="645bea0b27d63e4a9a300af5793ae7d2_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     The figure consists of six PCA plots arranged in a 2x3 grid, each representing a different population. The axes for all plots are PC1 (x-axis, ranging from -0.15 to 0.10) and PC2 (y-axis, ranging from -0.15 to 0.10). The populations are:
    </p>
    <ul style="list-style-type: none">
     <li>
      <b>
       Ju'hoan_South
      </b>
      : Shows a cluster of black circles. A few red circles are located near PC1 = 0.05, PC2 = -0.05. A small cluster of green circles is near PC1 = 0.08, PC2 = -0.12.
     </li>
     <li>
      <b>
       Taa_North
      </b>
      : Shows a cluster of black circles. A red circle is located near PC1 = 0.05, PC2 = 0.05. A cluster of green circles is near PC1 = 0.08, PC2 = 0.08.
     </li>
     <li>
      <b>
       Shua
      </b>
      : Shows a cluster of black circles. A red circle is located near PC1 = 0.05, PC2 = 0.05. A cluster of green circles is near PC1 = -0.02, PC2 = 0.02.
     </li>
     <li>
      <b>
       !Xuun
      </b>
      : Shows a cluster of black circles. A cluster of green circles is located near PC1 = 0.05, PC2 = -0.05. A red circle is located near PC1 = 0.08, PC2 = -0.12.
     </li>
     <li>
      <b>
       Kgalagadi
      </b>
      : Shows a cluster of black circles. A red circle is located near PC1 = 0.02, PC2 = 0.05. A cluster of green circles is located near PC1 = -0.02, PC2 = 0.02.
     </li>
     <li>
      <b>
       Naro
      </b>
      : Shows a cluster of black circles. A red circle is located near PC1 = 0.05, PC2 = -0.05. A cluster of green circles is located near PC1 = 0.08, PC2 = 0.08.
     </li>
    </ul>
    <div class="img-alt">
     Six PCA plots for different populations: Ju'hoan_South, Taa_North, Shua, !Xuun, Kgalagadi, and Naro. Each plot shows PC1 (x-axis, -0.15 to 0.10) and PC2 (y-axis, -0.15 to 0.10). Black circles represent the main population. Red circles represent excluded individuals, and green circles represent kept individuals. The populations are distributed across the plots, with some showing distinct clusters of excluded or kept individuals.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S7: Individuals excluded from populations.
    </b>
    Shown are the PCA plot in Figure 1 in the main text, with different populations highlighted. In red are the individuals we excluded, and in green those that were kept. See Supplementary Table
    <a href="#block-41-0">S1</a>
    for total sample sizes in each population.
   </p>
  </div>
  <div class="page" data-page-id="25">
   <img alt="Supplementary Figure S8: ADMIXTURE plots showing ancestry proportions for 30 populations across K=2 to K=6. The populations are listed on the right: BantuKenya, BantuSouthAfrica, Dinka, Himba, Kgalagadi, Mandenka, Mbukushu, Wambo, Tswana, Yoruba, YRI, Hadza, Hadza_TISH, Sandawe, Biaka, Mbuti, Pygmy_TISH, Damara, Gilana, Glui, Haillom, Khwe, Tshwa, Nama, Naro, Shua, !Xuun, !Hoan, Jul'hoan_North, Jul'hoan_South, Taa_East, Taa_North, Taa_West, Basque, French, Han, and Oroqen. The plots show how the genetic structure of these populations changes as the number of clusters (K) increases from 2 to 6. At K=2, populations are split into two broad groups: African (red/orange) and non-African (blue). As K increases, more specific subgroups within Africa emerge, such as the Pygmy groups and various Bantu and San populations. The non-African populations (Basque, French, Han, Oroqen) remain relatively homogeneous across all K values." src="97f61e67792478fb6ce089868e503063_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <div class="img-alt">
     Supplementary Figure S8: ADMIXTURE plots showing ancestry proportions for 30 populations across K=2 to K=6. The populations are listed on the right: BantuKenya, BantuSouthAfrica, Dinka, Himba, Kgalagadi, Mandenka, Mbukushu, Wambo, Tswana, Yoruba, YRI, Hadza, Hadza_TISH, Sandawe, Biaka, Mbuti, Pygmy_TISH, Damara, Gilana, Glui, Haillom, Khwe, Tshwa, Nama, Naro, Shua, !Xuun, !Hoan, Jul'hoan_North, Jul'hoan_South, Taa_East, Taa_North, Taa_West, Basque, French, Han, and Oroqen. The plots show how the genetic structure of these populations changes as the number of clusters (K) increases from 2 to 6. At K=2, populations are split into two broad groups: African (red/orange) and non-African (blue). As K increases, more specific subgroups within Africa emerge, such as the Pygmy groups and various Bantu and San populations. The non-African populations (Basque, French, Han, Oroqen) remain relatively homogeneous across all K values.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S8: Clustering analyses of the merged sequenced and genotyped samples.
    </b>
    We ran ADMIXTURE on all African individuals using different settings of
    <math>
     K
    </math>
    ; shown are the resulting clusters. The populations merged from Lachance et al. [20] are YRI (compare to Yoruba), Hadza_TISH (compare to Hadza), Pygmy_TISH, and Sandawe.
   </p>
  </div>
  <div class="page" data-page-id="26">
   <img alt="A diagram showing a simulated demography with two populations, pop1 and pop2, connected by a red arrow labeled '10%'. A heatmap showing ancestry proportions for Simulation 1. The y-axis is 'Ancestry' (0.0 to 1.0) and the x-axis is labeled 'pop1' and 'pop2'. The plot shows a clear division between the two populations, with a few individuals showing mixed ancestry. A scatter plot showing the decay of linkage disequilibrium (LD) for Simulation 1. The y-axis is 'Regression coefficient' (-0.02 to 0.10) and the x-axis is 'Distance (Morgans)' (0.00 to 0.20). Two series are shown: 'pop1-&gt;pop2' (black dots) and 'pop2-&gt;pop1' (grey dots). The black series shows a sharp initial drop in the regression coefficient, while the grey series remains relatively flat. A heatmap showing ancestry proportions for Simulation 2. The y-axis is 'Ancestry' (0.0 to 1.0) and the x-axis is labeled 'pop1' and 'pop2'. Similar to Simulation 1, it shows a clear division between the two populations with some mixed ancestry. A scatter plot showing the decay of linkage disequilibrium (LD) for Simulation 2. The y-axis is 'Regression coefficient' (0.00 to 0.12) and the x-axis is 'Distance (Morgans)' (0.00 to 0.20). Two series are shown: 'pop1-&gt;pop2' (black dots) and 'pop2-&gt;pop1' (grey dots). The black series shows a sharp initial drop in the regression coefficient, while the grey series remains relatively flat." src="bfb6d182d624680db577069bbc0b2a93_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A. Simulated demography
    </p>
    <p>
     Simulation 1
    </p>
    <p>
     B. ADMIXTURE (K=2)
    </p>
    <p>
     C. Decay of LD
    </p>
    <p>
     Simulation 2
    </p>
    <p>
     D. ADMIXTURE (K=2)
    </p>
    <p>
     E. Decay of LD
    </p>
    <div class="img-alt">
     A diagram showing a simulated demography with two populations, pop1 and pop2, connected by a red arrow labeled '10%'. A heatmap showing ancestry proportions for Simulation 1. The y-axis is 'Ancestry' (0.0 to 1.0) and the x-axis is labeled 'pop1' and 'pop2'. The plot shows a clear division between the two populations, with a few individuals showing mixed ancestry. A scatter plot showing the decay of linkage disequilibrium (LD) for Simulation 1. The y-axis is 'Regression coefficient' (-0.02 to 0.10) and the x-axis is 'Distance (Morgans)' (0.00 to 0.20). Two series are shown: 'pop1-&gt;pop2' (black dots) and 'pop2-&gt;pop1' (grey dots). The black series shows a sharp initial drop in the regression coefficient, while the grey series remains relatively flat. A heatmap showing ancestry proportions for Simulation 2. The y-axis is 'Ancestry' (0.0 to 1.0) and the x-axis is labeled 'pop1' and 'pop2'. Similar to Simulation 1, it shows a clear division between the two populations with some mixed ancestry. A scatter plot showing the decay of linkage disequilibrium (LD) for Simulation 2. The y-axis is 'Regression coefficient' (0.00 to 0.12) and the x-axis is 'Distance (Morgans)' (0.00 to 0.20). Two series are shown: 'pop1-&gt;pop2' (black dots) and 'pop2-&gt;pop1' (grey dots). The black series shows a sharp initial drop in the regression coefficient, while the grey series remains relatively flat.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S9: LD information identifies previously undetectable admixture events.
    </b>
    We performed simulations of two populations, one of which admixed with the other 40 generations in the past (see Section
    <a href="#block-49-2">3.4</a>
    for details). Shown are results from two simulations.
    <b>
     A.
    </b>
    The simulated demography.
    <b>
     B,D.
    </b>
    Results from running ADMIXTURE on the simulated data.
    <b>
     C,E.
    </b>
    Results from the measure of LD decay described in Section
    <a href="#block-49-2">3.4.</a>
   </p>
  </div>
  <div class="page" data-page-id="27">
   <img alt="Supplementary Figure S10: A line graph showing the decay of the regression coefficient with distance in Morgans for different admixture proportions. The y-axis is 'Regression coefficient' (0.00 to 0.30) and the x-axis is 'Distance (Morgans)' (0.00 to 0.20). The legend includes: 0% (sim) (grey x), 0% (theory) (grey line), 10% (sim) (red x), 10% (theory) (red line), 20% (sim) (blue x), 20% (theory) (blue line), and observed (black x). The 0% curves are near zero. The 10% and 20% curves show a decay from higher initial values, with the 20% curve starting at approximately 0.21 and the 10% curve at approximately 0.12. The observed data points closely follow the theoretical curves." src="2a23751710f7827065d2b99b6df588df_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <div class="img-alt">
     Supplementary Figure S10: A line graph showing the decay of the regression coefficient with distance in Morgans for different admixture proportions. The y-axis is 'Regression coefficient' (0.00 to 0.30) and the x-axis is 'Distance (Morgans)' (0.00 to 0.20). The legend includes: 0% (sim) (grey x), 0% (theory) (grey line), 10% (sim) (red x), 10% (theory) (red line), 20% (sim) (blue x), 20% (theory) (blue line), and observed (black x). The 0% curves are near zero. The 10% and 20% curves show a decay from higher initial values, with the 20% curve starting at approximately 0.21 and the 10% curve at approximately 0.12. The observed data points closely follow the theoretical curves.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S10: Estimating mixture proportions from LD.
    </b>
    We simulated genetic data under different demographies including admixture (Supplementary Information), and then estimated admixture proportions using the method described in the text. The colored and grey points represent the decay curve obtained in simulations (each curve is the average of five simulations of 100 Mb), and the lines are the theoretical curves. In black is the data from the Ju'hoan_North and Yoruba, treating the Ju'hoan_North as admixed.
   </p>
  </div>
  <div class="page" data-page-id="28">
   <img alt="Five panels (A-E) showing Admixture LD decay plots for different Khoisan populations. Each plot has 'Regression coefficient' on the y-axis (0.00 to 0.20) and 'Distance (Morgans)' on the x-axis (0.00 to 0.20). Data points are black 'x' marks, and a red line shows the fitted exponential decay. Panel A: Jul'hoan North. Panel B: Jul'hoan South. Panel C: ≠Hoan. Panel D: Taa North. Panel E: Taa East. Each panel includes a legend for the admixture components." src="4cec89a753c447a050c0171c274f2acb_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Figure S11 displays five panels (A-E) showing Admixture LD decay plots for different Khoisan populations. Each plot has 'Regression coefficient' on the y-axis (0.00 to 0.20) and 'Distance (Morgans)' on the x-axis (0.00 to 0.20). Data points are black 'x' marks, and a red line shows the fitted exponential decay. The panels are:
    </p>
    <ul style="list-style-type: none">
     <li>
      <b>
       A. Jul'hoan North
      </b>
      : Legend:
      <math>
       \times
      </math>
      Bantu
      <math>
       \to
      </math>
      Jul'hoan_North,
      <math>
       \times
      </math>
      Jul'hoan_North
      <math>
       \to
      </math>
      Bantu.
     </li>
     <li>
      <b>
       B. Jul'hoan South
      </b>
      : Legend:
      <math>
       \times
      </math>
      Bantu
      <math>
       \to
      </math>
      Jul'hoan_South,
      <math>
       \times
      </math>
      Jul'hoan_South
      <math>
       \to
      </math>
      Bantu.
     </li>
     <li>
      <b>
       C. ≠Hoan
      </b>
      : Legend:
      <math>
       \times
      </math>
      Bantu
      <math>
       \to
      </math>
      ≠Hoan,
      <math>
       \times
      </math>
      ≠Hoan
      <math>
       \to
      </math>
      Bantu.
     </li>
     <li>
      <b>
       D. Taa North
      </b>
      : Legend:
      <math>
       \times
      </math>
      Bantu
      <math>
       \to
      </math>
      Taa_North,
      <math>
       \times
      </math>
      Taa_North
      <math>
       \to
      </math>
      Bantu.
     </li>
     <li>
      <b>
       E. Taa East
      </b>
      : Legend:
      <math>
       \times
      </math>
      Bantu
      <math>
       \to
      </math>
      Taa_East,
      <math>
       \times
      </math>
      Taa_East
      <math>
       \to
      </math>
      Bantu.
     </li>
    </ul>
    <div class="img-alt">
     Five panels (A-E) showing Admixture LD decay plots for different Khoisan populations. Each plot has 'Regression coefficient' on the y-axis (0.00 to 0.20) and 'Distance (Morgans)' on the x-axis (0.00 to 0.20). Data points are black 'x' marks, and a red line shows the fitted exponential decay. Panel A: Jul'hoan North. Panel B: Jul'hoan South. Panel C: ≠Hoan. Panel D: Taa North. Panel E: Taa East. Each panel includes a legend for the admixture components.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S11: Admixture LD in populations that pass the three-population test.
    </b>
    We measured the decay of admixture LD on the five Khoisan populations that show no evidence of admixture in three-population tests. The method is described in Section
    <a href="#block-49-2">3.4.</a>
    Each panel shows an individual population; panel
    <b>
     A.
    </b>
    is a version of Figure 2A from the main text with the y-axis modified to be the same as the other panels. In all cases, the non-Khoisan population used in the analysis is the Yoruba. In red is the fitted exponential curve.
   </p>
  </div>
  <div class="page" data-page-id="29">
   <img alt="ROLLOFF analysis plot for Khwe, Ju'hoan_North, and Dinka." src="a2dcc4a0703102026ec86e82caa4985e_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Correlation
    </p>
    <p>
     Distance (cM)
    </p>
    <p>
     Khwe -&gt; Ju'hoan_North&lt;- Dinka
    </p>
    <p>
     This plot shows the correlation between LD decay in Ju'hoan_North and the product of LD differences between Khwe and Dinka. The y-axis is 'Correlation' (0.00 to 0.06) and the x-axis is 'Distance (cM)' (0 to 20). Data points are black 'x' marks, and a red line shows a fitted exponential decay curve starting at approximately 0.015 at 0 cM and dropping to near 0 by 10 cM.
    </p>
    <div class="img-alt">
     ROLLOFF analysis plot for Khwe, Ju'hoan_North, and Dinka.
    </div>
   </div>
   <img alt="ROLLOFF analysis plot for Yoruba, Ju'hoan_North, and French." src="19f5ea8afbf380781eb0645831a88987_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Correlation
    </p>
    <p>
     Distance (cM)
    </p>
    <p>
     Yoruba -&gt; Ju'hoan_North&lt;- French
    </p>
    <p>
     This plot shows the correlation between LD decay in Ju'hoan_North and the product of LD differences between Yoruba and French. The y-axis is 'Correlation' (0.00 to 0.06) and the x-axis is 'Distance (cM)' (0 to 20). Data points are black 'x' marks, and a red line shows a fitted exponential decay curve starting at approximately 0.015 at 0 cM and dropping to near 0 by 10 cM.
    </p>
    <div class="img-alt">
     ROLLOFF analysis plot for Yoruba, Ju'hoan_North, and French.
    </div>
   </div>
   <img alt="ROLLOFF analysis plot for Yoruba, Ju'hoan_North, and Taa_East." src="7119b28e39fa3784606bf8b8f44e4f9d_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Correlation
    </p>
    <p>
     Distance (cM)
    </p>
    <p>
     Yoruba -&gt; Ju'hoan_North&lt;- Taa_East
    </p>
    <p>
     This plot shows the correlation between LD decay in Ju'hoan_North and the product of LD differences between Yoruba and Taa_East. The y-axis is 'Correlation' (0.00 to 0.06) and the x-axis is 'Distance (cM)' (0 to 20). Data points are black 'x' marks, and a red line shows a fitted exponential decay curve starting at approximately 0.045 at 0 cM and dropping to near 0 by 10 cM.
    </p>
    <div class="img-alt">
     ROLLOFF analysis plot for Yoruba, Ju'hoan_North, and Taa_East.
    </div>
   </div>
   <img alt="ROLLOFF analysis plot for French, Ju'hoan_North, and Han." src="20e597e389dfd8d131e05ad6e1617dcd_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Correlation
    </p>
    <p>
     Distance (cM)
    </p>
    <p>
     French -&gt; Ju'hoan_North&lt;- Han
    </p>
    <p>
     This plot shows the correlation between LD decay in Ju'hoan_North and the product of LD differences between French and Han. The y-axis is 'Correlation' (0.00 to 0.06) and the x-axis is 'Distance (cM)' (0 to 20). Data points are black 'x' marks, and a red line shows a fitted exponential decay curve that remains very close to zero across the entire distance range.
    </p>
    <div class="img-alt">
     ROLLOFF analysis plot for French, Ju'hoan_North, and Han.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S12: ROLLOFF analysis of the Ju'hoan_North.
    </b>
    We explored the correlation between the decay of LD in the Ju'hoan_North and the divergence between other pairs of populations using ROLLOFF. At each pair of SNPs, we estimate the amount of LD in the Ju'hoan_North (as measured by a correlation in genotypes [27]) and the product of the differences in allele frequency between two reference populations. The reference populations in each panel are listed to either side of the Ju'hoan_North. We then calculate the correlation between these two values, binning pairs of SNPs by the genetic distance between them. Each point is the value of this correlation (the y-axis) plotted against the genetic distance bin (the x-axis). A detectable curve suggests that the target population (in this case the Ju'hoan) is admixed. Note a curve can be present even if the reference populations are quite distant from the true mixing populations. In this case, a curve is seen except when using two non-African populations as references. In red is the fitted exponential curve.
   </p>
  </div>
  <div class="page" data-page-id="30">
   <img alt="A 3x3 grid of nine scatter plots showing the ROLLOFF analysis for various southern African populations. Each plot has 'Correlation' on the y-axis (0.00 to 0.08) and 'Distance (cM)' on the x-axis (0 to 30). Data points are shown as black 'x' marks, and a fitted red exponential curve is overlaid. Shaded gray regions around the curves indicate confidence intervals. The populations shown are: Taa_East vs Taa_West, #Hoan vs Gilana, Mbukushu vs !Xuun, Tshwa vs Shua, Kgalagadi vs Tswana, Damara vs Khwe, Haillom vs Taa_North, Jul'hoan_South vs Naro, and Himba vs Wambo." src="bafe3665fa89ba09857af5a2532c79fe_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <div class="img-alt">
     A 3x3 grid of nine scatter plots showing the ROLLOFF analysis for various southern African populations. Each plot has 'Correlation' on the y-axis (0.00 to 0.08) and 'Distance (cM)' on the x-axis (0 to 30). Data points are shown as black 'x' marks, and a fitted red exponential curve is overlaid. Shaded gray regions around the curves indicate confidence intervals. The populations shown are: Taa_East vs Taa_West, #Hoan vs Gilana, Mbukushu vs !Xuun, Tshwa vs Shua, Kgalagadi vs Tswana, Damara vs Khwe, Haillom vs Taa_North, Jul'hoan_South vs Naro, and Himba vs Wambo.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S13: ROLLOFF analysis of all southern African populations.
    </b>
    For each southern African population, we ran ROLLOFF [27] using the Ju|'hoan_North and Yoruba as the mixing populations. The method is as described in Supplementary Figure
    <a href="#block-29-0">S12</a>
    and the Supplementary Material. Shown are the resulting curves for each population; in red are the fitted exponential curves.
   </p>
  </div>
  <div class="page" data-page-id="31">
   <img alt="A 3x3 grid of scatter plots showing residual correlation versus distance (cM) for nine southern African populations. Each plot compares two populations: Taa_East vs Taa_West, #Hoan vs Gllana, Mbukushu vs IXuun, Tshwa vs Shua, Kgalagadi vs Tswana, Damara vs Khwe, Haillom vs Taa_North, Jul'hoan_South vs Naro, and Himba vs Wambo. The y-axis for all plots is 'Residual correlation' ranging from -0.04 to 0.04, and the x-axis is 'Distance (cM)' ranging from 0 to 30. Data points are represented by 'x' markers, and a shaded gray area indicates the fitted exponential curve." src="48a08e5cabec8b75386679d8a57dec3e_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <div class="img-alt">
     A 3x3 grid of scatter plots showing residual correlation versus distance (cM) for nine southern African populations. Each plot compares two populations: Taa_East vs Taa_West, #Hoan vs Gllana, Mbukushu vs IXuun, Tshwa vs Shua, Kgalagadi vs Tswana, Damara vs Khwe, Haillom vs Taa_North, Jul'hoan_South vs Naro, and Himba vs Wambo. The y-axis for all plots is 'Residual correlation' ranging from -0.04 to 0.04, and the x-axis is 'Distance (cM)' ranging from 0 to 30. Data points are represented by 'x' markers, and a shaded gray area indicates the fitted exponential curve.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S14: Residuals from ROLLOFF analysis of all southern African populations.
    </b>
    Plotted are the residuals from the fit of the exponential curves for each population from Supplementary Figure
    <a href="#block-30-0">S13.</a>
    Residual correlation may indicate multiple waves of mixture in some populations.
   </p>
  </div>
  <div class="page" data-page-id="32">
   <p>
    A. Model demography
   </p>
   <img alt="Diagram of a demographic model showing population splits and mutation history." src="f1df41f68d1ddd39987bd08da7aeadc6_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     The diagram illustrates a demographic model with four populations: O, X, A, and Y. The population tree shows a split between O and X, followed by a split between X and A, and finally a split between A and Y. A grey line represents the history of two chromosomes used for SNP ascertainment, which diverges from the main population tree at the split between X and A. A time point 't' is marked on the grey line. Mutations are represented by stars: black stars indicate mutations that arose before the population split (in population A), and red stars indicate mutations that arose after the split (in population Y).
    </p>
    <div class="img-alt">
     Diagram of a demographic model showing population splits and mutation history.
    </div>
   </div>
   <p>
    B. Allelic spectrum in A (ancestral population)
   </p>
   <img alt="Histogram of the allelic spectrum in population A." src="051638d871c75230edb3d005fa668810_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A histogram showing the density of derived allele frequency in population A. The x-axis is labeled 'derived allele frequency' and ranges from 0 to 1. The y-axis is labeled 'density'. The distribution is unimodal and skewed towards zero. A prominent red peak is visible at zero frequency, representing mutations that arose on the lineage to Y. The rest of the distribution is represented by black bars.
    </p>
    <div class="img-alt">
     Histogram of the allelic spectrum in population A.
    </div>
   </div>
   <p>
    C. Allelic spectrum in X
   </p>
   <img alt="Histogram of the allelic spectrum in population X." src="bac21fd48fcd7f025c723590e07d1823_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A histogram showing the density of derived allele frequency in population X. The x-axis is labeled 'derived allele frequency' and ranges from 0 to 1. The y-axis is labeled 'density'. The distribution is unimodal and skewed towards zero. A red peak is present at zero frequency, but its relative size is smaller compared to the peak in population A. The rest of the distribution is represented by black bars.
    </p>
    <div class="img-alt">
     Histogram of the allelic spectrum in population X.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S15: Scheme for dating population splits.
    </b>
    <b>
     A. Demographic model.
    </b>
    Plotted is the demographic model used in our method for dating population split times. Populations are labeled in black, and the split time is denoted
    <math>
     t
    </math>
    . In grey is the history of the two chromosomes used for SNP ascertainment. Stars represent mutations, and are colored according to whether they arose before (black) or after (red) the population split.
    <b>
     B.
    </b>
    A hypothetical allelic spectrum in population A. The red peak at zero corresponds to the mutations that happened on the lineage to Y.
    <b>
     C.
    </b>
    The hypothetical allelic spectrum in X. Though alleles change frequency from A to X, the size of the red component of the peak at zero stays constant.
   </p>
  </div>
  <div class="page" data-page-id="33">
   <img alt="Scatter plot showing Estimated split time (generations) vs Simulated split time (generations). The plot shows a strong positive correlation, with data points clustered along the diagonal line y=x. Red stars represent true simulated values, and black circles represent estimates. The x-axis ranges from 0 to 2000, and the y-axis ranges from 0 to 2000." src="183007754364096b2d89f42200cf870f_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <table border="1">
     <thead>
      <tr>
       <th>
        Simulated split time (generations)
       </th>
       <th>
        Estimated split time (generations)
       </th>
       <th>
        Marker Type
       </th>
      </tr>
     </thead>
     <tbody>
      <tr>
       <td>
        400
       </td>
       <td>
        400
       </td>
       <td>
        Red Star
       </td>
      </tr>
      <tr>
       <td>
        400
       </td>
       <td>
        420
       </td>
       <td>
        Black Circle
       </td>
      </tr>
      <tr>
       <td>
        1200
       </td>
       <td>
        1050
       </td>
       <td>
        Black Circle
       </td>
      </tr>
      <tr>
       <td>
        1200
       </td>
       <td>
        1150
       </td>
       <td>
        Red Star
       </td>
      </tr>
      <tr>
       <td>
        1200
       </td>
       <td>
        1200
       </td>
       <td>
        Black Circle
       </td>
      </tr>
      <tr>
       <td>
        1200
       </td>
       <td>
        1250
       </td>
       <td>
        Black Circle
       </td>
      </tr>
      <tr>
       <td>
        2000
       </td>
       <td>
        1700
       </td>
       <td>
        Black Circle
       </td>
      </tr>
      <tr>
       <td>
        2000
       </td>
       <td>
        1900
       </td>
       <td>
        Red Star
       </td>
      </tr>
      <tr>
       <td>
        2000
       </td>
       <td>
        1950
       </td>
       <td>
        Black Circle
       </td>
      </tr>
     </tbody>
    </table>
    <div class="img-alt">
     Scatter plot showing Estimated split time (generations) vs Simulated split time (generations). The plot shows a strong positive correlation, with data points clustered along the diagonal line y=x. Red stars represent true simulated values, and black circles represent estimates. The x-axis ranges from 0 to 2000, and the y-axis ranges from 0 to 2000.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S16: Estimating split times in simulations without migration.
    </b>
    Shown are the simulated and inferred split times in simulations without migration. The red stars show the true simulated values, and the black points the estimates.
   </p>
  </div>
  <div class="page" data-page-id="34">
   <img alt="Two scatter plots, A and B, showing estimated split times (generations) on the y-axis versus admixture proportion in X on the x-axis. Plot A is for t = 400 generations and Plot B is for t = 2000 generations. Both plots show black points representing estimated split times and red stars representing true simulated values. The y-axis ranges from 500 to 2000 generations, and the x-axis ranges from 0.0 to 0.5." src="24b549c0c4b79c515b244625cf4f4978_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Figure S17 consists of two scatter plots, A and B, showing the relationship between the admixture proportion in population X (x-axis) and the estimated split time between populations X and Y (y-axis, in generations).
    </p>
    <p>
     <b>
      Plot A:
      <math>
       t = 400
      </math>
      generations
     </b>
    </p>
    <table border="1">
     <thead>
      <tr>
       <th>
        Admixture proportion in X
       </th>
       <th>
        Estimated split time (generations)
       </th>
      </tr>
     </thead>
     <tbody>
      <tr>
       <td>
        0.0
       </td>
       <td>
        ~550
       </td>
      </tr>
      <tr>
       <td>
        0.05
       </td>
       <td>
        ~750
       </td>
      </tr>
      <tr>
       <td>
        0.1
       </td>
       <td>
        ~900
       </td>
      </tr>
      <tr>
       <td>
        0.2
       </td>
       <td>
        ~1000
       </td>
      </tr>
      <tr>
       <td>
        0.3
       </td>
       <td>
        ~1200
       </td>
      </tr>
      <tr>
       <td>
        0.4
       </td>
       <td>
        ~1200
       </td>
      </tr>
     </tbody>
    </table>
    <p>
     <b>
      Plot B:
      <math>
       t = 2000
      </math>
      generations
     </b>
    </p>
    <table border="1">
     <thead>
      <tr>
       <th>
        Admixture proportion in X
       </th>
       <th>
        Estimated split time (generations)
       </th>
      </tr>
     </thead>
     <tbody>
      <tr>
       <td>
        0.0
       </td>
       <td>
        ~2000
       </td>
      </tr>
      <tr>
       <td>
        0.05
       </td>
       <td>
        ~2000
       </td>
      </tr>
      <tr>
       <td>
        0.1
       </td>
       <td>
        ~2000
       </td>
      </tr>
      <tr>
       <td>
        0.2
       </td>
       <td>
        ~2100
       </td>
      </tr>
      <tr>
       <td>
        0.3
       </td>
       <td>
        ~2100
       </td>
      </tr>
      <tr>
       <td>
        0.4
       </td>
       <td>
        ~2100
       </td>
      </tr>
     </tbody>
    </table>
    <div class="img-alt">
     Two scatter plots, A and B, showing estimated split times (generations) on the y-axis versus admixture proportion in X on the x-axis. Plot A is for t = 400 generations and Plot B is for t = 2000 generations. Both plots show black points representing estimated split times and red stars representing true simulated values. The y-axis ranges from 500 to 2000 generations, and the x-axis ranges from 0.0 to 0.5.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S17: Estimating split times in simulations with migration.
    </b>
    Shown are estimated split times between
    <math>
     X
    </math>
    and
    <math>
     Y
    </math>
    when both have experienced some level of admixture with an outgroup. The black points show estimated split times in the presence of admixture. The red stars show the true simulated values. In all simulations, population
    <math>
     Y
    </math>
    has 5% admixture from the outgroup that occurred 40 generations in the past, while population
    <math>
     X
    </math>
    has variable levels of admixture (plotted on the x-axis).
   </p>
  </div>
  <div class="page" data-page-id="35">
   <img alt="Scatter plot showing the estimated time since split from Jul'hoan_North (t) versus the fraction of non-Khoisan ancestry for various Khoisan populations. The y-axis ranges from 0 to 1500 generations, and the x-axis ranges from 0.0 to 0.6. Populations are categorized by color: NW Kalahari (black), SE Kalahari (red), and unclear (green). Each point includes a vertical error bar representing one standard error." src="152efae989544ee653283e8de26cc9b1_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <table border="1">
     <caption>
      Estimated data points from Supplementary Figure S18
     </caption>
     <thead>
      <tr>
       <th>
        Population
       </th>
       <th>
        fraction non-Khoisan ancestry
       </th>
       <th>
        t (generations since split from Jul'hoan_North)
       </th>
       <th>
        Category
       </th>
      </tr>
     </thead>
     <tbody>
      <tr>
       <td>
        Jul'hoan_South
       </td>
       <td>
        0.08
       </td>
       <td>
        ~80
       </td>
       <td>
        NW Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Naro
       </td>
       <td>
        0.09
       </td>
       <td>
        ~200
       </td>
       <td>
        unclear
       </td>
      </tr>
      <tr>
       <td>
        Taa_North
       </td>
       <td>
        0.10
       </td>
       <td>
        ~850
       </td>
       <td>
        SE Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Taa_West
       </td>
       <td>
        0.12
       </td>
       <td>
        ~650
       </td>
       <td>
        SE Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Taa_East
       </td>
       <td>
        0.17
       </td>
       <td>
        ~980
       </td>
       <td>
        SE Kalahari
       </td>
      </tr>
      <tr>
       <td>
        !Hoan
       </td>
       <td>
        0.23
       </td>
       <td>
        ~1050
       </td>
       <td>
        SE Kalahari
       </td>
      </tr>
      <tr>
       <td>
        !Xuun
       </td>
       <td>
        0.25
       </td>
       <td>
        ~400
       </td>
       <td>
        NW Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Gilana
       </td>
       <td>
        0.38
       </td>
       <td>
        ~1100
       </td>
       <td>
        SE Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Haillom
       </td>
       <td>
        0.43
       </td>
       <td>
        ~500
       </td>
       <td>
        NW Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Tshwa
       </td>
       <td>
        0.48
       </td>
       <td>
        ~950
       </td>
       <td>
        SE Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Shua
       </td>
       <td>
        0.62
       </td>
       <td>
        ~850
       </td>
       <td>
        SE Kalahari
       </td>
      </tr>
      <tr>
       <td>
        Khwe
       </td>
       <td>
        0.63
       </td>
       <td>
        ~580
       </td>
       <td>
        unclear
       </td>
      </tr>
     </tbody>
    </table>
    <div class="img-alt">
     Scatter plot showing the estimated time since split from Jul'hoan_North (t) versus the fraction of non-Khoisan ancestry for various Khoisan populations. The y-axis ranges from 0 to 1500 generations, and the x-axis ranges from 0.0 to 0.6. Populations are categorized by color: NW Kalahari (black), SE Kalahari (red), and unclear (green). Each point includes a vertical error bar representing one standard error.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S18: Dating the split time of the Khoisan populations.
    </b>
    We plot the estimated times since each Khoisan population split from the Jul'hoan_North, as a function of their level of non-Khoisan-admixture. The populations included are all the southern African groups in Figure 3 in the main text. The errors bars are one standard error (not including the error in the estimate of
    <math>
     \tau
    </math>
    ). Khoisan populations are colored according to whether they have strong evidence (from Figure 3 in the main text) as coming from the northwestern Kalahari cluster or the southeastern Kalahari cluster. Populations that have no clear grouping are colored in green. All split times are likely overestimated due to non-Khoisan admixture (see Supplementary Figure S
    <a href="#block-34-0">17)</a>
   </p>
  </div>
  <div class="page" data-page-id="36">
   <img alt="Supplementary Figure S19: Dating the split time of Taa_North. The figure consists of four panels (A and B, left and right) showing allelic spectra. Panel A (Taa_North) shows empirical (black) and fitted (grey for t=0, red for t estimated) spectra for derived allele counts 0-10. Panel B (simulation) shows empirical (black) and fitted (red for t estimated) spectra for derived allele counts 0-20. In all panels, the y-axis is 'Fraction' and the x-axis is 'Derived allele count'." src="f24d06b5e3b1d8ae12d4893e7619f6f5_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     <b>
      A. Fitted and empirical allelic spectra in the Taa_North
     </b>
    </p>
    <p>
     <b>
      B. Fitted and empirical allelic spectra in a simulation of
      <math>
       t = 2000
      </math>
      generations
     </b>
    </p>
    <p>
     Legend for all panels:
    </p>
    <ul style="list-style-type: none">
     <li>
      ● empirical spectrum
     </li>
     <li>
      ● fitted (
      <math>
       t = 0
      </math>
      )
     </li>
     <li>
      ● fitted (
      <math>
       t
      </math>
      estimated)
     </li>
    </ul>
    <div class="img-alt">
     Supplementary Figure S19: Dating the split time of Taa_North. The figure consists of four panels (A and B, left and right) showing allelic spectra. Panel A (Taa_North) shows empirical (black) and fitted (grey for t=0, red for t estimated) spectra for derived allele counts 0-10. Panel B (simulation) shows empirical (black) and fitted (red for t estimated) spectra for derived allele counts 0-20. In all panels, the y-axis is 'Fraction' and the x-axis is 'Derived allele count'.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S19: Dating the split time of Taa_North.
    </b>
    <b>
     A.
    </b>
    We plot the empirical allele frequency spectrum in the Taa_North at SNPs ascertained in a single Ju'hoan_North individual (in black). For comparison we plot the fitted allelic spectra if we assume the split time between Taa_North and the Ju'hoan_North is zero (in grey in the left panel) or if we allow the model to estimate the split time (in red in the right panel). Note that the empirical spectrum is non-linear, implying that the ancestral population was not of constant size.
    <b>
     B.
    </b>
    We plot the analogous spectra for a single simulation of a split time of 2,000 generations with no migration.
   </p>
  </div>
  <div class="page" data-page-id="37">
   <p>
    A. Maximum likelihood tree
   </p>
   <img alt="Maximum likelihood tree of populations including the Hadza." src="6ee57fd30c7e609827c2a11d0983eeba_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A phylogenetic tree showing the relationships between various populations. The tree is rooted at Chimp. The populations are: Chimp, Hadza, Yoruba, Dinka, Han, Oroqen, French, Basque, Taa_East, Taa_North, Jul'hoan_South, and Jul'hoan_North. The x-axis represents the Drift parameter, ranging from 0.00 to 0.12 with a scale bar of 10 s.e. The Hadza population is shown as a sister group to the Yoruba and Dinka populations.
    </p>
    <div class="img-alt">
     Maximum likelihood tree of populations including the Hadza.
    </div>
   </div>
   <p>
    B. Residual fit from tree in A.
   </p>
   <img alt="Residual fit matrix from the maximum likelihood tree in A." src="771c18f874d31c59c3b8c4e247be16ca_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A heatmap showing the residual fit from the maximum likelihood tree in A. The rows and columns represent the populations: Chimp, Taa_East, Taa_North, Jul'hoan_North, Jul'hoan_South, Yoruba, Dinka, Hadza, French, Oroqen, Basque, and Han. The color scale ranges from -18.1 SE (dark blue) to 18.1 SE (dark red). The diagonal elements are zero, and the off-diagonal elements show the residual fit between populations.
    </p>
    <div class="img-alt">
     Residual fit matrix from the maximum likelihood tree in A.
    </div>
   </div>
   <p>
    C. Maximum likelihood graph
   </p>
   <img alt="Maximum likelihood graph allowing for a single migration edge." src="0aa15f5c9c3edae230985491199cfe8b_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A phylogenetic graph showing the relationships between populations, allowing for a single migration edge. The tree structure is the same as in panel A, but an orange arrow indicates a migration edge from the Yoruba and Dinka clade to the Hadza population. The x-axis represents the Drift parameter, ranging from 0.00 to 0.12 with a scale bar of 10 s.e. A color scale for Migration weight ranges from 0 (yellow) to 0.5 (red).
    </p>
    <div class="img-alt">
     Maximum likelihood graph allowing for a single migration edge.
    </div>
   </div>
   <p>
    D. Residual fit from graph in C.
   </p>
   <img alt="Residual fit matrix from the maximum likelihood graph in C." src="ddc89e164666201115a1c006e4c3b6da_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A heatmap showing the residual fit from the maximum likelihood graph in C. The rows and columns represent the populations: Chimp, Taa_East, Taa_North, Jul'hoan_North, Jul'hoan_South, Yoruba, Dinka, Hadza, French, Oroqen, Basque, and Han. The color scale ranges from -5.2 SE (dark blue) to 5.2 SE (dark red). The diagonal elements are zero, and the off-diagonal elements show the residual fit between populations, with the migration edge from Yoruba/Dinka to Hadza being highlighted.
    </p>
    <div class="img-alt">
     Residual fit matrix from the maximum likelihood graph in C.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S20:
     <i>
      TreeMix
     </i>
     analysis of the Hadza.
    </b>
    Shown is the maximum likelihood tree of populations including the Hadza (A.), the residual fit from this tree (B.), the inferred graph allowing for a single migration edge (C.), and the residual fit from this graph (D.). See Supplementary text for discussion. Note that the choice of which edge to the Hadza is called the “migration” edge is arbitrary [36] ; for Figure 3 in the main text we force the non-Khoisan ancestry in the Hadza to be the “migration” edge.
   </p>
  </div>
  <div class="page" data-page-id="38">
   <p>
    A. Maximum likelihood tree
   </p>
   <img alt="Maximum likelihood tree of populations including the Sandawe individuals." src="27b09ea51378a0f896d21b3ebad0b22f_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A phylogenetic tree showing the relationships between various populations. The tree is rooted at Chimp. The populations are: Oroqen, Han, French, Basque, Sandawe, Dinka, Yoruba, Taa_East, Taa_North, Jul'hoan_South, and Jul'hoan_North. The x-axis represents the Drift parameter, ranging from 0.00 to 0.12. A scale bar indicates
     <math>
      H/10 \text{ s.e.}
     </math>
     .
    </p>
    <div class="img-alt">
     Maximum likelihood tree of populations including the Sandawe individuals.
    </div>
   </div>
   <p>
    B. Residual fit from tree in A.
   </p>
   <img alt="Residual fit heatmap from the maximum likelihood tree in A." src="673e9e5873f9a4b71bbe7bac2cf6b758_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A heatmap showing the residual fit from the maximum likelihood tree in A. The rows and columns represent the same populations as in the tree. The color scale ranges from -11.7 SE (red) to 11.7 SE (dark blue). The populations listed are: Chimp, Taa_East, Taa_North, Jul'hoan_North, Jul'hoan_South, Yoruba, Dinka, Sandawe, French, Oroqen, Basque, and Han.
    </p>
    <div class="img-alt">
     Residual fit heatmap from the maximum likelihood tree in A.
    </div>
   </div>
   <p>
    C. Maximum likelihood graph
   </p>
   <img alt="Maximum likelihood graph allowing for a single migration edge." src="455c842681aadb4969bd61d71f556744_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A phylogenetic graph showing the relationships between populations, similar to the tree in A, but with an additional migration edge. The migration edge is highlighted in orange, connecting the lineage leading to Sandawe with the lineage leading to the other populations. The x-axis represents the Drift parameter, ranging from 0.00 to 0.12. A scale bar indicates
     <math>
      H/10 \text{ s.e.}
     </math>
     . A color scale for the migration weight ranges from 0 (yellow) to 0.5 (red).
    </p>
    <div class="img-alt">
     Maximum likelihood graph allowing for a single migration edge.
    </div>
   </div>
   <p>
    D. Residual fit from graph in C.
   </p>
   <img alt="Residual fit heatmap from the maximum likelihood graph in C." src="7d06972584fb7598ad36e8b3df8f466e_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     A heatmap showing the residual fit from the maximum likelihood graph in C. The rows and columns represent the same populations as in the graph. The color scale ranges from -5.5 SE (red) to 5.5 SE (dark blue). The populations listed are: Chimp, Taa_East, Taa_North, Jul'hoan_North, Jul'hoan_South, Yoruba, Dinka, Sandawe, French, Oroqen, Basque, and Han.
    </p>
    <div class="img-alt">
     Residual fit heatmap from the maximum likelihood graph in C.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S21:
     <i>
      TreeMix
     </i>
     analysis of the Sandawe.
    </b>
    Shown is the maximum likelihood tree of populations including the Sandawe individuals. (A.), the residual fit from this tree (B.), the inferred graph allowing for a single migration edge (C.), and the residual fit from this graph (D.). See Supplementary text for discussion. Note that the choice of which edge to the Sandawe is called the “migration” edge is arbitrary [36] ; for Figure 3 in the main text we force the non-Khoisan ancestry in the Sandawe to be the “migration” edge.
   </p>
  </div>
  <div class="page" data-page-id="39">
   <img alt="Scatter plot of weighted correlation vs genetic distance for Dinka-&gt; Hadza &lt;- French." src="157e8c1d80a02c8eee893c2ea60a77a8_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Scatter plot showing Weighted correlation (Y-axis, ranging from -0.01 to 0.03) versus Genetic distance (cM) (X-axis, ranging from 0 to 20). The plot is titled "Dinka-&gt; Hadza &lt;- French". Data points are represented by black 'x' marks, and a red curve is fitted to the data. The correlation starts at approximately 0.015 at 0 cM, drops sharply to about -0.005 at 2 cM, and then remains relatively stable around -0.005 for the rest of the distance.
    </p>
    <div class="img-alt">
     Scatter plot of weighted correlation vs genetic distance for Dinka-&gt; Hadza &lt;- French.
    </div>
   </div>
   <img alt="Scatter plot of weighted correlation vs genetic distance for Dinka-&gt; Sandawe &lt;- French." src="befcd1fc29f753206dbe01036a9dca5e_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Scatter plot showing Weighted correlation (Y-axis, ranging from -0.01 to 0.03) versus Genetic distance (cM) (X-axis, ranging from 0 to 20). The plot is titled "Dinka-&gt; Sandawe &lt;- French". Data points are represented by black 'x' marks, and a red curve is fitted to the data. The correlation starts at approximately 0.035 at 0 cM, drops sharply to about -0.005 at 2 cM, and then remains relatively stable around -0.005 for the rest of the distance, showing a much more pronounced initial drop than the Hadza plot.
    </p>
    <div class="img-alt">
     Scatter plot of weighted correlation vs genetic distance for Dinka-&gt; Sandawe &lt;- French.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S22: Admixture linkage disequilibrium in the Hadza and Sandawe.
    </b>
    We ran ROLLOFF [27] on the Hadza and Sandawe, using the Dinka and French as reference populations. Shown are the resulting curves for the Hadza and Sandawe. There is a striking curve of admixture LD in the Sandawe, which is weaker in the Hadza.
   </p>
  </div>
  <div class="page" data-page-id="40">
   <img alt="A TreeMix phylogenetic tree showing the relationships between various human populations and chimpanzees. The tree is rooted with 'Chimp' at the top. The main branches lead to 'Mbuti' and 'Biaka' on the left, and a larger branch on the right containing 'Han', 'Oroqen', 'French', 'Basque', 'Dinka', and 'Yoruba'. Below these, a complex cluster of populations is shown, including 'Khwe' (blue), 'Jul'hoan_South' (green), 'Jul'hoan_North' (green), '!Xuun' (green), 'Hailom' (blue), 'Naro' (blue), 'Taa_East' (red), '#Hoan' (green), 'Shua' (blue), 'G||ana' (blue), 'Tshwa' (blue), 'Taa_North' (red), and 'Taa_West' (red). The tree is plotted against a 'Drift parameter' axis at the bottom, ranging from 0.00 to 0.12. A scale bar indicates 'H 10 s.e.'." src="32f80b2d70752613b6577797811f3a95_img.jpg"/>
   <div class="img-description" style="border: 1px solid #ccc; padding: 10px;">
    <p>
     Drift parameter
    </p>
    <p>
     0.00 0.02 0.04 0.06 0.08 0.10 0.12
    </p>
    <p>
     H
     <br/>
     10 s.e.
    </p>
    <div class="img-alt">
     A TreeMix phylogenetic tree showing the relationships between various human populations and chimpanzees. The tree is rooted with 'Chimp' at the top. The main branches lead to 'Mbuti' and 'Biaka' on the left, and a larger branch on the right containing 'Han', 'Oroqen', 'French', 'Basque', 'Dinka', and 'Yoruba'. Below these, a complex cluster of populations is shown, including 'Khwe' (blue), 'Jul'hoan_South' (green), 'Jul'hoan_North' (green), '!Xuun' (green), 'Hailom' (blue), 'Naro' (blue), 'Taa_East' (red), '#Hoan' (green), 'Shua' (blue), 'G||ana' (blue), 'Tshwa' (blue), 'Taa_North' (red), and 'Taa_West' (red). The tree is plotted against a 'Drift parameter' axis at the bottom, ranging from 0.00 to 0.12. A scale bar indicates 'H 10 s.e.'.
    </div>
   </div>
   <p>
    <b>
     Supplementary Figure S23:
     <i>
      TreeMix
     </i>
     analysis including the Mbuti and Biaka.
    </b>
    We used a modified
    <i>
     TreeMix
    </i>
    approach to build a tree of populations after subtracting out Bantu or Dinka-like ancestry. Shown is the resulting tree; see the Supplementary text for details and discussion.
   </p>
  </div>
  <div class="page" data-page-id="41">
   <h1>
    2 Supplementary Tables
   </h1>
   <table border="1">
    <thead>
     <tr>
      <th>
       Population
      </th>
      <th>
       Language family
      </th>
      <th>
       Linguistic subgroup
      </th>
      <th>
       # of samples
      </th>
     </tr>
    </thead>
    <tbody>
     <tr>
      <td>
       Taa_East
      </td>
      <td>
       Tuu
      </td>
      <td>
       Taa-Lower Nossob
      </td>
      <td>
       6
      </td>
     </tr>
     <tr>
      <td>
       Taa_North
      </td>
      <td>
       Tuu
      </td>
      <td>
       Taa-Lower Nossob
      </td>
      <td>
       6
      </td>
     </tr>
     <tr>
      <td>
       Taa_West
      </td>
      <td>
       Tuu
      </td>
      <td>
       Taa-Lower Nossob
      </td>
      <td>
       8
      </td>
     </tr>
     <tr>
      <td>
       !Xuun
      </td>
      <td>
       Kx'a
      </td>
      <td>
       Northwest Ju
      </td>
      <td>
       13
      </td>
     </tr>
     <tr>
      <td>
       Ju|'hoan_North
      </td>
      <td>
       Kx'a
      </td>
      <td>
       Southeast Ju
      </td>
      <td>
       16
      </td>
     </tr>
     <tr>
      <td>
       Ju|'hoan_South
      </td>
      <td>
       Kx'a
      </td>
      <td>
       Southeast Ju
      </td>
      <td>
       9
      </td>
     </tr>
     <tr>
      <td>
       ±Hoan
      </td>
      <td>
       Kx'a
      </td>
      <td>
       ±Hoan
      </td>
      <td>
       7
      </td>
     </tr>
     <tr>
      <td>
       Shua
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       East Kalahari Khoe
      </td>
      <td>
       10
      </td>
     </tr>
     <tr>
      <td>
       Tshwa
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       East Kalahari Khoe
      </td>
      <td>
       10
      </td>
     </tr>
     <tr>
      <td>
       Khwe
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       West Kalahari Khoe, Kxoe branch
      </td>
      <td>
       10
      </td>
     </tr>
     <tr>
      <td>
       Naro
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       West Kalahari Khoe, Naro branch
      </td>
      <td>
       10
      </td>
     </tr>
     <tr>
      <td>
       G|ui
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       West Kalahari Khoe, G||ana branch
      </td>
      <td>
       5
      </td>
     </tr>
     <tr>
      <td>
       G||ana
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       West Kalahari Khoe, G||ana branch
      </td>
      <td>
       5
      </td>
     </tr>
     <tr>
      <td>
       Hai||om
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       KhoeKhoe
      </td>
      <td>
       10
      </td>
     </tr>
     <tr>
      <td>
       Nama
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       KhoeKhoe
      </td>
      <td>
       16
      </td>
     </tr>
     <tr>
      <td>
       Damara
      </td>
      <td>
       Khoe-Kwadi
      </td>
      <td>
       KhoeKhoe
      </td>
      <td>
       15
      </td>
     </tr>
     <tr>
      <td>
       Kgalagadi
      </td>
      <td>
       Niger-Congo
      </td>
      <td>
       Bantu
      </td>
      <td>
       5
      </td>
     </tr>
     <tr>
      <td>
       Wambo
      </td>
      <td>
       Niger-Congo
      </td>
      <td>
       Bantu
      </td>
      <td>
       5
      </td>
     </tr>
     <tr>
      <td>
       Mbukushu
      </td>
      <td>
       Niger-Congo
      </td>
      <td>
       Bantu
      </td>
      <td>
       4
      </td>
     </tr>
     <tr>
      <td>
       Tswana
      </td>
      <td>
       Niger-Congo
      </td>
      <td>
       Bantu
      </td>
      <td>
       5
      </td>
     </tr>
     <tr>
      <td>
       Himba
      </td>
      <td>
       Niger-Congo
      </td>
      <td>
       Bantu
      </td>
      <td>
       5
      </td>
     </tr>
     <tr>
      <td>
       Hadza
      </td>
      <td>
       isolate
      </td>
      <td>
       Hadza
      </td>
      <td>
       7
      </td>
     </tr>
    </tbody>
   </table>
   <p>
    <b>
     Supplementary Table S1:
    </b>
    Summary of samples genotyped in this study.
   </p>
  </div>
  <div class="page" data-page-id="42">
   <table border="1">
    <thead>
     <tr>
      <th>
       Sample
      </th>
      <th>
       Population
      </th>
     </tr>
    </thead>
    <tbody>
     <tr>
      <td>
       BOT6.090
      </td>
      <td>
       Ju|'hoan_South
      </td>
     </tr>
     <tr>
      <td>
       NAM066
      </td>
      <td>
       Ju|'hoan_South
      </td>
     </tr>
     <tr>
      <td>
       NAM051
      </td>
      <td>
       Ju|'hoan_South
      </td>
     </tr>
     <tr>
      <td>
       BOT6.025
      </td>
      <td>
       Taa_North
      </td>
     </tr>
     <tr>
      <td>
       BOT6.255
      </td>
      <td>
       Shua
      </td>
     </tr>
     <tr>
      <td>
       NAM189
      </td>
      <td>
       !Xuun
      </td>
     </tr>
     <tr>
      <td>
       NAM195
      </td>
      <td>
       !Xuun
      </td>
     </tr>
     <tr>
      <td>
       BOT6.004
      </td>
      <td>
       Kgalagadi
      </td>
     </tr>
     <tr>
      <td>
       DR000071
      </td>
      <td>
       Hadza
      </td>
     </tr>
     <tr>
      <td>
       BOT6.058
      </td>
      <td>
       Naro
      </td>
     </tr>
    </tbody>
   </table>
   <p>
    <b>
     Supplementary Table S2:
    </b>
    Individuals removed from analysis.
   </p>
  </div>
  <div class="page" data-page-id="43">
   <table border="1">
    <thead>
     <tr>
      <th>
       Target Population
      </th>
      <th>
       “Mixing” populations
      </th>
      <th>
       Minimum
       <math>
        f_3
       </math>
      </th>
      <th>
       Z-score
      </th>
     </tr>
    </thead>
    <tbody>
     <tr>
      <td>
       Khwe
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.005
      </td>
      <td>
       -38.7
      </td>
     </tr>
     <tr>
      <td>
       Hai||om
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.005
      </td>
      <td>
       -33.9
      </td>
     </tr>
     <tr>
      <td>
       Tshwa
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.005
      </td>
      <td>
       -29.9
      </td>
     </tr>
     <tr>
      <td>
       Shua
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.004
      </td>
      <td>
       -28.8
      </td>
     </tr>
     <tr>
      <td>
       Tswana
      </td>
      <td>
       Yoruba, Taa_West
      </td>
      <td>
       -0.004
      </td>
      <td>
       -23.8
      </td>
     </tr>
     <tr>
      <td>
       !Xuun
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.004
      </td>
      <td>
       -20.3
      </td>
     </tr>
     <tr>
      <td>
       G||ana
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.005
      </td>
      <td>
       -21.3
      </td>
     </tr>
     <tr>
      <td>
       Kgalagadi
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.002
      </td>
      <td>
       -8.4
      </td>
     </tr>
     <tr>
      <td>
       Naro
      </td>
      <td>
       Ju|’hoan_North, Taa_North
      </td>
      <td>
       -0.0006
      </td>
      <td>
       -4.4
      </td>
     </tr>
     <tr>
      <td>
       Mbukushu
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.0008
      </td>
      <td>
       -3.9
      </td>
     </tr>
     <tr>
      <td>
       Taa_West
      </td>
      <td>
       Taa_North, Kgalagadi
      </td>
      <td>
       -0.0008
      </td>
      <td>
       -3.6
      </td>
     </tr>
     <tr>
      <td>
       Wambo
      </td>
      <td>
       Ju|’hoan_North, Yoruba
      </td>
      <td>
       -0.0003
      </td>
      <td>
       -1.6
      </td>
     </tr>
    </tbody>
   </table>
   <p>
    <b>
     Supplementary Table S3: Three-population tests for treeness.
    </b>
    We performed three-population tests on all possible combinations of populations. Shown are all populations with at least one negative
    <math>
     f_3
    </math>
    statistic, the names of the putative mixing populations that give rise to the minimum
    <math>
     f_3
    </math>
    statistic, the value of the statistic, and the Z-score. A Z-score of less than -3 corresponds to a p-value of less than 0.001. The populations labeled as “mixing” populations are those that give the minimum
    <math>
     f_3
    </math>
    statistic, and are not necessarily the populations that actually mixed historically.
   </p>
  </div>
  <div class="page" data-page-id="44">
   <table border="1">
    <thead>
     <tr>
      <th>
       Population
      </th>
      <th>
       Proportion
       <br/>
       non-Khoisan
       <br/>
       ancestry
      </th>
      <th>
       Date of
       <br/>
       mixture
       <br/>
       (gen.)
      </th>
     </tr>
    </thead>
    <tbody>
     <tr>
      <td>
       Ju|’hoan_North
      </td>
      <td>
       0.06
      </td>
      <td>
       35
      </td>
     </tr>
     <tr>
      <td>
       Ju|’hoan_South
      </td>
      <td>
       0.07
      </td>
      <td>
       35
      </td>
     </tr>
     <tr>
      <td>
       Naro
      </td>
      <td>
       0.09
      </td>
      <td>
       37
      </td>
     </tr>
     <tr>
      <td>
       Taa_North
      </td>
      <td>
       0.11
      </td>
      <td>
       30
      </td>
     </tr>
     <tr>
      <td>
       Taa_West
      </td>
      <td>
       0.12
      </td>
      <td>
       10
      </td>
     </tr>
     <tr>
      <td>
       Taa_East
      </td>
      <td>
       0.17
      </td>
      <td>
       11
      </td>
     </tr>
     <tr>
      <td>
       ǂHoan
      </td>
      <td>
       0.23
      </td>
      <td>
       11
      </td>
     </tr>
     <tr>
      <td>
       !Xuun
      </td>
      <td>
       0.25
      </td>
      <td>
       16
      </td>
     </tr>
     <tr>
      <td>
       G||ana
      </td>
      <td>
       0.38
      </td>
      <td>
       13
      </td>
     </tr>
     <tr>
      <td>
       Hai||om
      </td>
      <td>
       0.43
      </td>
      <td>
       28
      </td>
     </tr>
     <tr>
      <td>
       Tshwa
      </td>
      <td>
       0.48
      </td>
      <td>
       19
      </td>
     </tr>
     <tr>
      <td>
       Kgalagadi
      </td>
      <td>
       0.61
      </td>
      <td>
       23
      </td>
     </tr>
     <tr>
      <td>
       Shua
      </td>
      <td>
       0.62
      </td>
      <td>
       22
      </td>
     </tr>
     <tr>
      <td>
       Khwe
      </td>
      <td>
       0.63
      </td>
      <td>
       25
      </td>
     </tr>
     <tr>
      <td>
       Tswana
      </td>
      <td>
       0.76
      </td>
      <td>
       25
      </td>
     </tr>
     <tr>
      <td>
       Mbukushu
      </td>
      <td>
       0.90
      </td>
      <td>
       14
      </td>
     </tr>
     <tr>
      <td>
       Damara
      </td>
      <td>
       0.90
      </td>
      <td>
       25
      </td>
     </tr>
     <tr>
      <td>
       Himba
      </td>
      <td>
       0.93
      </td>
      <td>
       41
      </td>
     </tr>
     <tr>
      <td>
       Wambo
      </td>
      <td>
       0.93
      </td>
      <td>
       NA
      </td>
     </tr>
    </tbody>
   </table>
   <p>
    <b>
     Supplementary Table S4: Admixture parameters in southern Africa.
    </b>
    We report the admixture proportions and times for each southern African population displayed in Figure 2 in the main text.
   </p>
  </div>
  <div class="page" data-page-id="45">
   <h1>
    3 Supplementary Methods
   </h1>
   <h3>
    3.1 Data
   </h3>
   <h4>
    3.1.1 Sampling
   </h4>
   <p>
    The southern African samples included in this study were collected in various locations in Botswana and Namibia as part of a multidisciplinary project, after ethical clearance by the Review Board of the University of Leipzig and with prior permission of the Ministry of Youth, Sport and Culture of Botswana and the Ministry of Health and Social Services of Namibia. Informed consent was obtained from all donors by carefully explaining the aims of the study and answering any arising questions with the help of translators fluent in English/Afrikaans and the local lingua franca; when necessary, a second translation from the local lingua franca into the native language of a potential donor was provided by individuals within each sampling location. Approximately 2ml of saliva were collected in tubes containing 2ml of stabilizing buffer; DNA was extracted from the saliva with a modified salting-out method
    <a href="#block-61-0">[39]</a>
    .
   </p>
   <p>
    For the purposes of this study, a minimum of 10 unrelated individuals from each linguistic branch of the three Khoisan language families as given by Güldemann
    <a href="#block-61-0">[40]</a>
    were selected from the total number of samples collected in the field, as shown in Supplementary Table S1 and Supplementary Figure
    <a href="#block-18-0">S1;</a>
    only the
    <math>
     \pm
    </math>
    Hoan branch is represented by fewer than 10 individuals due to the small number of samples available. It should be noted that the “linguistic subgroup” given in the table does not represent the same level of linguistic relationship for all the populations. The group here called Ju|’hoan_North largely corresponds to what is known as Ju|’hoan, often simply referred to as San in the literature. The Ju|’hoan_South are also known as
    <math>
     \pm
    </math>
    Kx’au||’en. Since the dialectal boundaries are as yet uncertain and since both groups partly self-identified as Ju|’hoan, we here chose geographically defined labels. The HGDP “San” samples were included with the Ju|’hoan_North sample, since they clearly stem from this population [18] and empirically we cannot detect any genetic differentiation between them using the data from this study. For the Khwe, five samples each of the ||Xokhoe and ||Anikhoe subgroups were combined, while Damara and Nama individuals were chosen to represent the greatest diversity of traditional subgroups. Since not enough samples were available from all the different Taa dialects, the subgroups of Taa investigated here were chosen based on both linguistic and geographic criteria; they do not correspond to any single linguistic unit. The Taa_West group includes speakers of the West !Xoon and !Ama dialects, Taa_North comprises speakers of the East !Xoon dialect, and Taa_East includes speakers of the Tshaasi and
    <math>
     \pm
    </math>
    Huan dialects. In addition, five samples each from different Bantu-speaking groups from southwestern Zambia (Mbukushu), Namibia (Himba and Wambo), and Botswana (Kgalagadi and Tswana) were included.
   </p>
   <p>
    The Hadza samples are a subset of those from Henn et al. [17]. Genotypes from other populations were available from other sources, as described below.
   </p>
   <h4>
    3.1.2 Genotyping
   </h4>
   <p>
    Samples were sent to Affymetrix to be genotyped on the Human Origins array. Full details about this array are in Patterson et al. [19], but briefly, SNPs were ascertained by identifying heterozygous SNPs in low-coverage sequencing of single individuals of known ancestry. The SNPs on the array can thus be split into panels of SNPs discovered in different individuals. In all analyses, we consider only autosomal SNPs. Except where otherwise noted, we restrict ourselves to using the 150,425 SNPs discovered in a single Ju|’hoan_North (HGDP “San”) individual. The exception to this are all ROLLOFF analyses (e.g., Figures 2A and 2C in the main text), where we used all 565,259 SNPs on the array.
   </p>
  </div>
  <div class="page" data-page-id="46">
   <p>
    The Dinka genotypes were taken from Meyer et al. [21] . Genotypes from other populations were described in Patterson et al. [19] .
   </p>
   <h4>
    3.1.3 Merging data from Lachance et al. [20]
   </h4>
   <p>
    Full genome sequences for five Sandawe, five Hadza, and five Baka/Bakola individuals were obtained via Complete Genomics by Lachance et al. [20] and merged with publicly available Complete Genomics data from a number of other individuals. For our purposes, the most important of these samples are 1) the five Sandawe samples, 2) the five Hadza samples, for comparison to the Hadza we have genotyped, and 3) the HapMap “YRI” samples, since we have genotyped some of these exact samples on the Affymetrix Human Origins array (and thus can get estimates of genotype error rate). We extracted the genotypes at each SNP on the Human Origins array from the Complete Genomics sequences.
   </p>
   <p>
    To detect SNPs where genotyping (on either the array or via sequencing) performed poorly, we used a set of eight Yoruba (HapMap YRI) samples that were both sequenced by Complete Genomics and genotyped on the Human Origins array. We removed all SNPs where there were any discordant genotypes between these 8 samples. There were 10,888 such SNPs. We then merged the sequenced Hadza and Sandawe samples into the southern African dataset, again considering only autosomal SNPs. For all analyses involving the Hadza and Sandawe, we included these Hadza and Sandawe individuals. We used the sequenced YRI samples only for quality control.
   </p>
   <p>
    <b>
     Quality control.
    </b>
    Since the Hadza population is quite small, we first used plink
    <a href="#block-61-1">[41]</a>
    to test whether the two sets of Hadza samples (those genotyped on the array and those genotyped by sequencing) contained any relatives. We removed one individual that appeared to be the exact same individual in the two samples (
    <math>
     \hat{\pi} = 0.98
    </math>
    when using the
    <code>
     --genome
    </code>
    option in plink)
   </p>
   <p>
    We then wanted to ensure that there were no systematic differences between samples directly genotyped on the array and those genotyped by sequencing. To look for systematic effects, we used the clustering algorithm ADMIXTURE
    <a href="#block-61-1">[42]</a>
    . Two populations, the Hadza and the Yoruba/YRI, include some samples genotyped on the array and some genotyped via sequencing. For both of these populations, there do not appear to be any substantial differences between samples typed using the two methods (Section
    <a href="#block-47-5">3.2.2,</a>
    Supplementary Figure
    <a href="#block-25-0">S8)</a>
   </p>
   <h4>
    3.1.4 Filtering “outlier” individuals
   </h4>
   <p>
    As described in the main text, for analyses where we grouped individuals into populations, we removed genetic outliers. To identify individuals that were genetic outliers with respect to their population, we performed PCA on the genotype matrix using the SNPs ascertained in a single Ju’hoan_North individual (see Section 3.2). We examined each population in turn, and removed individuals that appear as outliers in their population (Supplementary Figure
    <a href="#block-24-0">S7)</a>
    . A list of all the individuals removed from subsequent analyses is in Supplementary Table
    <a href="#block-42-0">S2.</a>
    We furthermore excluded the G|ui population, for whom we could not identify a clear genetic cluster of individuals, since three samples clustered with the southeast Kalahari groups, and two with the Nama and G||ana.
   </p>
   <h3>
    3.2 Clustering analyses
   </h3>
   <p>
    We performed clustering analysis of the genotype matrix using both PCA [22] and ADMIXTURE
    <a href="#block-61-1">[42]</a>
    . The latter is a fast implementation of the admixture model of STRUCTURE
    <a href="#block-61-2">[43]</a>
    appropriate for genome-wide
   </p>
  </div>
  <div class="page" data-page-id="47">
   <p>
    data.
   </p>
   <h4>
    3.2.1 PCA
   </h4>
   <p>
    We first performed PCA [22] using the SNPs ascertained in the Ju|’hoan_North individual and including some non-African populations (Supplementary Figure
    <a href="#block-19-0">S2)</a>
    . Nearly all of the Khoisan fall along a cline between the least admixed Khoisan populations and the rest of the African populations. The one major exception is the Nama, who are scattered in the PCA plot, indicating differential relatedness to non-African populations. We examine this further in Section
    <a href="#block-48-1">3.2.3.</a>
   </p>
   <p>
    We then considered only the African populations (excluding the Hadza and Sandawe, as we analyze them separately in Sections
    <a href="#block-57-2">3.7.1</a>
    and
    <a href="#block-58-2">3.7.2)</a>
    . As described in the main text, we performed PCA using SNPs ascertained in either a Ju|’hoan_North (HGDP “San” individual) (Supplementary Figure
    <a href="#block-21-0">S4A</a>
    ), a Yoruba (Supplementary Figure
    <a href="#block-21-0">S4B</a>
    ), or a French (Supplementary Figure
    <a href="#block-21-0">S4C</a>
    ).
   </p>
   <p>
    <b>
     Substructure within Khoisan populations.
    </b>
    The PCA (Figure 1B in main text, or Supplementary Figure
    <a href="#block-21-0">S4A</a>
    ) indicates that the Taa_West and the Hai||om are genetically substructured populations: Half of the Taa_West individuals fall into the southeast Kalahari cluster, while the other half cluster with Nama and G||ana. This genetic substructure correlates with a major linguistic boundary: the individuals falling into the southeast Kalahari cluster speak the West !Xoon dialect of Taa, while three of the other individuals speak the !Ama dialect. In the case of the Hai||om, four individuals cluster close to the Tshwa and Khwe, while the other five cluster with the !Xuun. This genetic substructure in the Hai||om reflects geographic variation: the five individuals that show affinities with the !Xuun come from close to the Angolan border in northern Namibia, where the two groups are settled in close proximity, while the remaining Hai||om come from the Etosha area further to the southwest. Conversely, for some groups the known ethnolinguistic subdivisions do not correspond to genetic distinctions. Thus, the Damara sample included four individuals from Sesfontein, and the Nama sample included five Topnaar from the Kuiseb Valley; these groups are linguistically distinct
    <a href="#block-61-2">[44]</a>
    , but appear genetically indistinguishable from other Damara and Nama, respectively. Similarly, the two Khwe subgroups cannot be distinguished from each other in the analyses.
   </p>
   <p>
    <b>
     PCA projection.
    </b>
    In Supplementary Figure
    <a href="#block-21-0">S4C</a>
    , there is a shift of some Khoe-speaking populations on the y-axis. For the Nama, we show in Section
    <a href="#block-48-1">3.2.3</a>
    that this is due to recent European ancestry. For the other populations, we speculate that this may be due to eastern African ancestry. We performed a PCA projection where we first ran the analysis using only the Ju|’hoan_North, Yoruba, and Dinka, and then projected the remaining Khoisan populations on the identified PCs. In this analysis, the Yoruba represent western African populations and the Dinka represent eastern African populations. This analysis was performed on the French-ascertained SNPs, as these are the SNPs where a potential eastern African signal in some of the southern Africans is seen in Supplementary Figure
    <a href="#block-21-0">S4C</a>
    . The results are shown in Supplementary Figure
    <a href="#block-22-0">S5.</a>
    The projected samples fall on a line between their Bantu-speaking neighbors and the Ju|’hoan_North. This suggests that the majority of the variation in admixture in these populations is due to variable levels of admixture with their neighbors. However, we cannot rule out some level of admixture with non-Bantu-speaking populations.
   </p>
   <h4>
    3.2.2 ADMIXTURE analyses
   </h4>
   <p>
    We ran ADMIXTURE
    <a href="#block-61-1">[42]</a>
    on all of the individuals in the African populations (including the French, Basque, Han and Oroqen as reference non-African populations). To prepare the data for analysis, we thinned SNPs in
   </p>
  </div>
  <div class="page" data-page-id="48">
   <p>
    LD using plink
    <a href="#block-61-1">[41]</a>
    , as suggested by the authors
    <a href="#block-61-1">[42]</a>
    . The precise command was
    <code>
     --indep-pairwise 50 10 0.1
    </code>
    . Results for different numbers of clusters are shown in Supplementary Figure
    <a href="#block-25-0">S8</a>
    (this is for the combined set of southern and eastern African populations). We recapitulate previous results showing that clustering analyses find correlations in allele frequencies between southern African populations and the Mbuti, Biaka, Hadza, and Sandawe [16] (see
    <math>
     K = 4
    </math>
    ). We additionally recapitulate (at
    <math>
     K = 8
    </math>
    ) the PCA result showing detectable structure between northwestern and southeastern Kalahari populations.
   </p>
   <h4>
    3.2.3 European ancestry in the Nama
   </h4>
   <p>
    The positions of the Nama individuals in Supplementary Figure
    <a href="#block-19-0">S2</a>
    are suggestive of post-colonial European admixture, in accordance with historic documentation of European ancestry in some Nama groups [23]. To test this, we used four-population tests [26] of the form
    <math>
     [[\text{Yoruba}, X], [\text{Han}, \text{French}]]
    </math>
    , where
    <math>
     X
    </math>
    is any southern African population. A positive
    <math>
     f_4
    </math>
    statistic indicates gene flow between
    <math>
     X
    </math>
    and a population related to the French (or alternatively gene flow between populations related to the Yoruba and Han). The most strongly positive
    <math>
     f_4
    </math>
    statistic in the southern African populations is for the Nama (Supplementary Figure
    <a href="#block-23-0">S6A</a>
    ), as expected if they have experienced European admixture. To confirm the direction of this gene flow, we used ROLLOFF [27] to test if there is detectable admixture LD in the Nama. If we use the Ju|’hoan_North and the French as the putative mixing populations, there is clear admixture LD (Supplementary Figure
    <a href="#block-23-0">S
     <a href="#block-23-0">
      6B</a>
    </a>
    ), we date this mixture to approximately five generations (
    <math>
     \approx 150
    </math>
    years) ago. The single exponential curve does not perfectly fit the curve in the Nama at shorter genetic distances (Supplementary Figure S6B), indicating that they were likely admixed with some non-Khoisan group at the time of the European admixture. This means that the extremely recent inferred date of admixture in the Nama (five generations) may still be a slight overestimate.
   </p>
   <p>
    Interestingly, a few of the Khoe-speaking populations have slightly positive
    <math>
     f_4
    </math>
    statistics in this comparison, and in the Shua the
    <math>
     f_4
    </math>
    statistic is significantly greater than zero. We speculate that some of the Khoe-speaking populations have a low level of east African ancestry, and that the relevant east African population was itself admixed with a western Eurasian population. The Shua also show a detectable signal of admixture LD, though we estimate the admixture date as much older (44 generations). This potential signal of potential east African ancestry specifically in Khoe-speaking populations is of particular interest in the light of the hypothesis that the Khoe-Kwadi languages were brought to southern Africa by a pre-Bantu pastoralist immigration from eastern Africa
    <a href="#block-61-0">[40]</a>
    .
   </p>
   <p>
    Given that the Nama are the only pastoralist Khoisan group included in our dataset, their relationship to the other Khoisan populations is of particular interest. Unfortunately, the recent European admixture they have undergone prevents us from including them in further analyses. However, as shown by the PCA based on Ju|hoan SNPs (Fig. 1B in main text), the Nama do not stand out among the other Khoisan populations, notwithstanding their distinct life-style. Rather, they cluster closely with Tshwa and G||ana foragers, who also speak languages belonging to the Khoe-Kwadi family, on a cline leading to the southeastern Kalahari cluster. To what extent this genetic proximity of the Nama to foraging groups is due to extensive admixture between immigrating pastoralists and resident foragers
    <a href="#block-61-0">[40]</a>
    or rather to a cultural diffusion of pastoralism to indigenous hunter-gatherers [33] cannot be addressed at this point.
   </p>
   <h3>
    3.3 Three- and four-population tests
   </h3>
   <p>
    Three- and four-population tests for admixture are described most thoroughly in Reich et al. [26] and Patterson et al. [19]. We used the implementation of
    <math>
     f_3
    </math>
    and
    <math>
     f_4
    </math>
    statistics available as part of the
    <i>
     TreeMix
    </i>
   </p>
  </div>
  <div class="page" data-page-id="49">
   <p>
    package [36] . In all cases standard errors for
    <math>
     f
    </math>
    —statistics were calculated in blocks of 500 SNPs (i.e. -K 500).
   </p>
   <p>
    A significantly negative
    <math>
     f_3
    </math>
    statistic is evidence for admixture in the tested population. We performed all possible three-population tests on the southern African dataset after removing outliers; all populations with negative
    <math>
     f_3
    </math>
    statistics are shown in Supplementary Table
    <a href="#block-43-0">S3.</a>
    Note that genetic drift since admixture reduces the power of this test [26] .
   </p>
   <p>
    In various places, we use four-population tests. In these tests, of the form
    <math>
     f_4(A, B; C, D)
    </math>
    (where
    <math>
     A
    </math>
    ,
    <math>
     B
    </math>
    ,
    <math>
     C
    </math>
    , and
    <math>
     D
    </math>
    are populations) a significantly positive statistic indicates gene flow between populations related to either
    <math>
     A
    </math>
    and
    <math>
     C
    </math>
    or
    <math>
     B
    </math>
    and
    <math>
     D
    </math>
    , and a significantly negative statistic indicates gene flow between populations related to
    <math>
     A
    </math>
    and
    <math>
     D
    </math>
    or
    <math>
     B
    </math>
    and
    <math>
     C
    </math>
    .
   </p>
   <h3>
    3.4 Using the decay of linkage disequilibrium to test for historical admixture
   </h3>
   <h4>
    3.4.1 Motivation
   </h4>
   <p>
    A common approach to looking for historical admixture in a population is to use clustering analyses like those implemented in STRUCTURE
    <a href="#block-61-2">[43]</a>
    and PCA [22] . These are useful approaches to summarizing the major components of variation in genetic data. More formally, these approaches attempt to model the genotypes of each sampled individual as a linear combination of unobserved allele frequency vectors. These vectors (and the best linear combination of them for approximating the genotypes of each individual) are then inferred by some algorithm. PCA and STRUCTURE-like approaches differ only in how the approximation is chosen
    <a href="#block-61-2">[45]</a>
    . In applications to population history, the inferred allele frequency vectors are often interpreted as “ancestral” frequencies from some set of populations (in the STRUCTURE-like framework), and the linear combination leading to an individual’s genotype as “admixture” levels from each of these populations. However, the inferred populations need never have existed in reality. Consider two historical scenarios: 1) an individual with 50% ancestry from a population with an allele frequency of 1 and 50% ancestry from a population with an allele frequency of 0; and 2) an individual with 100% ancestry from a population with an allele frequency of 0.5. From the point of view of a clustering algorithm, these two scenarios are identical.
   </p>
   <p>
    The above hypothetical situation provides some intuition for situations where clustering approaches might mislead. Consider the situation depicted in Supplementary Figure
    <a href="#block-26-0">S9A</a>
    . Here, there are two populations that split apart 3,200 generations in the past. Then, 40 generations in the past, 10% of one of the populations was replaced by the other (the simulation command is given in Section
    <a href="#block-50-5">3.4.3)</a>
    . We now sample 20 individuals from each population in the present day and run ADMIXTURE
    <a href="#block-61-1">[42]</a>
    . With the above intuition, it is not surprising that the algorithm does not pick up the simulated admixture event (Supplementary Figure
    <a href="#block-26-0">S9B</a>
    ,D).
   </p>
   <p>
    Our goal here is to find a method that does detect admixture in this simple situation, and to estimate the admixture proportions. To do this, we will use the decay of linkage disequilibrium (LD) rather than the allele frequencies alone. Some aspects here are motivated by clustering approaches that use LD information
    <a href="#block-61-2">[46,</a>
    <a href="#block-61-2">47]</a>
    , and a related approach is taken by Myers et al.
    <a href="#block-61-2">[48]</a>
    .
   </p>
   <h4>
    3.4.2 Methods
   </h4>
   <p>
    Consider a population
    <math>
     C
    </math>
    , which has ancestry from two populations (
    <math>
     A
    </math>
    and
    <math>
     B
    </math>
    ) with admixture proportions
    <math>
     \alpha
    </math>
    and
    <math>
     1 - \alpha
    </math>
    . Now consider two loci separated by a genetic distance of
    <math>
     x
    </math>
    cM, and let the allele frequencies at these loci in population
    <math>
     A
    </math>
    be
    <math>
     f_1^A
    </math>
    and
    <math>
     f_2^A
    </math>
    , respectively. Define
    <math>
     f_1^B
    </math>
    ,
    <math>
     f_2^B
    </math>
    ,
    <math>
     f_1^C
    </math>
    , and
    <math>
     f_2^C
    </math>
    analogously. In a given population (say
    <math>
     A
    </math>
    ), define the standard measure of linkage disequilibrium
    <math>
     D_{12}^A = f_{12}^A - f_1^A f_2^A
    </math>
    , where
    <math>
     f_{12}^A
    </math>
    is the frequency of the haplotype carrying both alleles 1 and 2 in population
    <math>
     A
    </math>
    . Suppose populations
    <math>
     A
    </math>
    and
    <math>
     B
    </math>
   </p>
  </div>
  <div class="page" data-page-id="50">
   <p>
    are in linkage equilibrium, so that
    <math>
     D_{12}^A = D_{12}^B = 0
    </math>
    . Now let
    <math>
     C
    </math>
    result from admixture between populations
    <math>
     A
    </math>
    and
    <math>
     B
    </math>
    , and for the moment assume infinite population sizes. At time
    <math>
     t
    </math>
    generations after admixture, then
    <a href="#block-61-2">[49]</a>
    :
   </p>
   <p>
    <math display="block">
     D_{12}^C(t) = \alpha(1 - \alpha)e^{-tx}[f_1^A - f_1^B][f_2^A - f_2^B]. \quad (\text{S1})
    </math>
   </p>
   <p>
    Since
    <math>
     f_1^C = \alpha f_1^A + (1 - \alpha)f_1^B
    </math>
    , we can now write
    <math>
     f_1^B = \frac{f_1^C - \alpha f_1^A}{1 - \alpha}
    </math>
    . We thus have:
   </p>
   <p>
    <math display="block">
     D_{12}^C(t) = \frac{\alpha}{1 - \alpha}e^{-tx}[f_1^A - f_1^C][f_2^A - f_2^C] \quad (\text{S2})
    </math>
   </p>
   <p>
    Now assume we have genotyped
    <math>
     m
    </math>
    SNPs in
    <math>
     n_A
    </math>
    haplotypes from population
    <math>
     A
    </math>
    and
    <math>
     n_C
    </math>
    haplotypes from population
    <math>
     C
    </math>
    . We need to estimate both allele frequencies and linkage disequilibrium in
    <math>
     C
    </math>
    ; we do this by splitting the population in half. Let
    <math>
     \hat{D}_{ij}
    </math>
    be the estimated amount of linkage disequilibrium between SNPs
    <math>
     i
    </math>
    and
    <math>
     j
    </math>
    in population
    <math>
     C
    </math>
    (using one half of the population),
    <math>
     \hat{f}_i^C
    </math>
    be the estimated allele frequency of SNP
    <math>
     i
    </math>
    in population
    <math>
     C
    </math>
    (from the other half of the population),
    <math>
     \hat{f}_i^A
    </math>
    be the estimated allele frequency of SNP
    <math>
     i
    </math>
    in population
    <math>
     A
    </math>
    (in practice a population closely related to
    <math>
     A
    </math>
    rather than
    <math>
     A
    </math>
    itself), and
    <math>
     \hat{\delta}_i
    </math>
    be
    <math>
     \hat{f}_i^A - \hat{f}_i^C
    </math>
    . We now split pairs of SNPs into bins based on the genetic distance between them. We use a bin size of 0.01 cM. In each bin, we calculate the regression coefficient
    <math>
     \hat{\beta}_x
    </math>
    from a regression of
    <math>
     \hat{D}
    </math>
    on
    <math>
     \hat{\delta}_i\hat{\delta}_j
    </math>
    . If we let
    <math>
     s
    </math>
    be the set of all pairs
    <math>
     \{i, j\}
    </math>
    of SNPs in bin
    <math>
     x
    </math>
    , then
   </p>
   <p>
    <math display="block">
     \hat{\beta}_x = \frac{\sum_{\{i,j\} \in s} \hat{\delta}_i \hat{\delta}_j \hat{D}_{ij}}{\sum_{\{i,j\} \in s} \hat{\delta}_i^2 \hat{\delta}_j^2}. \quad (\text{S3})
    </math>
   </p>
   <p>
    This is a downwardly-biased estimate of
    <math>
     \beta_x
    </math>
    due to finite sample sizes, since
    <math>
     E[\hat{\delta}_i^2] \neq \delta_i^2
    </math>
    . To correct for this, note that
    <math>
     \delta_i^2
    </math>
    is simply an
    <math>
     f_2
    </math>
    statistic [26], and
    <math>
     \hat{\delta}_i^2
    </math>
    is the biased version of the
    <math>
     f_2
    </math>
    statistic. Now call
    <math>
     \hat{f}_2
    </math>
    the biased estimate of the
    <math>
     f_2
    </math>
    distance between
    <math>
     A
    </math>
    and
    <math>
     C
    </math>
    and
    <math>
     \hat{f}_2^*
    </math>
    the unbiased estimate of this distance (from Reich et al. [26]). We can calculate a corrected version of the regression coefficient, which we call
    <math>
     \hat{\beta}_x^*
    </math>
    :
   </p>
   <p>
    <math display="block">
     \hat{\beta}_x^* = \hat{\beta}_x \frac{\hat{f}_2^2}{\hat{f}_2^{*2}}. \quad (\text{S4})
    </math>
   </p>
   <p>
    Now, returning to the population genetic parameters, recall that (from Equation S2):
   </p>
   <p>
    <math display="block">
     \beta_x = \frac{\alpha}{1 - \alpha}e^{-tx}. \quad (\text{S5})
    </math>
   </p>
   <p>
    We thus fit an exponential curve to the decay of the regression coefficient with genetic distance using the
    <code>
     nls()
    </code>
    function in R
    <a href="#block-61-2">[50]</a>
    . To remove the effects of LD in the ancestral populations, we ignore distance bins less than 0.5 cM. The amplitude of this curve is an estimate of
    <math>
     \frac{\alpha}{1 - \alpha}
    </math>
    , and the decay rate is an estimate of
    <math>
     t
    </math>
    . The interpretation of the amplitude in terms of the admixture proportion relies heavily on the assumption that population
    <math>
     A
    </math>
    has experienced little genetic drift since the admixture event, and so may not be applicable in all situations (we show below via simulations that this approximation performs well in a situation like that of the Khoisan).
   </p>
   <h4>
    3.4.3 Simulations
   </h4>
   <p>
    The above theory is valid in the absence of drift and in the presence of phased haplotypes. To test how well this works in more realistic situations, we performed coalescent simulations using
    <code>
     macs
    </code>
    <a href="#block-61-2">[51]</a>
    . We simu-
   </p>
  </div>
  <div class="page" data-page-id="51">
   <p>
    lated two populations that diverged 3,200 generations ago, each of which has an effective population size of 10,000. One population then mixes into the other 40 generations ago with some admixture fraction
    <math>
     \alpha
    </math>
    . The parameters were chosen to be reasonable for the Yoruba and Ju|’hoan_North. We simulated
    <math>
     \alpha
    </math>
    of 0, 0.1, and 0.2. The macs parameters were (for e.g.,
    <math>
     \alpha = 0.1
    </math>
    ):
   </p>
   <pre>macs 80 100000000 -t 0.0004 -r 0.0004 -I 2 40 40 -em 0.001 2 1 4000 -em 0.001025 2 1 0 -ej 0.08 2 1</pre>
   <p>
    To mimic the effects of uncertain phasing, we randomly combined the simulated chromosomes into diploids, and re-phased them using fastPHASE
    <a href="#block-61-2">[52]</a>
    . We then used the above model to estimate the admixture proportions. We simulated five replicates of each
    <math>
     \alpha
    </math>
    , and averaged the resulting curves (Supplementary Figure
    <a href="#block-27-0">S10)</a>
    . We see that the curves are approximately those predicted by theory, though they slightly overestimate the true mixture proportions. At higher mixture proportions (30% or 40%), phasing errors become a major problem and
    <math>
     \alpha
    </math>
    is severely underestimated (not shown). Two representative simulations of
    <math>
     \alpha = 0.1
    </math>
    are shown in Supplementary Figures
    <a href="#block-26-0">9C</a>
    ,E for comparison to the results from ADMIXTURE.
   </p>
   <h4>
    3.4.4 Application to the Khoisan
   </h4>
   <p>
    We then applied this procedure to the five Khoisan populations that do not show significant evidence for admixture from three-population tests (Supplementary Table
    <a href="#block-43-0">S3)</a>
    . These are the Ju|’hoan_North, Ju|’hoan_South, !Hoan, Taa_North, and Taa_East. We phased the merged dataset using fastPHASE, combining all populations (using 20 states in the HMM; i.e.
    <math>
     K = 20
    </math>
    ). Genetic distances between SNPs were taken from the HapMap
    <a href="#block-61-2">[53]</a>
    (all genetic maps are highly correlated at the scale we are considering). We used the Yoruba as a reference non-Khoisan population, and use the admixed population itself as the other reference (as described in the theory presented above). All LD decay curves for these populations are shown in Supplementary Figure
    <a href="#block-28-0">S11.</a>
    All five Khoisan populations show a clear curve; we estimate that the Ju|’hoan_North are the least admixed population, with approximately 6% non-Khoisan ancestry.
   </p>
   <p>
    A potential concern is that demographic events other than admixture (like population bottlenecks) may also lead to substantial LD in some populations. This concern arises because we use the Khoisan population twice in Equation
    <a href="#block-50-0">S2</a>
    (population
    <math>
     C
    </math>
    )—both to calculate allele frequencies and to calculate linkage disequilibrium. Though we have used different individuals for these two calculations, there could be unmodeled relationships between the individuals in the two sets. To test the robustness of the curves, we used ROLLOFF [27]. In ROLLOFF, the target population is used only to calculate LD, and two other populations are used as representatives of the putative mixing populations; see Moorjani et al. [27] for details. While demographic effects in the target population may influence LD, under the null model that the target population is unadmixed, the influence on LD will not be correlated to differences in allele frequencies between two unrelated populations. Results for using the Ju|’hoan_North as the target population and various other pairs of populations as the mixing populations are shown in Supplementary Figure
    <a href="#block-29-0">S12.</a>
    There is a clear exponential decay of LD in nearly all cases. For example, the level of LD between two distant SNPs in the Ju|’hoan_North is correlated with the divergence of those SNPs between the Yoruba and the French (Supplementary Figure
    <a href="#block-29-0">S12)</a>
    ; this is not expected if the Ju|’hoan_North are unadmixed.
   </p>
   <h4>
    3.4.5 The
    <math>
     f_4
    </math>
    ratio test in the presence of admixed ancestral populations
   </h4>
   <p>
    The
    <math>
     f_4
    </math>
    ratio test was introduced in Reich et al. [26] as a method to estimate mixture proportions in an admixed group. In our case, imagine we had samples from Chimpanzee (C), Dinka (D), Yoruba (Y), an
   </p>
  </div>
  <div class="page" data-page-id="52">
   <p>
    unadmixed Khoisan population (S), and an admixed Khoisan population (X). In this setup, the chimpanzee is an outgroup, the Yoruba and population "S" represent populations related to the admixing populations, and the Dinka are a population that split from the Yoruba before the admixture. Following the derivation in Reich et al. [26], if we let
    <math>
     \alpha
    </math>
    be the amount of Yoruba-like ancestry in population X:
   </p>
   <p>
    <math display="block">
     \frac{f_4(C, D; X, Y)}{f_4(C, D; S, Y)} = 1 - \alpha \quad (\text{S6})
    </math>
   </p>
   <p>
    However, we do not have samples from S; instead, we
    <i>
     only
    </i>
    have samples from admixed Khoisan populations. Now let
    <math>
     \alpha_1
    </math>
    be the fraction of Yoruba-like ancestry in population X, and
    <math>
     \alpha_2
    </math>
    be the fraction of Yoruba-like ancestry in population S. If we assume the Yoruba-like mixture into X and S occurred from the same population, then:
   </p>
   <p>
    <math display="block">
     \frac{f_4(C, D; X, Y)}{f_4(C, D; S, Y)} = \frac{1 - \alpha_1}{1 - \alpha_2} \quad (\text{S7})
    </math>
   </p>
   <p>
    so
   </p>
   <p>
    <math display="block">
     \alpha_1 = 1 - (1 - \alpha_2) \frac{f_4(C, D; X, Y)}{f_4(C, D; S, Y)}. \quad (\text{S8})
    </math>
   </p>
   <p>
    Of course, using this approach requires an independent method for calculating
    <math>
     \alpha_2
    </math>
    . We use the Ju'hoan_North as population S, and estimate
    <math>
     \alpha_2
    </math>
    from the linkage disequilibrium patterns as described in the previous section.
   </p>
   <h3>
    3.5 Estimating mixture dates with ROLLOFF
   </h3>
   <p>
    We used ROLLOFF [27] to estimate admixture dates for all southern African populations. To do this we set the Ju'hoan_North and Yoruba as the two mixing populations (note that the date estimates in ROLLOFF are robust to improper specification of the mixing populations [27]), and ran ROLLOFF on each population separately (Supplementary Figure
    <a href="#block-30-0">S13)</a>
    . We generated standard errors on the date estimates by performing a jackknife where we drop each chromosome in turn, as in Moorjani et al. [27]. In this analysis, we used all the SNPs on the genotyping chip, and genetic distances from the HapMap
    <a href="#block-61-2">[53]</a>
    (all genetic maps are highly correlated at the scale we are considering). For the Ju'hoan_North, we used half the sample to estimate allele frequencies and half to estimate LD, as in Section
    <a href="#block-49-2">3.4.</a>
   </p>
   <p>
    ROLLOFF estimates a single date of admixture, while in reality there may be multiple waves (or continuous) admixture in the history of a population. In these scenarios, the rate at which admixture LD decays is no longer an exponential curve, but instead a
    <i>
     mixture
    </i>
    of multiple exponential curves with different decay rates. We thus examined the residual fit from fitting a single exponential curve to the LD decay in each population (Supplementary Figure
    <a href="#block-31-0">S14)</a>
    . For a few populations, a single exponential curve does not completely describe the data, especially at shorter genetic distances. This implies that for some populations, most notably the !Xuun, G||ana, Taa_East and Taa_West, there was likely some admixture before the date estimated by ROLLOFF.
   </p>
   <h3>
    3.6 Estimating population split times
   </h3>
   <h4>
    3.6.1 Motivation
   </h4>
   <p>
    Consider two populations, X and Y. These populations split at time
    <math>
     t
    </math>
    generations in the past, and our goal is to estimate
    <math>
     t
    </math>
    from genetic data (in our case, SNPs). There are two main approaches that have been applied to this problem in the past. The first approach is based on the observation that it is often impossible to write
   </p>
  </div>
  <div class="page" data-page-id="53">
   <p>
    down the probability of seeing genetic data under a given demographic model, but it is quite easy to
    <i>
     simulate
    </i>
    data under essentially any demographic model. It is thus possible to identify demographic parameters which generate simulated data approximately similar to those observed. This is what is now called approximate Bayesian computation (see Pritchard et al.
    <a href="#block-61-2">[43]</a>
    and Beaumont et al.
    <a href="#block-61-2">[54]</a>
    for a more formal description), and this approach has been applied to estimating split times between populations in a number of applications (e.g.
    <a href="#block-62-0">[55–</a>
    57]).
   </p>
   <p>
    The other approach to this problem does not rely on simulations, but uses an explicit expression for the joint allele frequency spectrum in two populations under a given demographic history
    <a href="#block-62-1">[58]</a>
    . The joint allele frequency spectrum is influenced by a number of demographic parameters, including the effective population sizes of the populations, the time at which they split, and other considerations; Gutenkunst et al.
    <a href="#block-62-1">[58]</a>
    estimate all of these parameters.
   </p>
   <p>
    In both approaches, the demographic history of the populations modeled is assumed to be simple—a constant population size, exponential growth, or bottleneck models (or some combination thereof) are popular due to mathematical convenience. However, true population history is almost certainly more complex than can be modeled. For estimates of population split times, however, the population demography is a nuisance parameter, and we do not wish to estimate it. We will attempt to estimate split times with an approach that, in principle, is valid in situations of arbitrary demographic complexity (with some caveats to come later). The approach we will take is most similar in spirit to Gutenkunst et al.
    <a href="#block-62-1">[58]</a>
    , but tailored specifically to our data. The main idea is roughly as follows: after the split of two populations, a given chromosome from one of the populations accumulates mutations at a clock-like rate. We wish to count those mutations, and convert this count to absolute time.
   </p>
   <h4>
    3.6.2 Methods
   </h4>
   <p>
    The demographic setting for the model is presented in Supplementary Figure
    <a href="#block-32-0">S15A</a>
    . We have an outgroup population
    <math>
     O
    </math>
    , and two populations whose split time
    <math>
     t
    </math>
    we wish to estimate,
    <math>
     X
    </math>
    and
    <math>
     Y
    </math>
    . The population ancestral to
    <math>
     X
    </math>
    and
    <math>
     Y
    </math>
    is called
    <math>
     A
    </math>
    . An important modeling assumption is that after populations split, there is no migration between them. Now imagine we have identified a number of sites that are heterozygous in a single individual from
    <math>
     Y
    </math>
    (in applications later on, this will be the Ju’hoan_North; recall that this is the exact ascertainment scheme used on the Human Origins array). Looking backwards in time, these are simply all the sites where a mutation has occurred on either of the two chromosomes before they coalesce, and can be split into two groups—the mutations that arose on the lineage to
    <math>
     Y
    </math>
    (these are the red stars in Supplementary Figure
    <a href="#block-32-0">S15A</a>
    ) and those that did not (and thus were polymorphic in
    <math>
     A
    </math>
    ; these are the black stars in Supplementary Figure
    <a href="#block-32-0">S15A</a>
    , and the allelic spectrum in
    <math>
     A
    </math>
    is shown in Supplementary Figure
    <a href="#block-32-0">S15B</a>
    ). Now consider the allele frequencies in
    <math>
     X
    </math>
    . The new mutations that arose on the lineage to
    <math>
     Y
    </math>
    are of course not polymorphic in
    <math>
     X
    </math>
    , which leads to a peak of alleles with frequency zero in both the ancestral population and in
    <math>
     X
    </math>
    (Supplementary Figure
    <a href="#block-32-0">S15B</a>
    ,C). More formally, let
    <math>
     f(x)
    </math>
    be the allelic spectrum in
    <math>
     A
    </math>
    conditional on ascertainment in a single individual from
    <math>
     Y
    </math>
    . This spectrum can be split into two parts:
   </p>
   <p>
    <math display="block">
     f(x) = \begin{cases} \lambda, &amp; \text{if } x = 0 \\ (1 - \lambda)g(x), &amp; \text{otherwise} \end{cases} \quad (\text{S9})
    </math>
   </p>
   <p>
    where
    <math>
     \lambda
    </math>
    is the fraction of SNPs that were non-polymorphic in
    <math>
     A
    </math>
    (i.e., that arose on the lineage to
    <math>
     Y
    </math>
    ), and
    <math>
     g(x)
    </math>
    is the polymorphic frequency spectrum. The key parameter for our purposes is
    <math>
     \lambda
    </math>
    . If population sizes are constant,
    <math>
     g(x)
    </math>
    is linear, but in more complex situations can take other forms
    <a href="#block-62-1">[59]</a>
    . We assume
    <math>
     g(x)
    </math>
    is a quadratic of the form
    <math>
     ax^2 + bx + c
    </math>
    . This form is motivated by the fact that the observed allelic spectra are
   </p>
  </div>
  <div class="page" data-page-id="54">
   <p>
    not linear (and thus inconsistent with a constant population size), so we took the next most complicated model, which seems approximately appropriate in practice (see e.g. Supplementary Figure
    <a href="#block-36-0">S19A</a>
    ).
   </p>
   <p>
    Now we need to write down the (conditional) allelic spectrum in
    <math>
     X
    </math>
    . To do this, we use the diffusion approximation to genetic drift. Let
    <math>
     \tau
    </math>
    be the drift length (on the diffusion timescale) between
    <math>
     A
    </math>
    and
    <math>
     X
    </math>
    (we show later on how this can be estimated). Now we can write down the allelic spectrum in
    <math>
     X
    </math>
    ,
    <math>
     h(x)
    </math>
    :
   </p>
   <p>
    <math display="block">
     h(x) = \int_{0}^{1} f(y) \kappa^*(x; y, \tau) dy \quad (\text{S10})
    </math>
   </p>
   <p>
    where
    <math>
     \kappa^*(x; y, \tau)
    </math>
    is the probability that an allele at frequency
    <math>
     y
    </math>
    in
    <math>
     A
    </math>
    is now at frequency
    <math>
     x
    </math>
    in
    <math>
     X
    </math>
    , given
    <math>
     \tau
    </math>
    . This is closely related to the Kimura transition probability
    <a href="#block-62-1">[60]</a>
    , which we call
    <math>
     \kappa(x; y, \tau)
    </math>
    . However, the Kimura transition probability is the
    <i>
     polymorphic
    </i>
    transition probability, while we want the probabilities of fixation as well:
   </p>
   <p>
    <math display="block">
     \kappa^*(x; y, \tau) = \begin{cases} 1 - \int_{0}^{1} \kappa(z; y, \tau) dz - (y - \int_{0}^{1} z \kappa(z; y, \tau) dz), &amp; x = 0 \\ \kappa(x; y, \tau), &amp; 0 &lt; x &lt; 1 \\ y - \int_{0}^{1} z \kappa(z; y, \tau) dz &amp; x = 1 \end{cases} \quad (\text{S11})
    </math>
   </p>
   <p>
    Now we can write down a likelihood for observed data. Let
    <math>
     n
    </math>
    be the number of SNPs, let
    <math>
     m_i
    </math>
    be the number of sampled alleles in
    <math>
     X
    </math>
    at SNP
    <math>
     i
    </math>
    , and let
    <math>
     c_i^D
    </math>
    be the number of counts of the derived allele at SNP
    <math>
     i
    </math>
    . Let
    <math>
     c^D
    </math>
    (with no subscript) be the vector of counts of derived alleles. The likelihood for the data is then:
   </p>
   <p>
    <math display="block">
     l(c^D | \lambda, g(x)) = \prod_{i=1}^{n} \int_{0}^{1} \text{Bin}(c_i^D; m, p) h(p) dp \quad (\text{S12})
    </math>
   </p>
   <p>
    where
    <math>
     \text{Bin}(c_i^D; m, p)
    </math>
    is the binomial sampling probability. This likelihood can be calculated using numerical integration. We now have three parameters to estimate: two parameters of the polymorphic spectrum in the ancestral population (
    <math>
     a
    </math>
    , and
    <math>
     b
    </math>
    ;
    <math>
     c
    </math>
    is just a scaling factor, which we fix as 1; we normalize the spectrum so that it is a true probability distribution), and
    <math>
     \lambda
    </math>
    . We start with a linear ancestral spectrum and optimize each parameter in turn until convergence. To calculate standard errors of the estimates of these parameters, we perform a block jackknife [26] in blocks of 500 SNPs.
   </p>
   <h4>
    3.6.3 Estimation of
    <math>
     \tau
    </math>
   </h4>
   <p>
    An important parameter in this model is
    <math>
     \tau
    </math>
    , the amount of genetic drift (in diffusion units) that has occurred on the branch from
    <math>
     A
    </math>
    to
    <math>
     X
    </math>
    . All the complexity of the changes in population size on this branch are absorbed into this parameter. Formally:
   </p>
   <p>
    <math display="block">
     \tau = \int_{0}^{t} \frac{1}{2N(s)} ds \quad (\text{S13})
    </math>
   </p>
   <p>
    where
    <math>
     N(s)
    </math>
    is the effective population size at time
    <math>
     s
    </math>
    in
    <math>
     X
    </math>
    . To estimate this, we rely on SNPs ascertained by virtue of being heterozygous in a single individual in an outgroup population (in applications later on this will be the Yoruba). At a given such SNP, let the derived allele frequency be
    <math>
     a
    </math>
    in
    <math>
     A
    </math>
    ,
    <math>
     x
    </math>
    in
    <math>
     X
    </math>
    , and
    <math>
     y
    </math>
    in
    <math>
     Y
    </math>
    . Now consider the following quantities:
   </p>
   <p>
    <math display="block">
     N = x(1 - x) \quad (\text{S14})
    </math>
   </p>
  </div>
  <div class="page" data-page-id="55">
   <p>
    and
   </p>
   <p>
    <math display="block">
     D = x(1 - y). \quad (\text{S15})
    </math>
   </p>
   <p>
    Now consider the expectations of these quantities. For
    <math>
     N
    </math>
    , this is simply the expected heterozygosity; for a coalescent derivation see Wakeley
    <a href="#block-62-1">[61]</a>
    :
   </p>
   <p>
    <math display="block">
     \begin{aligned} E[N] &amp;= E[x(1 - x)] \\ &amp;= a(1 - a)e^{-\tau} \end{aligned} \quad (\text{S16})
    </math>
   </p>
   <p>
    and for
    <math>
     D
    </math>
    , since
    <math>
     x
    </math>
    and
    <math>
     y
    </math>
    are conditionally independent given
    <math>
     a
    </math>
    :
   </p>
   <p>
    <math display="block">
     E[D] = a(1 - a). \quad (\text{S17})
    </math>
   </p>
   <p>
    We now need unbiased estimators of
    <math>
     N
    </math>
    and
    <math>
     D
    </math>
    . Let there be
    <math>
     n
    </math>
    SNPs in the panel used for calculating
    <math>
     \tau
    </math>
    , let
    <math>
     \hat{x}_i
    </math>
    be the estimated frequency of the derived allele at SNP
    <math>
     i
    </math>
    in population
    <math>
     X
    </math>
    , and let
    <math>
     \hat{y}_i
    </math>
    be the estimated frequency of the derived allele at SNP
    <math>
     i
    </math>
    in population
    <math>
     Y
    </math>
    . An estimator of
    <math>
     N
    </math>
    (which we call
    <math>
     \hat{N}
    </math>
    ) is:
   </p>
   <p>
    <math display="block">
     \hat{N} = B_x + \frac{1}{n} \sum_{i=1}^{n} \hat{x}_i(1 - \hat{x}_i) \quad (\text{S18})
    </math>
   </p>
   <p>
    where
    <math>
     B_x
    </math>
    is a correction to make this an unbiased estimator (the calculation for
    <math>
     B_x
    </math>
    is Equation 4 from the Supplementary Material in Pickrell and Pritchard [36]). The estimator of
    <math>
     D
    </math>
    is the trivial one:
   </p>
   <p>
    <math display="block">
     \hat{D} = \frac{1}{n} \sum_{i=1}^{n} \hat{x}_i(1 - \hat{y}_i) \quad (\text{S19})
    </math>
   </p>
   <p>
    We thus have an estimate of
    <math>
     \tau
    </math>
    :
   </p>
   <p>
    <math display="block">
     \hat{\tau} = -\log\left(\frac{\hat{N}}{\hat{D}}\right). \quad (\text{S20})
    </math>
   </p>
   <h4>
    3.6.4 Calibration
   </h4>
   <p>
    Once we've estimated
    <math>
     \lambda
    </math>
    , we would like to convert this to
    <math>
     t
    </math>
    .
    <math>
     \lambda
    </math>
    is the proportion of all SNPs ascertained using two chromosomes in population
    <math>
     Y
    </math>
    that arose on the lineage specific to
    <math>
     Y
    </math>
    (Figure
    <a href="#block-32-0">15A</a>
    ), and can thus be written in terms of the
    <i>
     total
    </i>
    number of all mutations specific to these chromosomes (both the red and black mutations in Figure
    <a href="#block-32-0">15A</a>
    ) and the number of these that arose since
    <math>
     t
    </math>
    (the red mutations in Figure
    <a href="#block-32-0">15A</a>
    ). The former is simply the heterozygosity in population
    <math>
     Y
    </math>
    (call this
    <math>
     h
    </math>
    ), and the latter is
    <math>
     2t\mu
    </math>
    , where
    <math>
     \mu
    </math>
    is the mutation rate. This assumes that the two chromosomes do not coalesce before
    <math>
     t
    </math>
    , which is a fair assumption in our case where the estimated drift on the Ju'hoan-North lineage is small. We can thus write:
   </p>
   <p>
    <math display="block">
     \lambda = \frac{2\mu t}{h} \quad (\text{S21})
    </math>
   </p>
   <p>
    and so:
   </p>
   <p>
    <math display="block">
     t = \lambda \frac{h}{2\mu}. \quad (\text{S22})
    </math>
   </p>
   <p>
    In practice the ratio of the heterozygosity to the mutation rate must be taken from outside estimates; see Section
    <a href="#block-56-4">3.6.6</a>
    for the specific numbers used in our applications in the Khoisan.
   </p>
  </div>
  <div class="page" data-page-id="56">
   <h4>
    3.6.5 Simulations
   </h4>
   <p>
    In this section, we validate the above method using simulations and test its robustness to violations of the model. In particular, in our case there is gene flow from a non-Khoisan group into the Khoisan, so the behavior of this method in such situations is quite important. First, using
    <code>
     ms
    </code>
    <a href="#block-62-1">[62]</a>
    , we simulated samples from populations with split times at different depths. All simulations used a demography like that in Figure
    <a href="#block-32-0">15,</a>
    with samples from an outgroup that split off 3,200 generations in the past, and two populations whose split time we wish to estimate. The exact
    <code>
     ms
    </code>
    command used, for a split time of 400 generations in the past, was:
   </p>
   <pre>ms 60 3000 -t 40 -r 40 50000 -I 3 20 20 20 -ej 0.01 3 2 -ej 0.08 2 1</pre>
   <p>
    For each simulation, we then generated two sets of SNPs: one ascertained by virtue of being heterozygous in a single sample from population
    <math>
     O
    </math>
    (the outgroup), and one ascertained by virtue of being heterozygous in a single sample from population
    <math>
     Y
    </math>
    . The procedure for estimating the split time is then as follows:
   </p>
   <ol style="list-style-type: none">
    <li>
     1. Estimate
     <math>
      \tau
     </math>
     (the drift from
     <math>
      A
     </math>
     to
     <math>
      X
     </math>
     ) using the SNPs ascertained in
     <math>
      O
     </math>
     and the method in Section
     <a href="#block-54-4">3.6.3</a>
    </li>
    <li>
     2. With
     <math>
      \tau
     </math>
     fixed, estimate
     <math>
      \lambda
     </math>
     using the SNPs ascertained in
     <math>
      Y
     </math>
     and the the method from Section
     <a href="#block-53-2">3.6.2</a>
    </li>
    <li>
     3. Convert the estimated
     <math>
      \lambda
     </math>
     to generations using the calibration (the mutation rate has been set for the simulation and thus is known, and the heterozygosity is estimated in each simulation)
    </li>
   </ol>
   <p>
    We performed five simulations each at population split times of 400, 1,200, and 2,000 generations. In all cases, the population split time is well-estimated (Supplementary Figure
    <a href="#block-33-0">S16)</a>
    . We then performed simulations where populations
    <math>
     X
    </math>
    and
    <math>
     Y
    </math>
    have experienced some admixture from the “outgroup”. In all cases, we simulated 5% admixture from
    <math>
     O
    </math>
    into
    <math>
     Y
    </math>
    , and variable levels of admixture from
    <math>
     O
    </math>
    into
    <math>
     X
    </math>
    . All admixture occurred 40 generations before present. These numbers were chosen to be appropriate for the Khoisan application. The precise
    <code>
     ms
    </code>
    command (for 5% admixture in
    <math>
     X
    </math>
    ) is:
   </p>
   <pre>ms 60 3000 -t 40 -r 40 50000 -I 3 20 20 20 -em 0.002 2 1 2000 -em 0.00205 2 1 0 -em 0.002 3 1 2000 -em 0.00205 3 1 0 -ej 0.01 3 2 -ej 0.08 2 1</pre>
   <p>
    We then performed the exact same estimation procedure to get the split times (Supplementary Figure
    <a href="#block-34-0">S17)</a>
    . In all cases, the admixture leads to overestimation of the split time. This is true even when there is no admixture into
    <math>
     X
    </math>
    (but only into
    <math>
     Y
    </math>
    ).
   </p>
   <h4>
    3.6.6 Application to the Khoisan
   </h4>
   <p>
    We then applied this method to date the split of the northwestern and southeastern Kalahari populations (the time of the first split in the southern Africans in Figure 3 in the main text). Some caveats of interpretation here are warranted. First, all the Khoisan populations have some level of admixture with non-Khoisan populations. There is thus no single “split time” in their history, and any method (like the one used here) that estimates a single such time will actually be estimating a composite of several signals. Second, we have made the modeling assumption that history involves populations splitting in two with no gene flow after the split. More complex demographies are quite plausible, but render the interpretation of a split time nearly meaningless (if populations continue to exchange migrants after “splitting”, they arguably have not split at all). We thus consider strong interpretations of split times estimated from genetic data to be impossible,
   </p>
  </div>
  <div class="page" data-page-id="57">
   <p>
    but we nonetheless find the estimates to be useful in constraining the set of historical hypotheses that are consistent with the data.
   </p>
   <p>
    For all applications to the Khoisan, the population
    <math>
     Y
    </math>
    is the Ju|’hoan_North, and
    <math>
     O
    </math>
    (the outgroup) is the Yoruba. All split times are thus split times between the Ju|’hoan_North and another population. We estimated
    <math>
     \tau
    </math>
    for each population using the set of SNPs ascertained in the Yoruba, and then estimated
    <math>
     \lambda
    </math>
    using the set of SNPs ascertained in the Ju|’hoan_North. To convert from
    <math>
     \lambda
    </math>
    to
    <math>
     t
    </math>
    , we need an estimate of the ratio of the heterozygosity in the Ju|’hoan_North to the mutation rate. We took the estimate of this ratio for the Yoruba from Sun et al.
    <a href="#block-62-1">[63]</a>
    and then used the fact that the heterozygosity in the Yoruba is 95% of that in the Ju|’hoan [21]. Specifically, we averaged this ratio across six Yoruba individuals (from Supplementary Table 8 in Sun et al.
    <a href="#block-62-1">[63]</a>
    ) and multiplied by 1.04 (to account for the estimated factor by which the heterozygosity in the Ju|’hoan_North is greater than that in the Yoruba) to get an estimate of
    <math>
     \frac{h}{\mu}
    </math>
    . To get from generations to years, we use a generation time of 30 years
    <a href="#block-62-1">[64]</a>
    .
   </p>
   <p>
    The resulting split times are shown in Supplementary Figure
    <a href="#block-35-0">S18.</a>
    We plot these as a function of non-Khoisan ancestry, as the latter tends to inflate estimates of the split time (Supplementary Figure
    <a href="#block-34-0">S
     <a href="#block-34-0">
      17</a>
     )
    </a>
    . As expected, regardless of the level of admixture, the northwestern Kalahari groups have more recent split times (from the Ju|’hoan_North, who are a northwestern Kalahari group) than the southeastern Kalahari groups. The split time of interest is that with the southeastern Kalahari groups (the red points in Supplementary Figure
    <a href="#block-35-0">S18)</a>
    . The population with the least non-Khoisan ancestry is the Taa_North; we show the empirical and fitted allele frequency spectra for this population in Supplementary Figure
    <a href="#block-36-0">S19A</a>
    , and the simulated allele frequency spectrum from a simulation with an older date of 2,000 generations in Supplementary Figure
    <a href="#block-36-0">S19B</a>
    . In the Taa_North we get a point estimate of
    <math>
     823 \pm 99
    </math>
    generations (
    <math>
     \approx 25,000 \pm 3,000
    </math>
    years). However, the simulations in Supplementary Figure S17 indicate that this is likely an overestimate, and perhaps a considerable overestimate. We thus conclude that the split between the northwest and southeast Kalahari groups occurred within the last 30,000 years, and perhaps much more recently than that.
   </p>
   <h3>
    3.7
    <i>
     TreeMix
    </i>
    analyses
   </h3>
   <h4>
    3.7.1 Analysis of the Hadza
   </h4>
   <p>
    We sought to understand the relationships of the Hadza to the southern African populations. To do this, we selected populations with little admixture to represent the southern African groups (the Taa_East, Taa_North, Ju|’hoan_South, and Ju|’hoan_North; see the next section for an analysis of all the Khoisan populations excluding the Damara), African non-Khoisan groups, and non-African groups. We included the chimpanzee sequence as an outgroup. We then built a tree of these populations using
    <i>
     TreeMix
    </i>
    [36], which fits a tree to the observed variance-covariance matrix of allele frequencies (Supplementary Figure
    <a href="#block-37-0">S20A</a>
    ). The Hadza do not group with the southern African populations in this analysis; however, they are poorly modeled by a tree, as seen in the residual fit from the tree (this is the observed covariance matrix subtracted by the covariance matrix corresponding to the tree model; Supplementary Figure
    <a href="#block-37-0">S20B</a>
    ).
   </p>
   <p>
    We then allowed
    <i>
     TreeMix
    </i>
    to build the best graph, allowing for a single admixture event (Supplementary Figure
    <a href="#block-37-0">S20C</a>
    ). The algorithm infers that the Hadza are admixed between a population related to the southern African Khoisan groups and a population that is most closely related to the Dinka, a northeastern African population. The fraction of Khoisan ancestry in the Hadza is estimated at
    <math>
     23 \pm 2\%
    </math>
    (from a block jackknife in blocks of 500 SNPs). The residual fit from this graph is shown in Supplementary Figure
    <a href="#block-37-0">S20D</a>
    . The residual covariance of the Hadza with all populations except the Yoruba are less than three standard errors away from the fitted model; for the fit of the covariance between the Yoruba and the Hadza, the fit is 3.5 standard errors away. Indeed, the Yoruba are particularly poorly fitted in this graph, and the worst fit in this graph
   </p>
  </div>
  <div class="page" data-page-id="58">
   <p>
    is for the fit between the Yoruba and the Chimpanzee (Supplementary Figure
    <a href="#block-37-0">S20D</a>
    ). This poor fit for the Yoruba may indicate archaic admixture (indeed, if we allow
    <i>
     TreeMix
    </i>
    to estimate a second migration edge, it estimates admixture from an archaic population into the Yoruba [not shown]). However, other explanations are possible, and we leave this for future study.
   </p>
   <p>
    We compared the
    <i>
     TreeMix
    </i>
    estimate of this Hadza admixture fraction to that obtained by
    <math>
     f_4
    </math>
    ancestry estimation. We thus calculated
    <math>
     \frac{f_4(\text{Chimp}, \text{Yoruba}; \text{Hadza}, \text{Dinka})}{f_4(\text{Chimp}, \text{Yoruba}; \text{Ju'hoan\_North}, \text{Dinka})}
    </math>
    , which is an approximation of the fraction of Ju'hoan-like ancestry in the Hadza (though necessarily a slight overestimate due to the non-Khoisan ancestry in the Ju'hoan_North). This estimate is
    <math>
     27 \pm 1.7\%
    </math>
    , which is consistent with the
    <i>
     TreeMix
    </i>
    estimate. To ensure that this estimate is reasonable, we replaced the Hadza by the Bantu-speakers from Kenya from the HGDP (who are an eastern African population not expected to have any Khoisan ancestry) and performed the same analysis. We get an estimate of
    <math>
     3 \pm 1.2\%
    </math>
    Khoisan ancestry, confirming the reliability of the estimate.
   </p>
   <p>
    As discussed in the main text, the major caveat to the interpretation of the Hadza result is that a plausible alternative interpretation for the failure of the tree [Chimp, Ju'hoan_North, [Dinka, Hadza]] is more archaic gene flow into the ancestors of the Dinka than into the ancestors of the Hadza. There is no signal of Neandertal or Denisova ancestry in the Dinka [21], so the source of archaic gene flow would have to be an undiscovered population. We thus prefer the interpretation that the Hadza share ancestry with the Khoisan, though we acknowledge the possibility that future work will challenge this interpretation.
   </p>
   <h4>
    3.7.2 Analysis of the Sandawe
   </h4>
   <p>
    To begin our analysis of the Sandawe, we performed the same analyses as done with the Hadza. We began by building the maximum likelihood tree of populations including the Sandawe using
    <i>
     TreeMix
    </i>
    (Supplementary Figure
    <a href="#block-38-0">S21A</a>
    ). Like the Hadza, the Sandawe are poorly fitted by a tree (Supplementary Figure
    <a href="#block-38-0">S21B</a>
    ), so we allowed a single migration edge. The inferred migration event is from a population related to the Khoisan, like we previously saw in the Hadza (Supplementary Figure
    <a href="#block-38-0">S21C</a>
    ). The
    <i>
     TreeMix
    </i>
    estimate is that the Sandawe trace about 18% of their ancestry to a population related to the Khoisan.
   </p>
   <h4>
    3.7.3 West Eurasian ancestry in the Sandawe and Hadza.
   </h4>
   <p>
    We noted that the fit of the Sandawe in the
    <i>
     TreeMix
    </i>
    graph is imperfect (Supplementary Figure
    <a href="#block-38-0">S21D</a>
    ). In particular, the relationship between the Sandawe and the European populations in these data is a poor fit. On inspection, the Hadza also show a similar signal, but to a lesser extent (Supplementary Figure
    <a href="#block-37-0">S20D</a>
    ). We thus examined the Sandawe and Hadza for evidence of west Eurasian ancestry. We used
    <math>
     f_4
    </math>
    statistics of the form [Chimp, X, [French, Han]], where X is either the Sandawe or the Hadza. In both cases, this tree fails. For the Hadza, this tree fails with a Z-score of
    <math>
     -4.2
    </math>
    (
    <math>
     p = 1.3 \times 10^{-5}
    </math>
    ), and for the Sandawe, this tree fails with a Z-score of
    <math>
     -7.2
    </math>
    (
    <math>
     p = 3 \times 10^{-13}
    </math>
    ). Both of these are consistent with west Eurasian (either European or, more likely, Arabian), gene flow into these populations. To further examine this, we turned to ROLLOFF. We used Dinka and French as representatives of the mixing populations (since date estimates are robust to improperly specified reference populations). The results are shown in Supplementary Figure
    <a href="#block-39-0">S22.</a>
    Both populations show a detectable curve, though the signal is much stronger in the Sandawe than in the Hadza. The implied dates are 89 generations (
    <math>
     \approx 2500
    </math>
    years) ago for the Hadza and 66 generations (
    <math>
     \approx 2000
    </math>
    years) ago for the Sandawe. These are qualitatively similar signals to those seen by Pagani et al.
    <a href="#block-62-1">[65]</a>
    in Ethiopian populations. There are two possible historical scenarios that could lead to these signals: either the Hadza and Sandawe both directly admixed with a western Eurasian population about 2,000 years ago, or
   </p>
  </div>
  <div class="page" data-page-id="59">
   <p>
    they admixed with an east African population that was itself admixed with a western Eurasian population. The latter possibility would be consistent with known east African admixture into the Sandawe [16] .
   </p>
   <h4>
    3.7.4 Modification of
    <i>
     TreeMix
    </i>
    to include known admixture
   </h4>
   <p>
    Since all of the southern African Khoisan populations are admixed with non-Khoisan populations, any attempt to build a tree relating these populations is complicated by admixture. We wanted to examine the historical relationships of these populations before the admixture. To do this, we used the composite likelihood approach of Pickrell and Pritchard [36] , as implemented in the software
    <i>
     TreeMix
    </i>
    . Briefly, the approach is to build a graph of populations (which allows for both population splits and mixtures) that best fits the sample covariance matrix of allele frequencies [36] . In all analyses, we calculate the standard errors on the entries in the covariance matrix in blocks of 500 SNPs.
   </p>
   <p>
    In the original
    <i>
     TreeMix
    </i>
    algorithm, one first builds the best-fitting tree of populations. However, this approach is not ideal if there are many admixed populations (as in our application here, where all of the Khoisan populations are admixed). To get around this, we allow for
    <i>
     known
    </i>
    admixture events to be incorporated into this tree-building step. Imagine that there are several populations that we think
    <i>
     a priori
    </i>
    might be unadmixed (in our applications, these are the Chimpanzee, Yoruba, Dinka, Europeans, and East Asians). We first build the best tree of these unadmixed populations using the standard
    <i>
     TreeMix
    </i>
    algorithm. Now assume we have an independent estimate of the admixture level of each Khoisan population, and imagine we know the source population for the mixture.
   </p>
   <p>
    To add a Khoisan population to the tree, for each existing branch in the tree, we put in a branch leading to the new population. We then force the known admixture event into the graph with a fixed weight, update the branch lengths, and store the likelihood of the graph. After testing all possible branches, we keep the maximum likelihood graph. We then try all possible nearest-neighbor interchanges to the topology of the graph (as in the original
    <i>
     TreeMix
    </i>
    algorithm), keeping the change only if it increases the likelihood. We do this for all populations. Finally, after adding all the populations with fixed admixture weights, we optimize the admixture weights, and attempt changes to the graph structure where the source populations for the admixture events are changed.
   </p>
   <p>
    To initialize the migration weights for each Khoisan population, we used the corrected
    <math>
     f_4
    </math>
    ratio estimates from Figure 2B in the main text. To initialize the source population for the mixture events, we chose the Yoruba for all populations except the Hadza and Sandawe, which we initialized as mixing with the Dinka. We additionally initialized the Hadza and Sandawe as having 5% and 10% ancestry, respectively, related to the French, for the reasons described in Section
    <a href="#block-58-2">3.7.3</a>
    (these proportions were chosen based on rough examination of the ADMIXTURE plot (Supplementary Figure
    <a href="#block-25-0">S8)</a>
    , but are only used for initialization; the algorithm then updates these proportions to fit the data. The final estimated proportions are 13% and 17%, respectively.
   </p>
   <p>
    To obtain a measure of confidence in the resulting tree, if there are
    <math>
     K
    </math>
    blocks of 500 SNPs, we performed a bootstrap analysis where we randomly sample
    <math>
     K
    </math>
    blocks from the genome (with replacement) and re-estimate the tree. We ran this bootstrap analysis 100 times, then counted the fraction of replicates supporting each split in the tree.
   </p>
   <h4>
    3.7.5 Analysis of the Mbuti and Biaka
   </h4>
   <p>
    It has been proposed that the Mbuti and Biaka hunter-gatherers from central Africa were once part of an Africa-wide hunter-gatherer population [16] . We thus tested whether these populations share the same signal of relatedness to the Khoisan as we see in the eastern Africans. We started by looking at
    <math>
     f_4
    </math>
    statistics
   </p>
  </div>
  <div class="page" data-page-id="60">
   <p>
    of the form [Chimp, Ju|'hoan_North, [X, Dinka]], where X is either the Mbuti or the Biaka. Recall that in the main text we show that, if X is the Hadza or Sandawe, this tree fails in a way that implies admixture between southern and eastern Africa. Here, these trees also fail (for Biaka,
    <math>
     Z = 6.4
    </math>
    ,
    <math>
     p = 7.7 \times 10^{-11}
    </math>
    ; for Mbuti,
    <math>
     Z = 3.1
    </math>
    ,
    <math>
     p = 0.001
    </math>
    ). However, these failures are in the
    <i>
     opposite
    </i>
    direction to those seen for the eastern Africans. Instead, these
    <math>
     f_4
    </math>
    statistic imply either archaic ancestry in the Mbuti and Biaka, or gene flow between the Khoisan and the Dinka. We examined whether this signal is robust to the choice of ascertainment panel, instead using the SNPs ascertained in a single Yoruba individual. In this panel, we see the same signals (for Biaka,
    <math>
     Z = 9.6
    </math>
    ,
    <math>
     p &lt; 1 \times 10^{-12}
    </math>
    ; for Mbuti,
    <math>
     Z = 6.8
    </math>
    ,
    <math>
     p = 5 \times 10^{-12}
    </math>
    ). The signal of relatedness of the central Africans to the Khoisan is thus qualitatively and quantitatively different than that seen for the eastern Africans.
   </p>
   <p>
    We used the approach in the previous section to build a tree relating the Mbuti and Biaka to the Khoisan, like that done in Figure 3 in the main text. As before, we initialized the Khoisan population as having a fraction of their ancestry related to the Yoruba. We initialized the Mbuti and Biaka as having 37% and 53%, respectively, of their ancestry related to the Yoruba (these fractions are only used for initialization, but are then updated in the algorithm). The resulting tree is shown in Supplementary Figure
    <a href="#block-40-0">S23.</a>
    As expected based on the
    <math>
     f_4
    </math>
    statistics, the Mbuti and Biaka do not fall on the same branch as the Khoisan, but instead appear to descend from a population that is an outgroup to everyone else.
   </p>
  </div>
  <div class="page" data-page-id="61">
   <h1>
    References
   </h1>
   <ul style="list-style-type: none">
    <li>
     [39] Quinque, D., Kittler, R., Kayser, M., Stoneking, M. &amp; Nasidze, I. Evaluation of saliva as a source of human DNA for population and association studies.
     <i>
      Analytical biochemistry
     </i>
     <b>
      353
     </b>
     , 272–277 (2006).
    </li>
    <li>
     [40] Güldemann, T. A linguist's view: Khoe-Kwadi speakers as the earliest food-producers of southern Africa.
     <i>
      Southern African Humanities
     </i>
     <b>
      20
     </b>
     , 93–132 (2008).
    </li>
    <li>
     [41] Purcell, S.
     <i>
      et al.
     </i>
     PLINK: a tool set for whole-genome association and population-based linkage analyses.
     <i>
      Am J Hum Genet
     </i>
     <b>
      81
     </b>
     , 559–75 (2007).
    </li>
    <li>
     [42] Alexander, D. H., Novembre, J. &amp; Lange, K. Fast model-based estimation of ancestry in unrelated individuals.
     <i>
      Genome Res
     </i>
     <b>
      19
     </b>
     , 1655–64 (2009).
    </li>
    <li>
     [43] Pritchard, J. K., Stephens, M. &amp; Donnelly, P. Inference of population structure using multilocus genotype data.
     <i>
      Genetics
     </i>
     <b>
      155
     </b>
     , 945–59 (2000).
    </li>
    <li>
     [44] Haacke, W. Linguistic hypotheses on the origin of Namibian Khoekhoe speakers.
     <i>
      Southern African Humanities
     </i>
     <b>
      20
     </b>
     , 163–77 (2008).
    </li>
    <li>
     [45] Engelhardt, B. E. &amp; Stephens, M. Analysis of population structure: a unifying framework and novel methods based on sparse factor analysis.
     <i>
      PLoS Genet
     </i>
     <b>
      6
     </b>
     (2010).
    </li>
    <li>
     [46] Falush, D., Stephens, M. &amp; Pritchard, J. K. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies.
     <i>
      Genetics
     </i>
     <b>
      164
     </b>
     , 1567–87 (2003).
    </li>
    <li>
     [47] Lawson, D. J., Hellenthal, G., Myers, S. &amp; Falush, D. Inference of population structure using dense haplotype data.
     <i>
      PLoS Genet
     </i>
     <b>
      8
     </b>
     , e1002453 (2012).
    </li>
    <li>
     [48] Myers, S.
     <i>
      et al.
     </i>
     LD patterns in dense variation data reveal information about the history of human populations worldwide.
     <i>
      Presented at the 61st Annutal Meeting of The American Society of Human Genetics
     </i>
     (2011).
    </li>
    <li>
     [49] Chakraborty, R. &amp; Weiss, K. M. Admixture as a tool for finding linked genes and detecting that difference from allelic association between loci.
     <i>
      Proc Natl Acad Sci U S A
     </i>
     <b>
      85
     </b>
     , 9119–23 (1988).
    </li>
    <li>
     [50] R Development Core Team.
     <i>
      R: A Language and Environment for Statistical Computing
     </i>
     . R Foundation for Statistical Computing, Vienna, Austria (2011). URL
     <a href="http://www.R-project.org">http://www.R-project.org</a>
     . ISBN 3-900051-07-0.
    </li>
    <li>
     [51] Chen, G. K., Marjoram, P. &amp; Wall, J. D. Fast and flexible simulation of DNA sequence data.
     <i>
      Genome Res
     </i>
     <b>
      19
     </b>
     , 136–42 (2009).
    </li>
    <li>
     [52] Scheet, P. &amp; Stephens, M. A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.
     <i>
      Am J Hum Genet
     </i>
     <b>
      78
     </b>
     , 629–44 (2006).
    </li>
    <li>
     [53] Myers, S., Bottolo, L., Freeman, C., McVean, G. &amp; Donnelly, P. A fine-scale map of recombination rates and hotspots across the human genome.
     <i>
      Science
     </i>
     <b>
      310
     </b>
     , 321–324 (2005).
    </li>
    <li>
     [54] Beaumont, M. A., Zhang, W. &amp; Balding, D. J. Approximate Bayesian computation in population genetics.
     <i>
      Genetics
     </i>
     <b>
      162
     </b>
     , 2025–35 (2002).
    </li>
   </ul>
  </div>
  <div class="page" data-page-id="62">
   <ul style="list-style-type: none">
    <li>
     [55] Wollstein, A.
     <i>
      et al.
     </i>
     Demographic history of Oceania inferred from genome-wide data.
     <i>
      Curr Biol
     </i>
     <b>
      20
     </b>
     , 1983–92 (2010).
    </li>
    <li>
     [56] Patin, E.
     <i>
      et al.
     </i>
     Inferring the demographic history of African farmers and pygmy hunter-gatherers using a multilocus resequencing data set.
     <i>
      PLoS Genet
     </i>
     <b>
      5
     </b>
     , e1000448 (2009).
    </li>
    <li>
     [57] Lohmueller, K. E., Bustamante, C. D. &amp; Clark, A. G. Methods for human demographic inference using haplotype patterns from genomewide single-nucleotide polymorphism data.
     <i>
      Genetics
     </i>
     <b>
      182
     </b>
     , 217–31 (2009).
    </li>
    <li>
     [58] Gutenkunst, R. N., Hernandez, R. D., Williamson, S. H. &amp; Bustamante, C. D. Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data.
     <i>
      PLoS Genet
     </i>
     <b>
      5
     </b>
     , e1000695 (2009).
    </li>
    <li>
     [59] Keinan, A., Mullikin, J. C., Patterson, N. &amp; Reich, D. Measurement of the human allele frequency spectrum demonstrates greater genetic drift in East Asians than in Europeans.
     <i>
      Nat Genet
     </i>
     <b>
      39
     </b>
     , 1251–5 (2007).
    </li>
    <li>
     [60] Kimura, M. Solution of a process of random genetic drift with a continuous model.
     <i>
      Proceedings of the National Academy of Sciences of the United States of America
     </i>
     <b>
      41
     </b>
     , 144 (1955).
    </li>
    <li>
     [61] Wakeley, J.
     <i>
      Coalescent Theory: An Introduction
     </i>
     (Roberts &amp; Company Publishers, 2009). URL
     <a href="http://books.google.com/books?id=x3ORAgAACAAJ">http://books.google.com/books?id=x3ORAgAACAAJ</a>
     .
    </li>
    <li>
     [62] Hudson, R. R. Generating samples under a Wright-Fisher neutral model of genetic variation.
     <i>
      Bioinformatics
     </i>
     <b>
      18
     </b>
     , 337–8 (2002).
    </li>
    <li>
     [63] Sun, J. X.
     <i>
      et al.
     </i>
     A direct characterization of human mutation based on microsatellites.
     <i>
      Nat Genet
     </i>
     (2012).
    </li>
    <li>
     [64] Fenner, J. Cross-cultural estimation of the human generation interval for use in genetics-based population divergence studies.
     <i>
      American journal of physical anthropology
     </i>
     <b>
      128
     </b>
     , 415–423 (2005).
    </li>
    <li>
     [65] Pagani, L.
     <i>
      et al.
     </i>
     Ethiopian genetic diversity reveals linguistic stratification and complex influences on the Ethiopian gene pool.
     <i>
      Am J Hum Genet
     </i>
     <b>
      91
     </b>
     , 83–96 (2012).
    </li>
   </ul>
  </div>
 </body>
</html>
